<!DOCTYPE html>

<html>

<head>

<style>

body {

  background-color: #33333f;

}

.paragraph {

  width: 400px;

  padding: 20px;

  font-family: Arial, sans-serif;

  font-size: 10pt;

  background-color: white;

  border-radius: 6px;

  position: absolute;

  display: none;

}



.paragraph::before {

  content: "";

  width: 12px;

  height: 12px;

  transform: rotate(45deg);

  background-color: white;

  position: absolute;

  left: -6px;

  top: 68px;

}

</style>

</head>

    <body>

        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1415pt" height="1295pt" viewBox="0.00 0.00 1414.81 1295.29">



         <g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 1291.2911)">



<title>prof</title>

<!-- alpaca -->

<g id="node1" class="node">

<title>alpaca</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1057.8623" cy="-955.6279" rx="35.8921" ry="18"/>

<text text-anchor="middle" x="1057.8623" y="-951.4279" font-family="Times,serif" font-size="14.00" fill="#000000">alpaca</text>

</g>

<!-- bat -->

<g id="node2" class="node">

<title>bat</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1074.8623" cy="-866.8279" rx="27" ry="18"/>

<text text-anchor="middle" x="1074.8623" y="-862.6279" font-family="Times,serif" font-size="14.00" fill="#000000">bat</text>

</g>

<!-- alpaca&#45;&gt;bat -->

<g id="edge1" class="edge">

<title>alpaca-&gt;bat</title>

<path fill="none" stroke="#000000" d="M1053.5591,-937.7473C1051.7913,-927.3871 1050.7913,-914.1871 1053.8623,-902.8279 1054.8454,-899.1914 1056.2972,-895.5451 1057.9864,-892.0433"/>

<polygon fill="#000000" stroke="#000000" points="1061.0891,-893.6647 1062.8533,-883.2177 1054.9594,-890.2844 1061.0891,-893.6647"/>

<text text-anchor="middle" x="1057.3623" y="-907.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge1_0"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7192909/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1024.0591,-912.7473 1024.0591,-934.7473 1051.0591,-934.7473 1051.0591,-912.7473 1024.0591,-912.7473"/>

<text text-anchor="start" x="1026.6685" y="-919.1473" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge1_0" class="icon" pointer-events="all"><circle cx="1064.7205" cy="-922.2149" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1064.7205" cy="-926.2149" r="0.75" fill="gold"/><rect x="1064.2205" y="-924.2149" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- bat&#45;&gt;alpaca -->

<g id="edge3" class="edge">

<title>bat-&gt;alpaca</title>

<path fill="none" stroke="#000000" d="M1071.4272,-884.7712C1069.078,-897.042 1065.9165,-913.5566 1063.2365,-927.5553"/>

<polygon fill="#000000" stroke="#000000" points="1059.7453,-927.1781 1061.3025,-937.6579 1066.6204,-928.4944 1059.7453,-927.1781"/>

<text text-anchor="middle" x="1072.3623" y="-907.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge3_1"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7192909/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1074.9272,-887.7712 1074.9272,-909.7712 1101.9272,-909.7712 1101.9272,-887.7712 1074.9272,-887.7712"/>

<text text-anchor="start" x="1077.5366" y="-894.1712" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge3_1" class="icon" pointer-events="all"><circle cx="1115.5885999999998" cy="-897.2388" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1115.5885999999998" cy="-901.2388" r="0.75" fill="gold"/><rect x="1115.0885999999998" y="-899.2388" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- civet -->

<g id="node3" class="node">

<title>civet</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1026.8623" cy="-600.4279" rx="29.6127" ry="18"/>

<text text-anchor="middle" x="1026.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">civet</text>

</g>

<!-- bat&#45;&gt;civet -->

<g id="edge2" class="edge">

<title>bat-&gt;civet</title>

<path fill="none" stroke="#000000" stroke-width="4.2" d="M1096.5335,-855.7956C1102.6452,-853.1064 1109.3944,-850.5199 1115.8623,-848.8279 1238.8904,-816.6436 1457.3671,-866.5212 1387.8623,-760.0279 1302.4148,-629.1076 1208.3977,-682.6544 1065.8623,-618.4279 1064.0266,-617.6007 1062.1443,-616.7478 1060.2456,-615.8838"/>

<polygon fill="#000000" stroke="#000000" stroke-width="4.2" points="1061.3783,-612.3605 1050.7561,-611.5388 1058.3184,-619.0433 1061.3783,-612.3605"/>

<text text-anchor="middle" x="1380.8623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">11</text>

<g id="a_edge2_2"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc2831534/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-1098.7956 1100.0335,-1120.7956 1127.0335,-1120.7956 1127.0335,-1098.7956 1100.0335,-1098.7956"/>

<text text-anchor="start" x="1102.6429" y="-1105.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_2" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-1108.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-1112.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-1110.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_3"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6880172/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-1074.7956 1100.0335,-1096.7956 1127.0335,-1096.7956 1127.0335,-1074.7956 1100.0335,-1074.7956"/>

<text text-anchor="start" x="1102.6429" y="-1081.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_3" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-1084.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-1088.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-1086.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_4"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7106130/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-1050.7956 1100.0335,-1072.7956 1127.0335,-1072.7956 1127.0335,-1050.7956 1100.0335,-1050.7956"/>

<text text-anchor="start" x="1102.6429" y="-1057.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_4" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-1060.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-1064.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-1062.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_5"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7163996/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-1026.7956 1100.0335,-1048.7956 1127.0335,-1048.7956 1127.0335,-1026.7956 1100.0335,-1026.7956"/>

<text text-anchor="start" x="1102.6429" y="-1033.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_5" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-1036.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-1040.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-1038.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_6"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7112277/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-1002.7956 1100.0335,-1024.7956 1127.0335,-1024.7956 1127.0335,-1002.7956 1100.0335,-1002.7956"/>

<text text-anchor="start" x="1102.6429" y="-1009.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_6" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-1012.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-1016.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-1014.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_7"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7154514/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-978.7956 1100.0335,-1000.7956 1127.0335,-1000.7956 1127.0335,-978.7956 1100.0335,-978.7956"/>

<text text-anchor="start" x="1102.6429" y="-985.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_7" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-988.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-992.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-990.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_8"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3119930/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-954.7956 1100.0335,-976.7956 1127.0335,-976.7956 1127.0335,-954.7956 1100.0335,-954.7956"/>

<text text-anchor="start" x="1102.6429" y="-961.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_8" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-964.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-968.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-966.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_9"><a xlink:href="https://api.semanticscholar.org/v1/paper/4bc7c77f00f5b46fea1d6ef83c44d7a58b687eef" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-930.7956 1100.0335,-952.7956 1127.0335,-952.7956 1127.0335,-930.7956 1100.0335,-930.7956"/>

<text text-anchor="start" x="1102.6429" y="-937.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_9" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-940.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-944.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-942.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_10"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3647518/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-906.7956 1100.0335,-928.7956 1127.0335,-928.7956 1127.0335,-906.7956 1100.0335,-906.7956"/>

<text text-anchor="start" x="1102.6429" y="-913.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_10" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-916.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-920.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-918.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_11"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7180649/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-882.7956 1100.0335,-904.7956 1127.0335,-904.7956 1127.0335,-882.7956 1100.0335,-882.7956"/>

<text text-anchor="start" x="1102.6429" y="-889.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_11" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-892.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-896.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-894.2632" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge2_12"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129017/pdf/" xlink:title="11">

<polygon fill="none" stroke="#000000" points="1100.0335,-858.7956 1100.0335,-880.7956 1127.0335,-880.7956 1127.0335,-858.7956 1100.0335,-858.7956"/>

<text text-anchor="start" x="1102.6429" y="-865.1956" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge2_12" class="icon" pointer-events="all"><circle cx="1140.6949" cy="-868.2632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.6949" cy="-872.2632" r="0.75" fill="gold"/><rect x="1140.1949" y="-870.2632" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- camel -->

<g id="node4" class="node">

<title>camel</title>

<ellipse fill="#add8e6" stroke="#000000" cx="180.8623" cy="-600.4279" rx="34.1852" ry="18"/>

<text text-anchor="middle" x="180.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">camel</text>

</g>

<!-- bat&#45;&gt;camel -->

<g id="edge4" class="edge">

<title>bat-&gt;camel</title>

<path fill="none" stroke="#000000" stroke-width="2.8" d="M1047.7563,-865.8155C896.9353,-860.0552 166.7892,-830.45 134.8623,-796.0279 123.9817,-784.297 132.6247,-775.8707 134.8623,-760.0279 138.4809,-734.407 159.6322,-666.2743 171.9561,-627.8248"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.8" points="175.2961,-628.8712 175.0297,-618.2797 168.633,-626.7256 175.2961,-628.8712"/>

<text text-anchor="middle" x="146.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">4</text>

<g id="a_edge4_13"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6880172/pdf/" xlink:title="4">

<polygon fill="none" stroke="#000000" points="1018.2563,-840.8155 1018.2563,-862.8155 1045.2563,-862.8155 1045.2563,-840.8155 1018.2563,-840.8155"/>

<text text-anchor="start" x="1020.8657" y="-847.2155" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge4_13" class="icon" pointer-events="all"><circle cx="1058.9177" cy="-850.2831" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1058.9177" cy="-854.2831" r="0.75" fill="gold"/><rect x="1058.4177" y="-852.2831" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge4_14"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6818114/pdf/" xlink:title="4">

<polygon fill="none" stroke="#000000" points="1018.2563,-816.8155 1018.2563,-838.8155 1045.2563,-838.8155 1045.2563,-816.8155 1018.2563,-816.8155"/>

<text text-anchor="start" x="1020.8657" y="-823.2155" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge4_14" class="icon" pointer-events="all"><circle cx="1058.9177" cy="-826.2831" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1058.9177" cy="-830.2831" r="0.75" fill="gold"/><rect x="1058.4177" y="-828.2831" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge4_15"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7149753/pdf/" xlink:title="4">

<polygon fill="none" stroke="#000000" points="1018.2563,-792.8155 1018.2563,-814.8155 1045.2563,-814.8155 1045.2563,-792.8155 1018.2563,-792.8155"/>

<text text-anchor="start" x="1020.8657" y="-799.2155" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge4_15" class="icon" pointer-events="all"><circle cx="1058.9177" cy="-802.2831" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1058.9177" cy="-806.2831" r="0.75" fill="gold"/><rect x="1058.4177" y="-804.2831" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge4_16"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7080179/pdf/" xlink:title="4">

<polygon fill="none" stroke="#000000" points="1018.2563,-768.8155 1018.2563,-790.8155 1045.2563,-790.8155 1045.2563,-768.8155 1018.2563,-768.8155"/>

<text text-anchor="start" x="1020.8657" y="-775.2155" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge4_16" class="icon" pointer-events="all"><circle cx="1058.9177" cy="-778.2831" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1058.9177" cy="-782.2831" r="0.75" fill="gold"/><rect x="1058.4177" y="-780.2831" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- dromedary -->

<g id="node5" class="node">

<title>dromedary</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1325.8623" cy="-778.0279" rx="52.7315" ry="18"/>

<text text-anchor="middle" x="1325.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">dromedary</text>

</g>

<!-- bat&#45;&gt;dromedary -->

<g id="edge5" class="edge">

<title>bat-&gt;dromedary</title>

<path fill="none" stroke="#000000" stroke-width="2.6" d="M1096.5617,-855.9008C1102.6735,-853.212 1109.4158,-850.5994 1115.8623,-848.8279 1178.9054,-831.5037 1201.803,-856.6631 1261.8623,-830.8279 1277.1287,-824.2608 1291.6339,-812.8886 1302.9632,-802.3763"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.6" points="1305.5175,-804.775 1310.2694,-795.3055 1300.6494,-799.7448 1305.5175,-804.775"/>

<text text-anchor="middle" x="1291.3623" y="-818.2279" font-family="Times,serif" font-size="14.00" fill="#000000">3</text>

<g id="a_edge5_17"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7154514/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="1100.0617,-830.9008 1100.0617,-852.9008 1127.0617,-852.9008 1127.0617,-830.9008 1100.0617,-830.9008"/>

<text text-anchor="start" x="1102.6711" y="-837.3008" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge5_17" class="icon" pointer-events="all"><circle cx="1140.7231" cy="-840.3684" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.7231" cy="-844.3684" r="0.75" fill="gold"/><rect x="1140.2231" y="-842.3684" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge5_18"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7180649/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="1100.0617,-806.9008 1100.0617,-828.9008 1127.0617,-828.9008 1127.0617,-806.9008 1100.0617,-806.9008"/>

<text text-anchor="start" x="1102.6711" y="-813.3008" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge5_18" class="icon" pointer-events="all"><circle cx="1140.7231" cy="-816.3684" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.7231" cy="-820.3684" r="0.75" fill="gold"/><rect x="1140.2231" y="-818.3684" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge5_19"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129017/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="1100.0617,-782.9008 1100.0617,-804.9008 1127.0617,-804.9008 1127.0617,-782.9008 1100.0617,-782.9008"/>

<text text-anchor="start" x="1102.6711" y="-789.3008" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge5_19" class="icon" pointer-events="all"><circle cx="1140.7231" cy="-792.3684" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1140.7231" cy="-796.3684" r="0.75" fill="gold"/><rect x="1140.2231" y="-794.3684" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- horse -->

<g id="node6" class="node">

<title>horse</title>

<ellipse fill="#add8e6" stroke="#000000" cx="55.8623" cy="-778.0279" rx="31.9012" ry="18"/>

<text text-anchor="middle" x="55.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">horse</text>

</g>

<!-- bat&#45;&gt;horse -->

<g id="edge6" class="edge">

<title>bat-&gt;horse</title>

<path fill="none" stroke="#000000" d="M1047.7428,-866.4754C894.3037,-864.4005 140.3087,-853.098 96.8623,-830.8279 85.5192,-825.0136 76.3626,-814.47 69.6042,-804.3253"/>

<polygon fill="#000000" stroke="#000000" points="72.4942,-802.3426 64.2876,-795.6417 66.5243,-805.9977 72.4942,-802.3426"/>

<text text-anchor="middle" x="100.3623" y="-818.2279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge6_20"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6849485/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1018.2428,-869.4754 1018.2428,-891.4754 1045.2428,-891.4754 1045.2428,-869.4754 1018.2428,-869.4754"/>

<text text-anchor="start" x="1020.8522" y="-875.8754" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge6_20" class="icon" pointer-events="all"><circle cx="1058.9042" cy="-878.943" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1058.9042" cy="-882.943" r="0.75" fill="gold"/><rect x="1058.4042" y="-880.943" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- human -->

<g id="node7" class="node">

<title>human</title>

<ellipse fill="#add8e6" stroke="#000000" cx="839.8623" cy="-689.2279" rx="37.1424" ry="18"/>

<text text-anchor="middle" x="839.8623" y="-685.0279" font-family="Times,serif" font-size="14.00" fill="#000000">human</text>

</g>

<!-- bat&#45;&gt;human -->

<g id="edge7" class="edge">

<title>bat-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="5.6" d="M1059.972,-851.2911C1032.4251,-822.5829 975.8437,-763.792 970.8623,-760.0279 941.5258,-737.8609 904.3506,-718.4709 877.1023,-705.6147"/>

<polygon fill="#000000" stroke="#000000" stroke-width="5.6" points="879.0409,-701.1124 867.9073,-701.3459 874.9142,-710.0013 879.0409,-701.1124"/>

<text text-anchor="middle" x="1012.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">18</text>

<g id="a_edge7_21"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7217226/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1262.2911 1063.472,-1284.2911 1090.472,-1284.2911 1090.472,-1262.2911 1063.472,-1262.2911"/>

<text text-anchor="start" x="1066.0814" y="-1268.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_21" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1271.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1275.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1273.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_22"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc5394239/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1238.2911 1063.472,-1260.2911 1090.472,-1260.2911 1090.472,-1238.2911 1063.472,-1238.2911"/>

<text text-anchor="start" x="1066.0814" y="-1244.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_22" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1247.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1251.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1249.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_23"><a xlink:href="https://api.semanticscholar.org/v1/paper/e2aee146a962f1d07174aa3266f6fa180d20bf4a" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1214.2911 1063.472,-1236.2911 1090.472,-1236.2911 1090.472,-1214.2911 1063.472,-1214.2911"/>

<text text-anchor="start" x="1066.0814" y="-1220.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_23" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1223.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1227.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1225.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_24"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7103766/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1190.2911 1063.472,-1212.2911 1090.472,-1212.2911 1090.472,-1190.2911 1063.472,-1190.2911"/>

<text text-anchor="start" x="1066.0814" y="-1196.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_24" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1199.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1203.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1201.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_25"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7175879/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1166.2911 1063.472,-1188.2911 1090.472,-1188.2911 1090.472,-1166.2911 1063.472,-1166.2911"/>

<text text-anchor="start" x="1066.0814" y="-1172.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_25" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1175.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1179.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1177.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_26"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.10.20056697" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1142.2911 1063.472,-1164.2911 1090.472,-1164.2911 1090.472,-1142.2911 1063.472,-1142.2911"/>

<text text-anchor="start" x="1066.0814" y="-1148.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_26" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1151.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1155.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1153.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_27"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7118423/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1118.2911 1063.472,-1140.2911 1090.472,-1140.2911 1090.472,-1118.2911 1063.472,-1118.2911"/>

<text text-anchor="start" x="1066.0814" y="-1124.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_27" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1127.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1131.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1129.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_28"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7087630/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1094.2911 1063.472,-1116.2911 1090.472,-1116.2911 1090.472,-1094.2911 1063.472,-1094.2911"/>

<text text-anchor="start" x="1066.0814" y="-1100.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_28" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1103.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1107.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1105.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_29"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7087236/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1070.2911 1063.472,-1092.2911 1090.472,-1092.2911 1090.472,-1070.2911 1063.472,-1070.2911"/>

<text text-anchor="start" x="1066.0814" y="-1076.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_29" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1079.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1083.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1081.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_30"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc5640221/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1046.2911 1063.472,-1068.2911 1090.472,-1068.2911 1090.472,-1046.2911 1063.472,-1046.2911"/>

<text text-anchor="start" x="1066.0814" y="-1052.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_30" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1055.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1059.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1057.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_31"><a xlink:href="https://api.semanticscholar.org/v1/paper/f84886d52a2dc934793dbaf2272f9a16c8da275c" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-1022.2911 1063.472,-1044.2911 1090.472,-1044.2911 1090.472,-1022.2911 1063.472,-1022.2911"/>

<text text-anchor="start" x="1066.0814" y="-1028.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_31" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1031.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1035.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1033.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_32"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7165814/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-998.2911 1063.472,-1020.2911 1090.472,-1020.2911 1090.472,-998.2911 1063.472,-998.2911"/>

<text text-anchor="start" x="1066.0814" y="-1004.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_32" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-1007.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-1011.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-1009.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_33"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.22.20075572" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-974.2911 1063.472,-996.2911 1090.472,-996.2911 1090.472,-974.2911 1063.472,-974.2911"/>

<text text-anchor="start" x="1066.0814" y="-980.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_33" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-983.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-987.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-985.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<text text-anchor="start" x="1066.0814" y="-956.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<polygon fill="none" stroke="#000000" points="1063.472,-926.2911 1063.472,-948.2911 1090.472,-948.2911 1090.472,-926.2911 1063.472,-926.2911"/>

<text text-anchor="start" x="1066.0814" y="-932.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="a_edge7_36"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7254014/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-902.2911 1063.472,-924.2911 1090.472,-924.2911 1090.472,-902.2911 1063.472,-902.2911"/>

<text text-anchor="start" x="1066.0814" y="-908.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_36" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-911.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-915.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-913.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_37"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7179739/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-878.2911 1063.472,-900.2911 1090.472,-900.2911 1090.472,-878.2911 1063.472,-878.2911"/>

<text text-anchor="start" x="1066.0814" y="-884.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_37" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-887.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-891.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-889.7587" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge7_38"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="18">

<polygon fill="none" stroke="#000000" points="1063.472,-854.2911 1063.472,-876.2911 1090.472,-876.2911 1090.472,-854.2911 1063.472,-854.2911"/>

<text text-anchor="start" x="1066.0814" y="-860.6911" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge7_38" class="icon" pointer-events="all"><circle cx="1104.1334" cy="-863.7587" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1104.1334" cy="-867.7587" r="0.75" fill="gold"/><rect x="1103.6334" y="-865.7587" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- pangolin -->

<g id="node8" class="node">

<title>pangolin</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1074.8623" cy="-778.0279" rx="44.6665" ry="18"/>

<text text-anchor="middle" x="1074.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">pangolin</text>

</g>

<!-- bat&#45;&gt;pangolin -->

<g id="edge8" class="edge">

<title>bat-&gt;pangolin</title>

<path fill="none" stroke="#000000" d="M1074.8623,-848.4285C1074.8623,-836.3228 1074.8623,-820.2355 1074.8623,-806.4953"/>

<polygon fill="#000000" stroke="#000000" points="1078.3624,-806.0999 1074.8623,-796.0999 1071.3624,-806.0999 1078.3624,-806.0999"/>

<text text-anchor="middle" x="1078.3623" y="-818.2279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge8_39"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7202812/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1061.8623,-823.4285 1061.8623,-845.4285 1088.8623,-845.4285 1088.8623,-823.4285 1061.8623,-823.4285"/>

<text text-anchor="start" x="1064.4717" y="-829.8285" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge8_39" class="icon" pointer-events="all"><circle cx="1102.5237" cy="-832.8960999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1102.5237" cy="-836.8960999999999" r="0.75" fill="gold"/><rect x="1102.0237" y="-834.8960999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- racoon -->

<g id="node9" class="node">

<title>racoon</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1174.8623" cy="-778.0279" rx="37.1213" ry="18"/>

<text text-anchor="middle" x="1174.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">racoon</text>

</g>

<!-- bat&#45;&gt;racoon -->

<g id="edge9" class="edge">

<title>bat-&gt;racoon</title>

<path fill="none" stroke="#000000" d="M1091.3752,-852.1644C1107.058,-838.2381 1130.8752,-817.0884 1149.0627,-800.9379"/>

<polygon fill="#000000" stroke="#000000" points="1151.82,-803.1702 1156.9735,-793.9132 1147.1721,-797.936 1151.82,-803.1702"/>

<text text-anchor="middle" x="1137.3623" y="-818.2279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge9_40"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3119930/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1094.8752,-855.1644 1094.8752,-877.1644 1121.8752,-877.1644 1121.8752,-855.1644 1094.8752,-855.1644"/>

<text text-anchor="start" x="1097.4846" y="-861.5644" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge9_40" class="icon" pointer-events="all"><circle cx="1135.5366" cy="-864.632" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1135.5366" cy="-868.632" r="0.75" fill="gold"/><rect x="1135.0366" y="-866.632" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- civet&#45;&gt;human -->

<g id="edge16" class="edge">

<title>civet-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="6.4" d="M1015.8741,-617.3661C1007.3937,-629.1168 994.6362,-644.2192 979.8623,-653.2279 951.6188,-670.4498 915.344,-679.481 886.7962,-684.1903"/>

<polygon fill="#000000" stroke="#000000" stroke-width="6.4" points="885.6985,-678.6929 876.6574,-685.737 887.3876,-689.7648 885.6985,-678.6929"/>

<text text-anchor="middle" x="1006.8623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">22</text>

<g id="a_edge16_41"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc2831534/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-592.3661 986.3741,-614.3661 1013.3741,-614.3661 1013.3741,-592.3661 986.3741,-592.3661"/>

<text text-anchor="start" x="988.9835" y="-598.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_41" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-601.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-605.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-603.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_42"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3647518/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-568.3661 986.3741,-590.3661 1013.3741,-590.3661 1013.3741,-568.3661 986.3741,-568.3661"/>

<text text-anchor="start" x="988.9835" y="-574.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_42" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-577.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-581.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-579.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_43"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3630921/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-544.3661 986.3741,-566.3661 1013.3741,-566.3661 1013.3741,-544.3661 986.3741,-544.3661"/>

<text text-anchor="start" x="988.9835" y="-550.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_43" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-553.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-557.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-555.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_44"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7260574/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-520.3661 986.3741,-542.3661 1013.3741,-542.3661 1013.3741,-520.3661 986.3741,-520.3661"/>

<text text-anchor="start" x="988.9835" y="-526.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_44" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-529.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-533.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-531.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_45"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.11.20054643" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-496.3661 986.3741,-518.3661 1013.3741,-518.3661 1013.3741,-496.3661 986.3741,-496.3661"/>

<text text-anchor="start" x="988.9835" y="-502.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_45" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-505.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-509.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-507.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_46"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.05.24.113175" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-472.3661 986.3741,-494.3661 1013.3741,-494.3661 1013.3741,-472.3661 986.3741,-472.3661"/>

<text text-anchor="start" x="988.9835" y="-478.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_46" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-481.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-485.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-483.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_47"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7106130/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-448.3661 986.3741,-470.3661 1013.3741,-470.3661 1013.3741,-448.3661 986.3741,-448.3661"/>

<text text-anchor="start" x="988.9835" y="-454.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_47" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-457.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-461.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-459.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_48"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7232928/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-424.3661 986.3741,-446.3661 1013.3741,-446.3661 1013.3741,-424.3661 986.3741,-424.3661"/>

<text text-anchor="start" x="988.9835" y="-430.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_48" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-433.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-437.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-435.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_49"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7163996/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-400.3661 986.3741,-422.3661 1013.3741,-422.3661 1013.3741,-400.3661 986.3741,-400.3661"/>

<text text-anchor="start" x="988.9835" y="-406.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_49" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-409.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-413.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-411.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_50"><a xlink:href="https://api.semanticscholar.org/10.1128/jvi.00442-08" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-376.3661 986.3741,-398.3661 1013.3741,-398.3661 1013.3741,-376.3661 986.3741,-376.3661"/>

<text text-anchor="start" x="988.9835" y="-382.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_50" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-385.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-389.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-387.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_51"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7081895/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-352.3661 986.3741,-374.3661 1013.3741,-374.3661 1013.3741,-352.3661 986.3741,-352.3661"/>

<text text-anchor="start" x="988.9835" y="-358.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_51" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-361.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-365.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-363.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_52"><a xlink:href="https://api.semanticscholar.org/v1/paper/a18236e9c75536c022502b9a9d0efdc6b0a3f0c2" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-328.3661 986.3741,-350.3661 1013.3741,-350.3661 1013.3741,-328.3661 986.3741,-328.3661"/>

<text text-anchor="start" x="988.9835" y="-334.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_52" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-337.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-341.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-339.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_53"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7112277/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-304.3661 986.3741,-326.3661 1013.3741,-326.3661 1013.3741,-304.3661 986.3741,-304.3661"/>

<text text-anchor="start" x="988.9835" y="-310.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_53" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-313.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-317.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-315.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_54"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7193031/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-280.3661 986.3741,-302.3661 1013.3741,-302.3661 1013.3741,-280.3661 986.3741,-280.3661"/>

<text text-anchor="start" x="988.9835" y="-286.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_54" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-289.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-293.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-291.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_55"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.06.20055327" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-256.3661 986.3741,-278.3661 1013.3741,-278.3661 1013.3741,-256.3661 986.3741,-256.3661"/>

<text text-anchor="start" x="988.9835" y="-262.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_55" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-265.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-269.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-267.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_56"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7154514/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-232.3661 986.3741,-254.3661 1013.3741,-254.3661 1013.3741,-232.3661 986.3741,-232.3661"/>

<text text-anchor="start" x="988.9835" y="-238.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_56" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-241.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-245.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-243.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_57"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3119930/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-208.3661 986.3741,-230.3661 1013.3741,-230.3661 1013.3741,-208.3661 986.3741,-208.3661"/>

<text text-anchor="start" x="988.9835" y="-214.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_57" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-217.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-221.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-219.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_58"><a xlink:href="https://api.semanticscholar.org/v1/paper/4bc7c77f00f5b46fea1d6ef83c44d7a58b687eef" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-184.3661 986.3741,-206.3661 1013.3741,-206.3661 1013.3741,-184.3661 986.3741,-184.3661"/>

<text text-anchor="start" x="988.9835" y="-190.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_58" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-193.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-197.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-195.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_59"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6021449/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-160.3661 986.3741,-182.3661 1013.3741,-182.3661 1013.3741,-160.3661 986.3741,-160.3661"/>

<text text-anchor="start" x="988.9835" y="-166.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_59" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-169.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-173.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-171.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_60"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc2572002/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-136.3661 986.3741,-158.3661 1013.3741,-158.3661 1013.3741,-136.3661 986.3741,-136.3661"/>

<text text-anchor="start" x="988.9835" y="-142.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_60" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-145.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-149.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-147.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_61"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7120285/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-112.3661 986.3741,-134.3661 1013.3741,-134.3661 1013.3741,-112.3661 986.3741,-112.3661"/>

<text text-anchor="start" x="988.9835" y="-118.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_61" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-121.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-125.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-123.8337" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge16_62"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129017/pdf/" xlink:title="22">

<polygon fill="none" stroke="#000000" points="986.3741,-88.3661 986.3741,-110.3661 1013.3741,-110.3661 1013.3741,-88.3661 986.3741,-88.3661"/>

<text text-anchor="start" x="988.9835" y="-94.7661" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge16_62" class="icon" pointer-events="all"><circle cx="1027.0355" cy="-97.8337" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1027.0355" cy="-101.8337" r="0.75" fill="gold"/><rect x="1026.5355" y="-99.8337" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- camel&#45;&gt;human -->

<g id="edge13" class="edge">

<title>camel-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="7" d="M214.5493,-604C268.838,-609.8904 378.995,-622.3951 471.8623,-636.4279 513.8257,-642.7688 523.9231,-646.7286 565.8623,-653.2279 645.5918,-665.5836 738.8052,-677.2406 793.3848,-683.7855"/>

<polygon fill="#000000" stroke="#000000" stroke-width="7" points="792.7867,-689.8825 803.4422,-684.986 794.2388,-677.7188 792.7867,-689.8825"/>

<text text-anchor="middle" x="572.8623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">25</text>

<g id="a_edge13_63"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.06.20055327" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-579 218.0493,-601 245.0493,-601 245.0493,-579 218.0493,-579"/>

<text text-anchor="start" x="220.6587" y="-585.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_63" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-588.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-592.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-590.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_64"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6257029/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-555 218.0493,-577 245.0493,-577 245.0493,-555 218.0493,-555"/>

<text text-anchor="start" x="220.6587" y="-561.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_64" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-564.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-568.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-566.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_65"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6505341/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-531 218.0493,-553 245.0493,-553 245.0493,-531 218.0493,-531"/>

<text text-anchor="start" x="220.6587" y="-537.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_65" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-540.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-544.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-542.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_66"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc4809235/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-507 218.0493,-529 245.0493,-529 245.0493,-507 218.0493,-507"/>

<text text-anchor="start" x="220.6587" y="-513.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_66" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-516.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-520.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-518.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_67"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7087575/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-483 218.0493,-505 245.0493,-505 245.0493,-483 218.0493,-483"/>

<text text-anchor="start" x="220.6587" y="-489.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_67" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-492.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-496.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-494.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_68"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6723642/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-459 218.0493,-481 245.0493,-481 245.0493,-459 218.0493,-459"/>

<text text-anchor="start" x="220.6587" y="-465.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_68" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-468.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-472.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-470.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_69"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.05.24.113175" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-435 218.0493,-457 245.0493,-457 245.0493,-435 218.0493,-435"/>

<text text-anchor="start" x="220.6587" y="-441.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_69" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-444.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-448.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-446.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<text text-anchor="start" x="220.6587" y="-417.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="a_edge13_71"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7267182/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-387 218.0493,-409 245.0493,-409 245.0493,-387 218.0493,-387"/>

<text text-anchor="start" x="220.6587" y="-393.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_71" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-396.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-400.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-398.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_72"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7232928/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-363 218.0493,-385 245.0493,-385 245.0493,-363 218.0493,-363"/>

<text text-anchor="start" x="220.6587" y="-369.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_72" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-372.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-376.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-374.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_73"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7103766/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-339 218.0493,-361 245.0493,-361 245.0493,-339 218.0493,-339"/>

<text text-anchor="start" x="220.6587" y="-345.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_73" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-348.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-352.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-350.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_74"><a xlink:href="https://api.semanticscholar.org/v1/paper/a18236e9c75536c022502b9a9d0efdc6b0a3f0c2" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-315 218.0493,-337 245.0493,-337 245.0493,-315 218.0493,-315"/>

<text text-anchor="start" x="220.6587" y="-321.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_74" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-324.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-328.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-326.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_75"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7149753/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-291 218.0493,-313 245.0493,-313 245.0493,-291 218.0493,-291"/>

<text text-anchor="start" x="220.6587" y="-297.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_75" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-300.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-304.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-302.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_76"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129563/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-267 218.0493,-289 245.0493,-289 245.0493,-267 218.0493,-267"/>

<text text-anchor="start" x="220.6587" y="-273.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_76" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-276.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-280.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-278.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_77"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7087630/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-243 218.0493,-265 245.0493,-265 245.0493,-243 218.0493,-243"/>

<text text-anchor="start" x="220.6587" y="-249.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_77" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-252.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-256.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-254.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_78"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7087236/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-219 218.0493,-241 245.0493,-241 245.0493,-219 218.0493,-219"/>

<text text-anchor="start" x="220.6587" y="-225.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_78" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-228.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-232.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-230.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_79"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7252187/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-195 218.0493,-217 245.0493,-217 245.0493,-195 218.0493,-195"/>

<text text-anchor="start" x="220.6587" y="-201.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_79" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-204.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-208.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-206.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_80"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7198075/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-171 218.0493,-193 245.0493,-193 245.0493,-171 218.0493,-171"/>

<text text-anchor="start" x="220.6587" y="-177.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_80" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-180.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-184.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-182.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_81"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7157479/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-147 218.0493,-169 245.0493,-169 245.0493,-147 218.0493,-147"/>

<text text-anchor="start" x="220.6587" y="-153.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_81" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-156.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-160.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-158.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_82"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7148737/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-123 218.0493,-145 245.0493,-145 245.0493,-123 218.0493,-123"/>

<text text-anchor="start" x="220.6587" y="-129.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_82" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-132.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-136.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-134.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_83"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7165814/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-99 218.0493,-121 245.0493,-121 245.0493,-99 218.0493,-99"/>

<text text-anchor="start" x="220.6587" y="-105.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_83" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-108.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-112.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-110.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_84"><a xlink:href="https://api.semanticscholar.org/10.1101/2019.12.18.881391" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-75 218.0493,-97 245.0493,-97 245.0493,-75 218.0493,-75"/>

<text text-anchor="start" x="220.6587" y="-81.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_84" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-84.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-88.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-86.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_85"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6448012/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-51 218.0493,-73 245.0493,-73 245.0493,-51 218.0493,-51"/>

<text text-anchor="start" x="220.6587" y="-57.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_85" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-60.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-64.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-62.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_86"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7162632/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-27 218.0493,-49 245.0493,-49 245.0493,-27 218.0493,-27"/>

<text text-anchor="start" x="220.6587" y="-33.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_86" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-36.4676" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-40.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-38.4676" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge13_87"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6362578/pdf/" xlink:title="25">

<polygon fill="none" stroke="#000000" points="218.0493,-3 218.0493,-25 245.0493,-25 245.0493,-3 218.0493,-3"/>

<text text-anchor="start" x="220.6587" y="-9.4" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge13_87" class="icon" pointer-events="all"><circle cx="258.71070000000003" cy="-12.467600000000001" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="258.71070000000003" cy="-16.4676" r="0.75" fill="gold"/><rect x="258.21070000000003" y="-14.467600000000001" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- dromedary&#45;&gt;human -->

<g id="edge22" class="edge">

<title>dromedary-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="3.6" d="M1310.3577,-760.5394C1298.4203,-748.3562 1280.8003,-732.9561 1261.8623,-725.2279 1195.1763,-698.0148 983.4228,-691.3695 887.6825,-689.7491"/>

<polygon fill="#000000" stroke="#000000" stroke-width="3.6" points="887.4722,-686.2455 877.4182,-689.5873 887.3618,-693.2446 887.4722,-686.2455"/>

<text text-anchor="middle" x="1292.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">8</text>

<g id="a_edge22_88"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7260574/pdf/" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-735.5394 1313.8577,-757.5394 1340.8577,-757.5394 1340.8577,-735.5394 1313.8577,-735.5394"/>

<text text-anchor="start" x="1316.4671" y="-741.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_88" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-745.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-749.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-747.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_89"><a xlink:href="https://api.semanticscholar.org/v1/paper/e2aee146a962f1d07174aa3266f6fa180d20bf4a" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-711.5394 1313.8577,-733.5394 1340.8577,-733.5394 1340.8577,-711.5394 1313.8577,-711.5394"/>

<text text-anchor="start" x="1316.4671" y="-717.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_89" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-721.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-725.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-723.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_90"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.11.20054643" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-687.5394 1313.8577,-709.5394 1340.8577,-709.5394 1340.8577,-687.5394 1313.8577,-687.5394"/>

<text text-anchor="start" x="1316.4671" y="-693.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_90" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-697.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-701.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-699.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_91"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110516/pdf/" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-663.5394 1313.8577,-685.5394 1340.8577,-685.5394 1340.8577,-663.5394 1313.8577,-663.5394"/>

<text text-anchor="start" x="1316.4671" y="-669.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_91" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-673.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-677.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-675.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_92"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7193031/pdf/" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-639.5394 1313.8577,-661.5394 1340.8577,-661.5394 1340.8577,-639.5394 1313.8577,-639.5394"/>

<text text-anchor="start" x="1316.4671" y="-645.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_92" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-649.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-653.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-651.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_93"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7154514/pdf/" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-615.5394 1313.8577,-637.5394 1340.8577,-637.5394 1340.8577,-615.5394 1313.8577,-615.5394"/>

<text text-anchor="start" x="1316.4671" y="-621.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_93" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-625.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-629.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-627.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_94"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7180649/pdf/" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-591.5394 1313.8577,-613.5394 1340.8577,-613.5394 1340.8577,-591.5394 1313.8577,-591.5394"/>

<text text-anchor="start" x="1316.4671" y="-597.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_94" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-601.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-605.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-603.007" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge22_95"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129017/pdf/" xlink:title="8">

<polygon fill="none" stroke="#000000" points="1313.8577,-567.5394 1313.8577,-589.5394 1340.8577,-589.5394 1340.8577,-567.5394 1313.8577,-567.5394"/>

<text text-anchor="start" x="1316.4671" y="-573.9394" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge22_95" class="icon" pointer-events="all"><circle cx="1354.5191" cy="-577.007" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1354.5191" cy="-581.007" r="0.75" fill="gold"/><rect x="1354.0191" y="-579.007" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- horse&#45;&gt;bat -->

<g id="edge25" class="edge">

<title>horse-&gt;bat</title>

<path fill="none" stroke="#000000" d="M54.4227,-796.0311C54.5591,-807.8058 56.9839,-822.4981 66.8623,-830.8279 85.7816,-846.7812 857.7101,-862.6388 1037.6458,-866.1231"/>

<polygon fill="#000000" stroke="#000000" points="1037.618,-869.6231 1047.6836,-866.3165 1037.7529,-862.6244 1037.618,-869.6231"/>

<text text-anchor="middle" x="70.3623" y="-818.2279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge25_96"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6849485/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="24.9227,-799.0311 24.9227,-821.0311 51.9227,-821.0311 51.9227,-799.0311 24.9227,-799.0311"/>

<text text-anchor="start" x="27.5321" y="-805.4311" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge25_96" class="icon" pointer-events="all"><circle cx="65.5841" cy="-808.4987" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="65.5841" cy="-812.4987" r="0.75" fill="gold"/><rect x="65.0841" y="-810.4987" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- horse&#45;&gt;camel -->

<g id="edge26" class="edge">

<title>horse-&gt;camel</title>

<path fill="none" stroke="#000000" d="M67.8478,-760.9989C89.9671,-729.5717 137.2991,-662.3224 163.1373,-625.6116"/>

<polygon fill="#000000" stroke="#000000" points="166.0195,-627.5974 168.913,-617.4054 160.2952,-623.5685 166.0195,-627.5974"/>

<text text-anchor="middle" x="134.3623" y="-685.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge26_97"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc5709570/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="71.3478,-735.9989 71.3478,-757.9989 98.3478,-757.9989 98.3478,-735.9989 71.3478,-735.9989"/>

<text text-anchor="start" x="73.9572" y="-742.3989" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge26_97" class="icon" pointer-events="all"><circle cx="112.00919999999999" cy="-745.4665" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="112.00919999999999" cy="-749.4665" r="0.75" fill="gold"/><rect x="111.50919999999999" y="-747.4665" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- human&#45;&gt;bat -->

<g id="edge27" class="edge">

<title>human-&gt;bat</title>

<path fill="none" stroke="#000000" d="M877.1827,-690.0345C964.9696,-692.8343 1180.2151,-705.2602 1220.8623,-760.0279 1230.3978,-772.876 1229.3564,-782.4687 1220.8623,-796.0279 1217.5411,-801.3295 1148.5131,-833.3713 1106.5774,-852.4852"/>

<polygon fill="#000000" stroke="#000000" points="1104.9686,-849.3718 1097.3147,-856.6977 1107.8665,-855.7438 1104.9686,-849.3718"/>

<text text-anchor="middle" x="1230.3623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge27_98"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7077191/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="880.6827,-693.0345 880.6827,-715.0345 907.6827,-715.0345 907.6827,-693.0345 880.6827,-693.0345"/>

<text text-anchor="start" x="883.2921" y="-699.4345" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge27_98" class="icon" pointer-events="all"><circle cx="921.3441" cy="-702.5020999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="921.3441" cy="-706.5020999999999" r="0.75" fill="gold"/><rect x="920.8441" y="-704.5020999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- human&#45;&gt;civet -->

<g id="edge30" class="edge">

<title>human-&gt;civet</title>

<path fill="none" stroke="#000000" d="M866.6548,-676.505C900.3072,-660.5246 958.0156,-633.1209 994.2132,-615.9318"/>

<polygon fill="#000000" stroke="#000000" points="995.8375,-619.0351 1003.3694,-611.5839 992.8348,-612.7119 995.8375,-619.0351"/>

<text text-anchor="middle" x="954.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge30_99"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7121850/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="861.9048,-651.505 861.9048,-673.505 888.9048,-673.505 888.9048,-651.505 861.9048,-651.505"/>

<text text-anchor="start" x="864.5142" y="-657.905" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge30_99" class="icon" pointer-events="all"><circle cx="902.5662" cy="-660.9725999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="902.5662" cy="-664.9725999999999" r="0.75" fill="gold"/><rect x="902.0662" y="-662.9725999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- human&#45;&gt;camel -->

<g id="edge28" class="edge">

<title>human-&gt;camel</title>

<path fill="none" stroke="#000000" d="M802.6422,-686.7676C734.9829,-682.0501 587.4014,-670.6313 463.8623,-653.2279 377.6149,-641.0779 277.6482,-621.0022 222.6078,-609.4217"/>

<polygon fill="#000000" stroke="#000000" points="223.0493,-605.9377 212.5415,-607.2929 221.6009,-612.7862 223.0493,-605.9377"/>

<text text-anchor="middle" x="467.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge28_100"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7126747/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="773.1422,-689.7676 773.1422,-711.7676 800.1422,-711.7676 800.1422,-689.7676 773.1422,-689.7676"/>

<text text-anchor="start" x="775.7516" y="-696.1676" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge28_100" class="icon" pointer-events="all"><circle cx="813.8036000000001" cy="-699.2352" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="813.8036000000001" cy="-703.2352" r="0.75" fill="gold"/><rect x="813.3036000000001" y="-701.2352" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- pig -->

<g id="node11" class="node">

<title>pig</title>

<ellipse fill="#add8e6" stroke="#000000" cx="459.8623" cy="-600.4279" rx="27" ry="18"/>

<text text-anchor="middle" x="459.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">pig</text>

</g>

<!-- human&#45;&gt;pig -->

<g id="edge32" class="edge">

<title>human-&gt;pig</title>

<path fill="none" stroke="#000000" stroke-width="2.4" d="M807.9617,-679.5507C770.203,-668.2967 705.3192,-649.5851 648.8623,-636.4279 596.2312,-624.1622 534.6362,-613.0213 496.4288,-606.4826"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.4" points="496.7442,-602.9861 486.2994,-604.7632 495.5727,-609.8874 496.7442,-602.9861"/>

<text text-anchor="middle" x="716.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">2</text>

<g id="a_edge32_101"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7118731/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="778.4617,-654.5507 778.4617,-676.5507 805.4617,-676.5507 805.4617,-654.5507 778.4617,-654.5507"/>

<text text-anchor="start" x="781.0711" y="-660.9507" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge32_101" class="icon" pointer-events="all"><circle cx="819.1231" cy="-664.0183" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="819.1231" cy="-668.0183" r="0.75" fill="gold"/><rect x="818.6231" y="-666.0183" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge32_102"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7121850/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="778.4617,-630.5507 778.4617,-652.5507 805.4617,-652.5507 805.4617,-630.5507 778.4617,-630.5507"/>

<text text-anchor="start" x="781.0711" y="-636.9507" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge32_102" class="icon" pointer-events="all"><circle cx="819.1231" cy="-640.0183" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="819.1231" cy="-644.0183" r="0.75" fill="gold"/><rect x="818.6231" y="-642.0183" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- chimpanzee -->

<g id="node22" class="node">

<title>chimpanzee</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1131.8623" cy="-600.4279" rx="57.303" ry="18"/>

<text text-anchor="middle" x="1131.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">chimpanzee</text>

</g>

<!-- human&#45;&gt;chimpanzee -->

<g id="edge29" class="edge">

<title>human-&gt;chimpanzee</title>

<path fill="none" stroke="#000000" d="M875.9589,-684.464C912.3778,-679.0713 969.9247,-668.9358 1017.8623,-653.2279 1044.1597,-644.6109 1072.3921,-631.5662 1094.1821,-620.5991"/>

<polygon fill="#000000" stroke="#000000" points="1095.787,-623.7095 1103.1019,-616.0451 1092.604,-617.4751 1095.787,-623.7095"/>

<text text-anchor="middle" x="1063.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge29_103"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7138062/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="879.4589,-659.464 879.4589,-681.464 906.4589,-681.464 906.4589,-659.464 879.4589,-659.464"/>

<text text-anchor="start" x="882.0683" y="-665.864" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge29_103" class="icon" pointer-events="all"><circle cx="920.1203" cy="-668.9316" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="920.1203" cy="-672.9316" r="0.75" fill="gold"/><rect x="919.6203" y="-670.9316" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- mosquito -->

<g id="node23" class="node">

<title>mosquito</title>

<ellipse fill="#add8e6" stroke="#000000" cx="931.8623" cy="-600.4279" rx="46.9709" ry="18"/>

<text text-anchor="middle" x="931.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">mosquito</text>

</g>

<!-- human&#45;&gt;mosquito -->

<g id="edge31" class="edge">

<title>human-&gt;mosquito</title>

<path fill="none" stroke="#000000" d="M856.7365,-672.9406C870.8236,-659.3435 891.1016,-639.7708 907.0402,-624.3866"/>

<polygon fill="#000000" stroke="#000000" points="909.7055,-626.6784 914.4699,-617.2153 904.8441,-621.6419 909.7055,-626.6784"/>

<text text-anchor="middle" x="898.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge31_104"><a xlink:href="https://api.semanticscholar.org/10.1101/2020.04.11.20061143" xlink:title="1">

<polygon fill="none" stroke="#000000" points="827.2365,-647.9406 827.2365,-669.9406 854.2365,-669.9406 854.2365,-647.9406 827.2365,-647.9406"/>

<text text-anchor="start" x="829.8459" y="-654.3406" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge31_104" class="icon" pointer-events="all"><circle cx="867.8979" cy="-657.4082" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="867.8979" cy="-661.4082" r="0.75" fill="gold"/><rect x="867.3979" y="-659.4082" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- swine -->

<g id="node24" class="node">

<title>swine</title>

<ellipse fill="#add8e6" stroke="#000000" cx="751.8623" cy="-600.4279" rx="33.6247" ry="18"/>

<text text-anchor="middle" x="751.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">swine</text>

</g>

<!-- human&#45;&gt;swine -->

<g id="edge33" class="edge">

<title>human-&gt;swine</title>

<path fill="none" stroke="#000000" d="M822.1184,-673.3818C815.3563,-667.2053 807.6476,-659.9951 800.8623,-653.2279 791.6852,-644.0752 781.9076,-633.6795 773.5198,-624.5472"/>

<polygon fill="#000000" stroke="#000000" points="775.8805,-621.9416 766.5566,-616.9102 770.7079,-626.658 775.8805,-621.9416"/>

<text text-anchor="middle" x="804.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge33_105"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7126747/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="792.6184,-648.3818 792.6184,-670.3818 819.6184,-670.3818 819.6184,-648.3818 792.6184,-648.3818"/>

<text text-anchor="start" x="795.2278" y="-654.7818" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge33_105" class="icon" pointer-events="all"><circle cx="833.2798" cy="-657.8494" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="833.2798" cy="-661.8494" r="0.75" fill="gold"/><rect x="832.7798" y="-659.8494" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- pangolin&#45;&gt;human -->

<g id="edge39" class="edge">

<title>pangolin-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="2.6" d="M1059.1033,-760.9557C1047.2372,-749.1402 1029.964,-734.0152 1011.8623,-725.2279 972.3947,-706.0687 922.9935,-697.0746 887.2729,-692.871"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.6" points="887.4384,-689.3684 877.1173,-691.7611 886.6778,-696.3269 887.4384,-689.3684"/>

<text text-anchor="middle" x="1040.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">3</text>

<g id="a_edge39_106"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7217226/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="1062.6033,-735.9557 1062.6033,-757.9557 1089.6033,-757.9557 1089.6033,-735.9557 1062.6033,-735.9557"/>

<text text-anchor="start" x="1065.2127" y="-742.3557" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge39_106" class="icon" pointer-events="all"><circle cx="1103.2647" cy="-745.4232999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1103.2647" cy="-749.4232999999999" r="0.75" fill="gold"/><rect x="1102.7647" y="-747.4232999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge39_107"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7202812/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="1062.6033,-711.9557 1062.6033,-733.9557 1089.6033,-733.9557 1089.6033,-711.9557 1062.6033,-711.9557"/>

<text text-anchor="start" x="1065.2127" y="-718.3557" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge39_107" class="icon" pointer-events="all"><circle cx="1103.2647" cy="-721.4232999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1103.2647" cy="-725.4232999999999" r="0.75" fill="gold"/><rect x="1102.7647" y="-723.4232999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge39_108"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7193031/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="1062.6033,-687.9557 1062.6033,-709.9557 1089.6033,-709.9557 1089.6033,-687.9557 1062.6033,-687.9557"/>

<text text-anchor="start" x="1065.2127" y="-694.3557" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge39_108" class="icon" pointer-events="all"><circle cx="1103.2647" cy="-697.4232999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1103.2647" cy="-701.4232999999999" r="0.75" fill="gold"/><rect x="1102.7647" y="-699.4232999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- racoon&#45;&gt;human -->

<g id="edge41" class="edge">

<title>racoon-&gt;human</title>

<path fill="none" stroke="#000000" d="M1148.725,-765.1379C1123.4731,-753.1747 1083.8631,-735.6851 1047.8623,-725.2279 993.5662,-709.4564 929.1698,-699.6195 886.5669,-694.2923"/>

<polygon fill="#000000" stroke="#000000" points="886.8408,-690.7998 876.4911,-693.0657 885.9949,-697.7486 886.8408,-690.7998"/>

<text text-anchor="middle" x="1098.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge41_109"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc3119930/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1127.475,-740.1379 1127.475,-762.1379 1154.475,-762.1379 1154.475,-740.1379 1127.475,-740.1379"/>

<text text-anchor="start" x="1130.0844" y="-746.5379" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge41_109" class="icon" pointer-events="all"><circle cx="1168.1363999999999" cy="-749.6055" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1168.1363999999999" cy="-753.6055" r="0.75" fill="gold"/><rect x="1167.6363999999999" y="-751.6055" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- bird -->

<g id="node10" class="node">

<title>bird</title>

<ellipse fill="#add8e6" stroke="#000000" cx="244.8623" cy="-778.0279" rx="27" ry="18"/>

<text text-anchor="middle" x="244.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">bird</text>

</g>

<!-- bird&#45;&gt;human -->

<g id="edge10" class="edge">

<title>bird-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="3" d="M265.8894,-766.7071C270.7317,-764.3278 275.9146,-761.9574 280.8623,-760.0279 332.9957,-739.6964 346.8189,-735.3025 401.8623,-725.2279 542.1035,-699.5595 710.9752,-692.1827 791.9624,-690.0703"/>

<polygon fill="#000000" stroke="#000000" stroke-width="3" points="792.4176,-693.5603 802.3289,-689.8165 792.2462,-686.5624 792.4176,-693.5603"/>

<text text-anchor="middle" x="405.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">5</text>

<g id="a_edge10_110"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7152116/pdf/" xlink:title="5">

<polygon fill="none" stroke="#000000" points="269.3894,-741.7071 269.3894,-763.7071 296.3894,-763.7071 296.3894,-741.7071 269.3894,-741.7071"/>

<text text-anchor="start" x="271.9988" y="-748.1071" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge10_110" class="icon" pointer-events="all"><circle cx="310.05080000000004" cy="-751.1746999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="310.05080000000004" cy="-755.1746999999999" r="0.75" fill="gold"/><rect x="309.55080000000004" y="-753.1746999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge10_111"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7150074/pdf/" xlink:title="5">

<polygon fill="none" stroke="#000000" points="269.3894,-717.7071 269.3894,-739.7071 296.3894,-739.7071 296.3894,-717.7071 269.3894,-717.7071"/>

<text text-anchor="start" x="271.9988" y="-724.1071" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge10_111" class="icon" pointer-events="all"><circle cx="310.05080000000004" cy="-727.1746999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="310.05080000000004" cy="-731.1746999999999" r="0.75" fill="gold"/><rect x="309.55080000000004" y="-729.1746999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge10_112"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc4124145/pdf/" xlink:title="5">

<polygon fill="none" stroke="#000000" points="269.3894,-693.7071 269.3894,-715.7071 296.3894,-715.7071 296.3894,-693.7071 269.3894,-693.7071"/>

<text text-anchor="start" x="271.9988" y="-700.1071" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge10_112" class="icon" pointer-events="all"><circle cx="310.05080000000004" cy="-703.1746999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="310.05080000000004" cy="-707.1746999999999" r="0.75" fill="gold"/><rect x="309.55080000000004" y="-705.1746999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge10_113"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7152125/pdf/" xlink:title="5">

<polygon fill="none" stroke="#000000" points="269.3894,-669.7071 269.3894,-691.7071 296.3894,-691.7071 296.3894,-669.7071 269.3894,-669.7071"/>

<text text-anchor="start" x="271.9988" y="-676.1071" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge10_113" class="icon" pointer-events="all"><circle cx="310.05080000000004" cy="-679.1746999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="310.05080000000004" cy="-683.1746999999999" r="0.75" fill="gold"/><rect x="309.55080000000004" y="-681.1746999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge10_114"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="5">

<polygon fill="none" stroke="#000000" points="269.3894,-645.7071 269.3894,-667.7071 296.3894,-667.7071 296.3894,-645.7071 269.3894,-645.7071"/>

<text text-anchor="start" x="271.9988" y="-652.1071" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge10_114" class="icon" pointer-events="all"><circle cx="310.05080000000004" cy="-655.1746999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="310.05080000000004" cy="-659.1746999999999" r="0.75" fill="gold"/><rect x="309.55080000000004" y="-657.1746999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- bird&#45;&gt;pig -->

<g id="edge11" class="edge">

<title>bird-&gt;pig</title>

<path fill="none" stroke="#000000" stroke-width="2.6" d="M254.2084,-761.1061C271.2722,-731.6906 310.235,-670.9733 358.8623,-636.4279 378.3357,-622.5937 403.6635,-613.5833 424.0839,-608.0366"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.6" points="425.1381,-611.3798 433.9618,-605.5153 423.4069,-604.5972 425.1381,-611.3798"/>

<text text-anchor="middle" x="322.3623" y="-685.0279" font-family="Times,serif" font-size="14.00" fill="#000000">3</text>

<g id="a_edge11_115"><a xlink:href="https://api.semanticscholar.org/10.1093/molbev/msaa117" xlink:title="3">

<polygon fill="none" stroke="#000000" points="232.9584,-736.1061 232.9584,-758.1061 259.9584,-758.1061 259.9584,-736.1061 232.9584,-736.1061"/>

<text text-anchor="start" x="235.5678" y="-742.5061" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge11_115" class="icon" pointer-events="all"><circle cx="273.6198" cy="-745.5736999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="273.6198" cy="-749.5736999999999" r="0.75" fill="gold"/><rect x="273.1198" y="-747.5736999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge11_116"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6803248/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="232.9584,-712.1061 232.9584,-734.1061 259.9584,-734.1061 259.9584,-712.1061 232.9584,-712.1061"/>

<text text-anchor="start" x="235.5678" y="-718.5061" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge11_116" class="icon" pointer-events="all"><circle cx="273.6198" cy="-721.5736999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="273.6198" cy="-725.5736999999999" r="0.75" fill="gold"/><rect x="273.1198" y="-723.5736999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge11_117"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7114555/pdf/" xlink:title="3">

<polygon fill="none" stroke="#000000" points="232.9584,-688.1061 232.9584,-710.1061 259.9584,-710.1061 259.9584,-688.1061 232.9584,-688.1061"/>

<text text-anchor="start" x="235.5678" y="-694.5061" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge11_117" class="icon" pointer-events="all"><circle cx="273.6198" cy="-697.5736999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="273.6198" cy="-701.5736999999999" r="0.75" fill="gold"/><rect x="273.1198" y="-699.5736999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- pig&#45;&gt;human -->

<g id="edge40" class="edge">

<title>pig-&gt;human</title>

<path fill="none" stroke="#000000" d="M484.8401,-607.7895C532.6594,-621.8725 633.1161,-651.4027 640.8623,-653.2279 692.8905,-665.487 753.4183,-675.8974 793.9344,-682.3222"/>

<polygon fill="#000000" stroke="#000000" points="793.6196,-685.8156 804.0413,-683.908 794.7047,-678.9002 793.6196,-685.8156"/>

<text text-anchor="middle" x="644.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge40_118"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6880172/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="488.3401,-610.7895 488.3401,-632.7895 515.3401,-632.7895 515.3401,-610.7895 488.3401,-610.7895"/>

<text text-anchor="start" x="490.9495" y="-617.1895" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge40_118" class="icon" pointer-events="all"><circle cx="529.0015" cy="-620.2570999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="529.0015" cy="-624.2570999999999" r="0.75" fill="gold"/><rect x="528.5015" y="-622.2570999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- bovine -->

<g id="node12" class="node">

<title>bovine</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1160.8623" cy="-955.6279" rx="37.1424" ry="18"/>

<text text-anchor="middle" x="1160.8623" y="-951.4279" font-family="Times,serif" font-size="14.00" fill="#000000">bovine</text>

</g>

<!-- canine -->

<g id="node13" class="node">

<title>canine</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1160.8623" cy="-866.8279" rx="36.4761" ry="18"/>

<text text-anchor="middle" x="1160.8623" y="-862.6279" font-family="Times,serif" font-size="14.00" fill="#000000">canine</text>

</g>

<!-- bovine&#45;&gt;canine -->

<g id="edge12" class="edge">

<title>bovine-&gt;canine</title>

<path fill="none" stroke="#000000" d="M1160.8623,-937.2285C1160.8623,-925.1228 1160.8623,-909.0355 1160.8623,-895.2953"/>

<polygon fill="#000000" stroke="#000000" points="1164.3624,-894.8999 1160.8623,-884.8999 1157.3624,-894.8999 1164.3624,-894.8999"/>

<text text-anchor="middle" x="1164.3623" y="-907.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge12_119"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7089282/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1131.3623,-912.2285 1131.3623,-934.2285 1158.3623,-934.2285 1158.3623,-912.2285 1131.3623,-912.2285"/>

<text text-anchor="start" x="1133.9717" y="-918.6285" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge12_119" class="icon" pointer-events="all"><circle cx="1172.0237" cy="-921.6961" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1172.0237" cy="-925.6961" r="0.75" fill="gold"/><rect x="1171.5237" y="-923.6961" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- cat -->

<g id="node14" class="node">

<title>cat</title>

<ellipse fill="#add8e6" stroke="#000000" cx="916.8623" cy="-511.6279" rx="27" ry="18"/>

<text text-anchor="middle" x="916.8623" y="-507.4279" font-family="Times,serif" font-size="14.00" fill="#000000">cat</text>

</g>

<!-- cat&#45;&gt;human -->

<g id="edge14" class="edge">

<title>cat-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="2.4" d="M903.7698,-527.5792C893.0395,-541.3567 878.1576,-562.1847 868.8623,-582.4279 857.1493,-607.9363 849.3604,-639.1125 844.8315,-661.1801"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.4" points="841.3531,-660.7307 842.8672,-671.2168 848.2228,-662.0753 841.3531,-660.7307"/>

<text text-anchor="middle" x="872.3623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">2</text>

<g id="a_edge14_120"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7198075/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="874.2698,-554.5792 874.2698,-576.5792 901.2698,-576.5792 901.2698,-554.5792 874.2698,-554.5792"/>

<text text-anchor="start" x="876.8792" y="-560.9792" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge14_120" class="icon" pointer-events="all"><circle cx="914.9312" cy="-564.0468" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="914.9312" cy="-568.0468" r="0.75" fill="gold"/><rect x="914.4312" y="-566.0468" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge14_121"><a xlink:href="https://api.semanticscholar.org/v1/paper/e2aee146a962f1d07174aa3266f6fa180d20bf4a" xlink:title="2">

<polygon fill="none" stroke="#000000" points="874.2698,-530.5792 874.2698,-552.5792 901.2698,-552.5792 901.2698,-530.5792 874.2698,-530.5792"/>

<text text-anchor="start" x="876.8792" y="-536.9792" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge14_121" class="icon" pointer-events="all"><circle cx="914.9312" cy="-540.0468" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="914.9312" cy="-544.0468" r="0.75" fill="gold"/><rect x="914.4312" y="-542.0468" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- cattle -->

<g id="node15" class="node">

<title>cattle</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1374.8623" cy="-600.4279" rx="31.8954" ry="18"/>

<text text-anchor="middle" x="1374.8623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">cattle</text>

</g>

<!-- dog -->

<g id="node16" class="node">

<title>dog</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1106.8623" cy="-511.6279" rx="27" ry="18"/>

<text text-anchor="middle" x="1106.8623" y="-507.4279" font-family="Times,serif" font-size="14.00" fill="#000000">dog</text>

</g>

<!-- cattle&#45;&gt;dog -->

<g id="edge15" class="edge">

<title>cattle-&gt;dog</title>

<path fill="none" stroke="#000000" d="M1365.6819,-583.052C1358.2762,-570.9248 1346.6651,-555.5453 1331.8623,-547.6279 1257.4565,-507.8311 1223.9725,-552.89 1142.8623,-529.6279 1140.7339,-529.0175 1138.5808,-528.2824 1136.4417,-527.4628"/>

<polygon fill="#000000" stroke="#000000" points="1137.8385,-524.2536 1127.2739,-523.4538 1135.0338,-530.6672 1137.8385,-524.2536"/>

<text text-anchor="middle" x="1354.3623" y="-551.8279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge15_122"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7117484/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1369.1819,-558.052 1369.1819,-580.052 1396.1819,-580.052 1396.1819,-558.052 1369.1819,-558.052"/>

<text text-anchor="start" x="1371.7913" y="-564.452" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge15_122" class="icon" pointer-events="all"><circle cx="1409.8433" cy="-567.5196" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1409.8433" cy="-571.5196" r="0.75" fill="gold"/><rect x="1409.3433" y="-569.5196" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- dog&#45;&gt;human -->

<g id="edge21" class="edge">

<title>dog-&gt;human</title>

<path fill="none" stroke="#000000" d="M1128.5926,-522.5239C1163.5414,-541.4866 1225.6384,-582.0168 1197.8623,-618.4279 1160.7604,-667.064 975.9334,-682.6108 887.354,-687.3143"/>

<polygon fill="#000000" stroke="#000000" points="887.1103,-683.822 877.3001,-687.8233 887.4644,-690.8131 887.1103,-683.822"/>

<text text-anchor="middle" x="1207.3623" y="-596.2279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge21_123"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7152112/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1132.0926,-525.5239 1132.0926,-547.5239 1159.0926,-547.5239 1159.0926,-525.5239 1132.0926,-525.5239"/>

<text text-anchor="start" x="1134.702" y="-531.9239" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge21_123" class="icon" pointer-events="all"><circle cx="1172.754" cy="-534.9915" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1172.754" cy="-538.9915" r="0.75" fill="gold"/><rect x="1172.254" y="-536.9915" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- flea -->

<g id="node20" class="node">

<title>flea</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1142.8623" cy="-422.8279" rx="27" ry="18"/>

<text text-anchor="middle" x="1142.8623" y="-418.6279" font-family="Times,serif" font-size="14.00" fill="#000000">flea</text>

</g>

<!-- dog&#45;&gt;flea -->

<g id="edge20" class="edge">

<title>dog-&gt;flea</title>

<path fill="none" stroke="#000000" d="M1113.9747,-494.0839C1119.1026,-481.435 1126.1189,-464.1281 1131.9526,-449.7384"/>

<polygon fill="#000000" stroke="#000000" points="1135.2511,-450.9179 1135.7646,-440.3355 1128.7639,-448.2879 1135.2511,-450.9179"/>

<text text-anchor="middle" x="1132.3623" y="-463.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge20_124"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6797873/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1084.4747,-469.0839 1084.4747,-491.0839 1111.4747,-491.0839 1111.4747,-469.0839 1084.4747,-469.0839"/>

<text text-anchor="start" x="1087.0841" y="-475.4839" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge20_124" class="icon" pointer-events="all"><circle cx="1125.1361" cy="-478.55150000000003" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1125.1361" cy="-482.55150000000003" r="0.75" fill="gold"/><rect x="1124.6361" y="-480.55150000000003" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- cow -->

<g id="node17" class="node">

<title>cow</title>

<ellipse fill="#add8e6" stroke="#000000" cx="33.8623" cy="-866.8279" rx="27.2467" ry="18"/>

<text text-anchor="middle" x="33.8623" y="-862.6279" font-family="Times,serif" font-size="14.00" fill="#000000">cow</text>

</g>

<!-- cow&#45;&gt;horse -->

<g id="edge17" class="edge">

<title>cow-&gt;horse</title>

<path fill="none" stroke="#000000" d="M33,-848.4318C32.9881,-838.1386 33.7633,-825.1737 36.8623,-814.0279 37.7994,-810.6575 39.0844,-807.2446 40.5479,-803.9316"/>

<polygon fill="#000000" stroke="#000000" points="43.682,-805.4895 45.0037,-794.9774 37.4151,-802.3708 43.682,-805.4895"/>

<text text-anchor="middle" x="40.3623" y="-818.2279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge17_125"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc5709570/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="3.5,-823.4318 3.5,-845.4318 30.5,-845.4318 30.5,-823.4318 3.5,-823.4318"/>

<text text-anchor="start" x="6.1094" y="-829.8318" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge17_125" class="icon" pointer-events="all"><circle cx="44.1614" cy="-832.8994" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="44.1614" cy="-836.8994" r="0.75" fill="gold"/><rect x="43.6614" y="-834.8994" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- deer -->

<g id="node18" class="node">

<title>deer</title>

<ellipse fill="#add8e6" stroke="#000000" cx="171.8623" cy="-778.0279" rx="27.8073" ry="18"/>

<text text-anchor="middle" x="171.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">deer</text>

</g>

<!-- deer&#45;&gt;camel -->

<g id="edge18" class="edge">

<title>deer-&gt;camel</title>

<path fill="none" stroke="#000000" d="M171.1646,-759.8647C170.5093,-738.5957 169.8943,-702.3157 171.8623,-671.2279 172.7552,-657.1231 174.6391,-641.532 176.4536,-628.5792"/>

<polygon fill="#000000" stroke="#000000" points="179.9569,-628.8106 177.9383,-618.4099 173.0303,-627.7993 179.9569,-628.8106"/>

<text text-anchor="middle" x="175.3623" y="-685.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge18_126"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7126047/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="166.4146,-734.8647 166.4146,-756.8647 193.4146,-756.8647 193.4146,-734.8647 166.4146,-734.8647"/>

<text text-anchor="start" x="169.024" y="-741.2647" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge18_126" class="icon" pointer-events="all"><circle cx="207.076" cy="-744.3322999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="207.076" cy="-748.3322999999999" r="0.75" fill="gold"/><rect x="206.576" y="-746.3322999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- snail -->

<g id="node19" class="node">

<title>snail</title>

<ellipse fill="#add8e6" stroke="#000000" cx="216.8623" cy="-689.2279" rx="29.0509" ry="18"/>

<text text-anchor="middle" x="216.8623" y="-685.0279" font-family="Times,serif" font-size="14.00" fill="#000000">snail</text>

</g>

<!-- deer&#45;&gt;snail -->

<g id="edge19" class="edge">

<title>deer-&gt;snail</title>

<path fill="none" stroke="#000000" d="M180.5387,-760.9064C187.0335,-748.0901 196.0371,-730.3228 203.4554,-715.6842"/>

<polygon fill="#000000" stroke="#000000" points="206.6606,-717.102 208.0589,-706.5999 200.4166,-713.9378 206.6606,-717.102"/>

<text text-anchor="middle" x="202.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge19_127"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7126047/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="184.0387,-763.9064 184.0387,-785.9064 211.0387,-785.9064 211.0387,-763.9064 184.0387,-763.9064"/>

<text text-anchor="start" x="186.6481" y="-770.3064" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge19_127" class="icon" pointer-events="all"><circle cx="224.7001" cy="-773.374" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="224.7001" cy="-777.374" r="0.75" fill="gold"/><rect x="224.2001" y="-775.374" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- snail&#45;&gt;camel -->

<g id="edge45" class="edge">

<title>snail-&gt;camel</title>

<path fill="none" stroke="#000000" d="M209.7498,-671.6839C204.6552,-659.1171 197.6967,-641.9527 191.8857,-627.6191"/>

<polygon fill="#000000" stroke="#000000" points="195.0839,-626.1918 188.0832,-618.2394 188.5967,-628.8218 195.0839,-626.1918"/>

<text text-anchor="middle" x="206.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge45_128"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7126047/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="213.2498,-646.6839 213.2498,-668.6839 240.2498,-668.6839 240.2498,-646.6839 213.2498,-646.6839"/>

<text text-anchor="start" x="215.8592" y="-653.0839" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge45_128" class="icon" pointer-events="all"><circle cx="253.91119999999998" cy="-656.1514999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="253.91119999999998" cy="-660.1514999999999" r="0.75" fill="gold"/><rect x="253.41119999999998" y="-658.1514999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- snail&#45;&gt;pig -->

<g id="edge46" class="edge">

<title>snail-&gt;pig</title>

<path fill="none" stroke="#000000" d="M226.5575,-672.216C234.2852,-660.2697 246.2217,-644.9572 260.8623,-636.4279 287.8391,-620.7117 373.3789,-609.4572 423.0912,-604.0504"/>

<polygon fill="#000000" stroke="#000000" points="423.5467,-607.5218 433.1215,-602.986 422.8079,-600.5609 423.5467,-607.5218"/>

<text text-anchor="middle" x="264.3623" y="-640.6279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge46_129"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7126047/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="230.0575,-647.216 230.0575,-669.216 257.0575,-669.216 257.0575,-647.216 230.0575,-647.216"/>

<text text-anchor="start" x="232.6669" y="-653.616" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge46_129" class="icon" pointer-events="all"><circle cx="270.7189" cy="-656.6836" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="270.7189" cy="-660.6836" r="0.75" fill="gold"/><rect x="270.2189" y="-658.6836" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- flea&#45;&gt;human -->

<g id="edge24" class="edge">

<title>flea-&gt;human</title>

<path fill="none" stroke="#000000" d="M1116.2805,-426.4601C1056.347,-435.1879 913.3731,-459.1986 880.8623,-493.6279 858.3286,-517.4914 846.7083,-612.9129 842.1778,-661.0921"/>

<polygon fill="#000000" stroke="#000000" points="838.6822,-660.8876 841.2699,-671.1615 845.6539,-661.5162 838.6822,-660.8876"/>

<text text-anchor="middle" x="863.3623" y="-551.8279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge24_130"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6797873/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1086.7805,-429.4601 1086.7805,-451.4601 1113.7805,-451.4601 1113.7805,-429.4601 1086.7805,-429.4601"/>

<text text-anchor="start" x="1089.3899" y="-435.8601" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge24_130" class="icon" pointer-events="all"><circle cx="1127.4418999999998" cy="-438.9277" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1127.4418999999998" cy="-442.9277" r="0.75" fill="gold"/><rect x="1126.9418999999998" y="-440.9277" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- fix -->

<g id="node21" class="node">

<title>fix</title>

<ellipse fill="#add8e6" stroke="#000000" cx="316.8623" cy="-778.0279" rx="27" ry="18"/>

<text text-anchor="middle" x="316.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">fix</text>

</g>

<!-- fix&#45;&gt;human -->

<g id="edge23" class="edge">

<title>fix-&gt;human</title>

<path fill="none" stroke="#000000" d="M337.8008,-766.4687C342.6477,-764.1017 347.8548,-761.7967 352.8623,-760.0279 421.0713,-735.9337 440.4746,-736.9238 511.8623,-725.2279 611.2003,-708.9526 728.702,-698.0589 792.6301,-692.8269"/>

<polygon fill="#000000" stroke="#000000" points="793.1106,-696.2996 802.7963,-692.0058 792.5469,-689.3223 793.1106,-696.2996"/>

<text text-anchor="middle" x="515.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge23_131"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="333.0508,-741.4687 333.0508,-763.4687 360.0508,-763.4687 360.0508,-741.4687 333.0508,-741.4687"/>

<text text-anchor="start" x="335.6602" y="-747.8687" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge23_131" class="icon" pointer-events="all"><circle cx="373.7122" cy="-750.9363" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="373.7122" cy="-754.9363" r="0.75" fill="gold"/><rect x="373.2122" y="-752.9363" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- mosquito&#45;&gt;cat -->

<g id="edge35" class="edge">

<title>mosquito-&gt;cat</title>

<path fill="none" stroke="#000000" d="M928.8268,-582.4579C926.753,-570.1809 923.9632,-553.6651 921.5991,-539.67"/>

<polygon fill="#000000" stroke="#000000" points="925.01,-538.8485 919.8932,-529.5712 918.1078,-540.0145 925.01,-538.8485"/>

<text text-anchor="middle" x="929.3623" y="-551.8279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge35_132"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7158197/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="932.3268,-557.4579 932.3268,-579.4579 959.3268,-579.4579 959.3268,-557.4579 932.3268,-557.4579"/>

<text text-anchor="start" x="934.9362" y="-563.8579" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge35_132" class="icon" pointer-events="all"><circle cx="972.9882" cy="-566.9254999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="972.9882" cy="-570.9254999999999" r="0.75" fill="gold"/><rect x="972.4882" y="-568.9254999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- mosquito&#45;&gt;dog -->

<g id="edge36" class="edge">

<title>mosquito-&gt;dog</title>

<path fill="none" stroke="#000000" d="M960.3685,-585.963C992.1909,-569.8154 1043.4994,-543.78 1076.1792,-527.1974"/>

<polygon fill="#000000" stroke="#000000" points="1077.9418,-530.2278 1085.2757,-522.5815 1074.7743,-523.9855 1077.9418,-530.2278"/>

<text text-anchor="middle" x="1039.3623" y="-551.8279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge36_133"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7158197/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="963.8685,-560.963 963.8685,-582.963 990.8685,-582.963 990.8685,-560.963 963.8685,-560.963"/>

<text text-anchor="start" x="966.4779" y="-567.363" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge36_133" class="icon" pointer-events="all"><circle cx="1004.5299" cy="-570.4306" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1004.5299" cy="-574.4306" r="0.75" fill="gold"/><rect x="1004.0299" y="-572.4306" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- midges -->

<g id="node25" class="node">

<title>midges</title>

<ellipse fill="#add8e6" stroke="#000000" cx="400.8623" cy="-778.0279" rx="38.8632" ry="18"/>

<text text-anchor="middle" x="400.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">midges</text>

</g>

<!-- midges&#45;&gt;human -->

<g id="edge34" class="edge">

<title>midges-&gt;human</title>

<path fill="none" stroke="#000000" d="M430.2925,-766.0538C436.368,-763.8481 442.7721,-761.7131 448.8623,-760.0279 571.0092,-726.2279 719.2077,-704.478 793.1608,-694.8901"/>

<polygon fill="#000000" stroke="#000000" points="794.0466,-698.3053 803.5212,-693.5637 793.1577,-691.362 794.0466,-698.3053"/>

<text text-anchor="middle" x="601.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge34_134"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129920/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="425.5425,-741.0538 425.5425,-763.0538 452.5425,-763.0538 452.5425,-741.0538 425.5425,-741.0538"/>

<text text-anchor="start" x="428.1519" y="-747.4538" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge34_134" class="icon" pointer-events="all"><circle cx="466.20390000000003" cy="-750.5214" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="466.20390000000003" cy="-754.5214" r="0.75" fill="gold"/><rect x="465.70390000000003" y="-752.5214" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- mosquitoes -->

<g id="node26" class="node">

<title>mosquitoes</title>

<ellipse fill="#add8e6" stroke="#000000" cx="512.8623" cy="-778.0279" rx="55.0555" ry="18"/>

<text text-anchor="middle" x="512.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">mosquitoes</text>

</g>

<!-- mosquitoes&#45;&gt;human -->

<g id="edge37" class="edge">

<title>mosquitoes-&gt;human</title>

<path fill="none" stroke="#000000" d="M554.4999,-766.2444C561.9426,-764.1589 569.6288,-762.0187 576.8623,-760.0279 654.4368,-738.6781 745.0787,-714.4308 797.1453,-700.57"/>

<polygon fill="#000000" stroke="#000000" points="798.2652,-703.8938 807.029,-697.9402 796.4653,-697.1292 798.2652,-703.8938"/>

<text text-anchor="middle" x="703.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge37_135"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="549.7499,-741.2444 549.7499,-763.2444 576.7499,-763.2444 576.7499,-741.2444 549.7499,-741.2444"/>

<text text-anchor="start" x="552.3593" y="-747.6444" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge37_135" class="icon" pointer-events="all"><circle cx="590.4113" cy="-750.712" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="590.4113" cy="-754.712" r="0.75" fill="gold"/><rect x="589.9113" y="-752.712" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- mosquitos -->

<g id="node27" class="node">

<title>mosquitos</title>

<ellipse fill="#add8e6" stroke="#000000" cx="636.8623" cy="-778.0279" rx="50.9832" ry="18"/>

<text text-anchor="middle" x="636.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">mosquitos</text>

</g>

<!-- mosquitos&#45;&gt;human -->

<g id="edge38" class="edge">

<title>mosquitos-&gt;human</title>

<path fill="none" stroke="#000000" d="M669.0282,-763.9573C705.4097,-748.0426 764.5563,-722.1696 802.8324,-705.4262"/>

<polygon fill="#000000" stroke="#000000" points="804.3283,-708.5921 812.0873,-701.3777 801.5228,-702.1789 804.3283,-708.5921"/>

<text text-anchor="middle" x="760.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge38_136"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129920/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="664.2782,-738.9573 664.2782,-760.9573 691.2782,-760.9573 691.2782,-738.9573 664.2782,-738.9573"/>

<text text-anchor="start" x="666.8876" y="-745.3573" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge38_136" class="icon" pointer-events="all"><circle cx="704.9396" cy="-748.4249" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="704.9396" cy="-752.4249" r="0.75" fill="gold"/><rect x="704.4396" y="-750.4249" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- rat -->

<g id="node28" class="node">

<title>rat</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1178.8623" cy="-511.6279" rx="27" ry="18"/>

<text text-anchor="middle" x="1178.8623" y="-507.4279" font-family="Times,serif" font-size="14.00" fill="#000000">rat</text>

</g>

<!-- rat&#45;&gt;flea -->

<g id="edge42" class="edge">

<title>rat-&gt;flea</title>

<path fill="none" stroke="#000000" d="M1171.7498,-494.0839C1166.6219,-481.435 1159.6056,-464.1281 1153.772,-449.7384"/>

<polygon fill="#000000" stroke="#000000" points="1156.9606,-448.2879 1149.9599,-440.3355 1150.4735,-450.9179 1156.9606,-448.2879"/>

<text text-anchor="middle" x="1167.3623" y="-463.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge42_137"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6797873/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1175.2498,-469.0839 1175.2498,-491.0839 1202.2498,-491.0839 1202.2498,-469.0839 1175.2498,-469.0839"/>

<text text-anchor="start" x="1177.8592" y="-475.4839" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge42_137" class="icon" pointer-events="all"><circle cx="1215.9112" cy="-478.55150000000003" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1215.9112" cy="-482.55150000000003" r="0.75" fill="gold"/><rect x="1215.4112" y="-480.55150000000003" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- rodent -->

<g id="node29" class="node">

<title>rodent</title>

<ellipse fill="#add8e6" stroke="#000000" cx="740.8623" cy="-778.0279" rx="35.9154" ry="18"/>

<text text-anchor="middle" x="740.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">rodent</text>

</g>

<!-- rodent&#45;&gt;human -->

<g id="edge43" class="edge">

<title>rodent-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="2.4" d="M758.5615,-762.1522C774.0917,-748.2221 796.8642,-727.7959 814.3601,-712.1026"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.4" points="816.8754,-714.5482 821.9825,-705.2655 812.2013,-709.3373 816.8754,-714.5482"/>

<text text-anchor="middle" x="803.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">2</text>

<g id="a_edge43_138"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1533478/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="729.0615,-737.1522 729.0615,-759.1522 756.0615,-759.1522 756.0615,-737.1522 729.0615,-737.1522"/>

<text text-anchor="start" x="731.6709" y="-743.5522" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge43_138" class="icon" pointer-events="all"><circle cx="769.7229" cy="-746.6197999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="769.7229" cy="-750.6197999999999" r="0.75" fill="gold"/><rect x="769.2229" y="-748.6197999999999" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge43_139"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="729.0615,-713.1522 729.0615,-735.1522 756.0615,-735.1522 756.0615,-713.1522 729.0615,-713.1522"/>

<text text-anchor="start" x="731.6709" y="-719.5522" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge43_139" class="icon" pointer-events="all"><circle cx="769.7229" cy="-722.6197999999999" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="769.7229" cy="-726.6197999999999" r="0.75" fill="gold"/><rect x="769.2229" y="-724.6197999999999" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- sandflies -->

<g id="node30" class="node">

<title>sandflies</title>

<ellipse fill="#add8e6" stroke="#000000" cx="839.8623" cy="-778.0279" rx="45.2208" ry="18"/>

<text text-anchor="middle" x="839.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">sandflies</text>

</g>

<!-- sandflies&#45;&gt;human -->

<g id="edge44" class="edge">

<title>sandflies-&gt;human</title>

<path fill="none" stroke="#000000" d="M839.8623,-759.6285C839.8623,-747.5228 839.8623,-731.4355 839.8623,-717.6953"/>

<polygon fill="#000000" stroke="#000000" points="843.3624,-717.2999 839.8623,-707.2999 836.3624,-717.2999 843.3624,-717.2999"/>

<text text-anchor="middle" x="843.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge44_140"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="810.3623,-734.6285 810.3623,-756.6285 837.3623,-756.6285 837.3623,-734.6285 810.3623,-734.6285"/>

<text text-anchor="start" x="812.9717" y="-741.0285" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge44_140" class="icon" pointer-events="all"><circle cx="851.0237000000001" cy="-744.0961" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="851.0237000000001" cy="-748.0961" r="0.75" fill="gold"/><rect x="850.5237000000001" y="-746.0961" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- squirrel -->

<g id="node31" class="node">

<title>squirrel</title>

<ellipse fill="#add8e6" stroke="#000000" cx="1264.8623" cy="-511.6279" rx="40.5654" ry="18"/>

<text text-anchor="middle" x="1264.8623" y="-507.4279" font-family="Times,serif" font-size="14.00" fill="#000000">squirrel</text>

</g>

<!-- squirrel&#45;&gt;flea -->

<g id="edge47" class="edge">

<title>squirrel-&gt;flea</title>

<path fill="none" stroke="#000000" d="M1243.6115,-496.1601C1223.1201,-481.245 1192.029,-458.6148 1169.8087,-442.4414"/>

<polygon fill="#000000" stroke="#000000" points="1171.6192,-439.4302 1161.4744,-436.3751 1167.4998,-445.0898 1171.6192,-439.4302"/>

<text text-anchor="middle" x="1217.3623" y="-463.0279" font-family="Times,serif" font-size="14.00" fill="#000000">1</text>

<g id="a_edge47_141"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc6797873/pdf/" xlink:title="1">

<polygon fill="none" stroke="#000000" points="1247.1115,-471.1601 1247.1115,-493.1601 1274.1115,-493.1601 1274.1115,-471.1601 1247.1115,-471.1601"/>

<text text-anchor="start" x="1249.7209" y="-477.5601" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge47_141" class="icon" pointer-events="all"><circle cx="1287.7729" cy="-480.6277" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="1287.7729" cy="-484.6277" r="0.75" fill="gold"/><rect x="1287.2729" y="-482.6277" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

<!-- ticks -->

<g id="node32" class="node">

<title>ticks</title>

<ellipse fill="#add8e6" stroke="#000000" cx="932.8623" cy="-778.0279" rx="29.0509" ry="18"/>

<text text-anchor="middle" x="932.8623" y="-773.8279" font-family="Times,serif" font-size="14.00" fill="#000000">ticks</text>

</g>

<!-- ticks&#45;&gt;human -->

<g id="edge48" class="edge">

<title>ticks-&gt;human</title>

<path fill="none" stroke="#000000" stroke-width="2.4" d="M916.6629,-762.5601C902.2621,-748.8097 880.9939,-728.5019 864.4879,-712.7414"/>

<polygon fill="#000000" stroke="#000000" stroke-width="2.4" points="866.4681,-709.7929 856.8185,-705.4184 861.634,-714.8556 866.4681,-709.7929"/>

<text text-anchor="middle" x="898.3623" y="-729.4279" font-family="Times,serif" font-size="14.00" fill="#000000">2</text>

<g id="a_edge48_142"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7129920/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="920.1629,-737.5601 920.1629,-759.5601 947.1629,-759.5601 947.1629,-737.5601 920.1629,-737.5601"/>

<text text-anchor="start" x="922.7723" y="-743.9601" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge48_142" class="icon" pointer-events="all"><circle cx="960.8243" cy="-747.0277" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="960.8243" cy="-751.0277" r="0.75" fill="gold"/><rect x="960.3243" y="-749.0277" width="1" height="6" fill="gold"/></g></a>

</g>

<g id="a_edge48_143"><a xlink:href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmc7110368/pdf/" xlink:title="2">

<polygon fill="none" stroke="#000000" points="920.1629,-713.5601 920.1629,-735.5601 947.1629,-735.5601 947.1629,-713.5601 920.1629,-713.5601"/>

<text text-anchor="start" x="922.7723" y="-719.9601" font-family="Times,serif" font-size="14.00" fill="#000000">link</text>

<g id="icon_a_edge48_143" class="icon" pointer-events="all"><circle cx="960.8243" cy="-723.0277" r="8" fill="none" stroke="gold" stroke-width="1.5"/><circle cx="960.8243" cy="-727.0277" r="0.75" fill="gold"/><rect x="960.3243" y="-725.0277" width="1" height="6" fill="gold"/></g></a>

</g>

</g>

</g>

</svg>

    <div id="info_a_edge1_0" class= "paragraph">

<h3>paragraph source</h3>

<p>a strain of alphacoronavirus was identified in 7 h. cf. ruber nesting in the same cave (faucon cave) and caught the same year (2009). likewise, 2 other h. cf. ruber from the batouala cave but caught in 2010 were infected by the same alphacoronavirus infecting the 7 h. cf. ruber from the faucon cave. corman et al. 14 had already identified a closely related strain of alphacoronavirus in 2 h. cf. ruber from the faucon cave in 2009. altogether, these findings first suggest the transmission of covs between individuals of the same species through close contacts probably increased by social and roosting behavior, large colony size and reduced space in these caves 26 ; this strain of alphacoronavirus could be enzootic in h. cf. ruber in this area of gabon and roosts harboring a large number of bats nesting together promotes viral diffusion in the colony 29, 35 , and second, local movements of bats individuals in the region imputable to seasonal/reproductive or opportunistic roosting behaviors, as the two caves being only 67 km apart. the migratory behavior of some bats provides an opportunity for pathogens to spread over long distances, as reported for flying foxes of the species eidolon helvum able to migrate more than 1000 km 36 38 . in ghana, pfefferle et al. 6 identified a strain of alphacoronavirus in h. cf. ruber closely related to the human coronavirus 229e. our finding argues that close relatives of the human coronavirus 229e exist in african bats 14 . the phylogenetic proximity of the cov strains identified in this study with the human cov-229e strain suggests a risk of emergence of this virus in humans in africa. furthermore, the 10gb0354 alphacoronavirus strain (mg963188), from h. gigas bats caught in 2010 in the faucon cave, was phylogenetically close (93.8% nucleotide identity) to the alpaca cov strain detected in the vicugna pacos species, commonly known as alpaca (jq410000). maganga et al. 17 and corman et al. 14 already described two gabonese cov strains from hipposiderid bats caught in 2009 in the faucon cave closely related to alpaca cov. indeed, this coronavirus was described in 2007 during the epidemic of severe respiratory disorders associated with abortions in alpacas. it could be the direct progeny of its common ancestor with hcov-229e, sharing with the latter a nucleotide www.nature.com/scientificreports www.nature.com/scientificreports/ identity of 92.2% 39, 40 , on 440 bases of the nsp12 (polymerase). the phylogenetic link between these 2 covs could originate from an inter-species transmission, as a result of the opportunistic evolution of their common ancestor. as a result of genetic mutations and recombinations, the alpaca cov would have passed from bats to alpaca and vice versa. phylogenetic analysis also revealed that two betacoronaviruses from 2 h. gigas, caught in 2013 in the faucon cave, were phylogenetically close to a bat betacoronavirus named the zaria bat coronavirus (zcov), detected in nigeria in 2008 in a species of chiroptera genetically close to h. gigas, h. commersoni 27 , suggesting that phylogenetic proximity between host species may promote inter-species transmission 41 .

</p>

</div>



<div id="info_a_edge2_2" class= "paragraph">

<h3>paragraph source</h3>

<p>a strain of alphacoronavirus was identified in 7 h. cf. ruber nesting in the same cave (faucon cave) and caught the same year (2009). likewise, 2 other h. cf. ruber from the batouala cave but caught in 2010 were infected by the same alphacoronavirus infecting the 7 h. cf. ruber from the faucon cave. corman et al. 14 had already identified a closely related strain of alphacoronavirus in 2 h. cf. ruber from the faucon cave in 2009. altogether, these findings first suggest the transmission of covs between individuals of the same species through close contacts probably increased by social and roosting behavior, large colony size and reduced space in these caves 26 ; this strain of alphacoronavirus could be enzootic in h. cf. ruber in this area of gabon and roosts harboring a large number of bats nesting together promotes viral diffusion in the colony 29, 35 , and second, local movements of bats individuals in the region imputable to seasonal/reproductive or opportunistic roosting behaviors, as the two caves being only 67 km apart. the migratory behavior of some bats provides an opportunity for pathogens to spread over long distances, as reported for flying foxes of the species eidolon helvum able to migrate more than 1000 km 36 38 . in ghana, pfefferle et al. 6 identified a strain of alphacoronavirus in h. cf. ruber closely related to the human coronavirus 229e. our finding argues that close relatives of the human coronavirus 229e exist in african bats 14 . the phylogenetic proximity of the cov strains identified in this study with the human cov-229e strain suggests a risk of emergence of this virus in humans in africa. furthermore, the 10gb0354 alphacoronavirus strain (mg963188), from h. gigas bats caught in 2010 in the faucon cave, was phylogenetically close (93.8% nucleotide identity) to the alpaca cov strain detected in the vicugna pacos species, commonly known as alpaca (jq410000). maganga et al. 17 and corman et al. 14 already described two gabonese cov strains from hipposiderid bats caught in 2009 in the faucon cave closely related to alpaca cov. indeed, this coronavirus was described in 2007 during the epidemic of severe respiratory disorders associated with abortions in alpacas. it could be the direct progeny of its common ancestor with hcov-229e, sharing with the latter a nucleotide www.nature.com/scientificreports www.nature.com/scientificreports/ identity of 92.2% 39, 40 , on 440 bases of the nsp12 (polymerase). the phylogenetic link between these 2 covs could originate from an inter-species transmission, as a result of the opportunistic evolution of their common ancestor. as a result of genetic mutations and recombinations, the alpaca cov would have passed from bats to alpaca and vice versa. phylogenetic analysis also revealed that two betacoronaviruses from 2 h. gigas, caught in 2013 in the faucon cave, were phylogenetically close to a bat betacoronavirus named the zaria bat coronavirus (zcov), detected in nigeria in 2008 in a species of chiroptera genetically close to h. gigas, h. commersoni 27 , suggesting that phylogenetic proximity between host species may promote inter-species transmission 41 .

</p>

</div>



<div id="info_a_edge3_1" class= "paragraph">

<h3>paragraph source</h3>

<p>in addition to rodents, we also measured the sads-cov susceptibility of cell lines from humans, monkeys, chickens, and dogs, revealing a remarkably broad spectrum of tropism (table 1 and fig. 1) . as for the ability of sads-cov to grow efficiently in human cell lines, we should not underestimate the risk that this bat-origin cov may "jump" from pigs to humans. it is noteworthy that camel workers with high rates of exposure to camel nasal and oral secretions had evidence of mers-cov infection (36) . considering that sars-cov and mers-cov originated from bats and spread from one species to another through intermediate hosts (civets and camels, respectively), sads-cov may pose a similar risk to human health through transmission from pigs or other susceptible hosts.

</p>

</div>



<div id="info_a_edge4_13" class= "paragraph">

<h3>paragraph source</h3>

<p>in addition to rodents, we also measured the sads-cov susceptibility of cell lines from humans, monkeys, chickens, and dogs, revealing a remarkably broad spectrum of tropism (table 1 and fig. 1) . as for the ability of sads-cov to grow efficiently in human cell lines, we should not underestimate the risk that this bat-origin cov may "jump" from pigs to humans. it is noteworthy that camel workers with high rates of exposure to camel nasal and oral secretions had evidence of mers-cov infection (36) . considering that sars-cov and mers-cov originated from bats and spread from one species to another through intermediate hosts (civets and camels, respectively), sads-cov may pose a similar risk to human health through transmission from pigs or other susceptible hosts.

</p>

</div>



<div id="info_a_edge4_14" class= "paragraph">

<h3>paragraph source</h3>

<p>a newly enteric coronavirus, pdcov, was identified in the usa in 2014, and this coronavirus caused clinical signs in swine similar to those of pedv (wang et al., 2014a) . in this study, a novel peav (peav-gd-ch/2017) strain was identified from suckling piglets with diarrhea, and this strain shares high identities with the other two peav strains that were previously reported (gong et al., 2017; pan et al., 2017) . these novel peavs are most identical to bat-hku2, with 95% nucleotide identity, and have the same genome organization and trs motif for each orf. the greatest difference between peav and bat-hku2 is their s proteins, which share 85% amino acid identity each other, a value much lower compared with those of other proteins (table 1) . this difference is caused by amino acid mutations in the s protein, particularly in the ntd in the s1 subunit, which has been proven to be the key factor determining issue tropism and the host range of coronaviruses (lu et al., 2015) . in addition to its low amino acid identity with the s protein of hku2-like coronavirus, it shares low amino acid identity (lower than 28%) with s proteins of all known coronaviruses. thus, clarifying the origin of the s proteins of peav and hku2-like coronavirus is important for determining the origin and evolutionary history of these coronaviruses. a previous study showed that the extreme ntd in the s1 subunit of peav is structurally similar to that of nl63, while the rest of the s1 subunit is structurally similar to that of mhv (pan et al., 2017) . in addition, a short peptide in the s protein of bat-hku2 was found to be homologous to a corresponding peptide within the receptor-binding motif (rbm) in the s1 subunit of sars-cov (lau et al., 2007) . we also analyzed the arginine (position 545) at cleavage sites s1/s2 of peav and found that it is conserved in several beta-covs in this study. moreover, the phylogenetic tree based on the s protein presents a uniform evolutionary relationship; these bat-hku2like coronaviruses cluster with a newly identified rat-cov, lrnv, which represents a novel species of coronaviruses. all of these strains form a distinct lineage and cluster with beta-covs but are separate from all four known subgroups of beta-covs (fig. 3) , which may indicate that these strains are part of a novel subgroup of beta-covs. these results suggest that peav and hku2-like coronaviruses may have some relations with beta-covs and most likely resulted from recombination with the backbone of alpha-cov and the s gene from an unrecognized beta-cov. another large difference between peav and bat-hku2 is n protein, which share about 93.9% amino acid identity, as n protein is a multifunctional protein for coronaviruses, which is involving in virus replication, budding and pathogenesis et al. (mcbride et al., 2014) . while the role of these mutations in n protein between peav and bat-hku2 should further investigate. the origin and emergence time of a newly emerged pathogen are important issues to answer to determine the evolutionary history and plan methods of prevention for these new pathogens. for example, previous sars research reported that the interspecies transfer of sarslike coronaviruses from bats to the amplifying host (e.g., civet) occurred in 1998 and that interspecies transfer from civet to humans occurred in 2002 (chinese, 2004; hon et al., 2008; lau et al., 2010; song et al., 2005) . these results provide insight into the origin and evolutionary history of sars coronavirus. the origin and divergence time of other coronaviruses have also been estimated previously; the divergence time of bovine-cov and hcov-oc43 could be dated back to the end of the 19th century to the beginning of the 20th century and was estimated at 1910 (vijgen et al., 2006) . the tmrca for all pdcov strains was reported at 1991, approximately 24 years before pdcov was identified (woo et al., 2017) . in this study, we also addressed the emergence time and evolutionary history of peav and of the other coronaviruses based on the rdrp gene by bayesian analysis. in anticipation, the mean divergence time of bovine-cov and hcov-oc43 was estimated at 1914, and the mean tmrca of human and civet sarsr-cov was estimated at 2001 in our analysis (fig. 4) . these results are highly consistent with those of a previous report discussed above and further indicate that our analysis is unbiased. the emergence time of peav was estimated at 1926 (95% hpd, 1864 to 1984) based on our analysis, which indicates that peav is not newly emerged and may have circulated in swine herds for several decades since interspecies transmission from bat to swine occurred. in addition, these hku2-like coronaviruses have a common ancestor with hcov-nl63 and hcov-229e, and the divergence time was estimated at 277, which indicates that these hku2-like coronaviruses have a long evolutionary history. the mean tmrca for alpha-cov, beta-cov, and gamma-cov, as well as those of all coronaviruses estimated in this study, were later compared with those of a previous report (woo et al., 2012) . nevertheless, the mean tmrca coincides with the regions with 95% hpd to each other. the evolutionary rate of different coronaviruses was estimated previously; 4.3  10 4 substitutions per site per year was estimated for hcov-oc43 (vijgen et al., 2005) , and the mean evolutionary rate for group 1b coronaviruses was estimated to be 3  10 4 substitutions per site per year (pyrc et al., 2006) . in addition, the evolutionary rate for all coronaviruses was estimated to be 1.3  10 4 substitutions per site per year (woo et al., 2012) , which is estimated to be 1.93  10 4 (95% hpd, 1.27  10 4 to 3.57  10 4 ) nucleotide substitutions per site per year for the rdrp gene in this study, and all of these results are comparable to each other.

</p>

</div>



<div id="info_a_edge4_15" class= "paragraph">

<h3>paragraph source</h3>

<p>in the case of sars-cov, wild animals such as palm civets, sold as a delicacy on chinese 'wet markets', were initially identified as the immediate source of the virus [16] , but from 2005 insectivorous rhinolophid bats came into focus as the original reservoir, from where the virus was possibly transmitted to civets and other market species and from them to humans [17, 18] (see [19] for a recent review on bat coronaviruses). however, it took until 2013 to discover a bat coronavirus that is more than 95% identical to sars-cov and uses the same receptor on the surface of host cells, the angiotensin-converting enzyme 2 (ace2) [20] . in the case of mers coronavirus (mers-cov), bats were again suspected to be the reservoir as a few coronaviruses with high sequence similarity to mers-cov were discovered in african and european bats [21, 22] , but in recent months the picture has changed somewhat and dromedary camels are now the main suspects of being the reservoir from where the zoonotic transmission into the human population originates [23, 24] .

</p>

</div>



<div id="info_a_edge4_16" class= "paragraph">

<h3>paragraph source</h3>

<p>as the ctr of the s 1 subunit contains the protein rbd for most covs, also mutations in this part of s have been associated with changes in the virus' tropism. perhaps the most well-known example of viral cross-species transmission involves the sars-cov. studies support a transmission model in which a sars-like cov was transmitted from rhinolophus bats to palm civets, which subsequently transmitted the palm civet-adapted virus to humans at local food markets in southern china . according to this model, sars-like viruses adapted to both the palm civet and human host, which was reflected in the rapid viral evolution observed for these viruses within these species . two-amino acid substitutions within the rbd were elucidated that are of relevance for binding to the ace2 proteins of palm civets and humans (li et al., 2005b qu et al., 2005) . from these studies it appears that due to strong conservation of ace2 between mammalian species only a few amino acid alterations within the rbd are needed to change coronavirus host species tropism. indeed serial passage of sars-covs in vitro or in vivo can rapidly lead to adaptation to new host species (roberts et al., 2007) . sars-like viruses isolated from bats displayed major differences including a deletion in the ace2 rbm compared to human sars-cov (drexler et al., 2010; ren et al., 2008) and as a consequence were unable of using human ace2 as an entry receptor (becker et al., 2008) . however, recently a novel sarslike btcov was identified, which could use ace2 of rhinolophus bats, palm civets as well as of humans as a functional receptor . these findings not only provide further evidence that bats are indeed the natural reservoir for sars-like covs, but also that these bat coronaviruses can directly include human ace2 in their receptor repertoire. the detection of sequences of sars-cov-like viruses in palm civets and raccoon dogs (guan et al., 2003; tu et al., 2004) therefore probably reflects the unusually wide host range of these viruses. a similar promiscuous receptor usage is also observed for mers-cov which binds to dpp4 of many species (barlan et al., 2014; eckerle et al., 2014; falzarano et al., 2014; m uller et al., 2012; van doremalen et al., 2014) as indicated earlier.

</p>

</div>



<div id="info_a_edge5_17" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses mainly cause respiratory and gastrointestinal tract infections and are genetically classified into four major genera: alphacoronavirus, betacoronavirus, gammacoronavirus, and deltacoronavirus (li, 2016) . the former two genera primarily infect mammals, whereas the latter two predominantly infect birds (tang et al., 2015) . six kinds of human covs have been previously identified. these include hcov-nl63 and hcov-229e, which belong to the alphacoronavirus genus; and hcov-oc43, hcov-hku1, severe acute respiratory syndrome coronavirus (sars-cov), and middle east respiratory syndrome coronavirus (mers-cov), which belong to the beta-coronavirus genus (tang et al., 2015) . coronaviruses did not attract worldwide attention until the 2003 sars pandemic, followed by the 2012 mers and, most recently, the 2019-ncov outbreaks (china cdc, 2020; song et al., 2019) . sars-cov and mers-cov are considered highly pathogenic (cui et al., 2019) , and it is very likely that both sars-cov and mers-cov were transmitted from bats to palm civets (guan et al., 2003) or dromedary camels (drosten et al., 2014) , and finally to humans (cui et al., 2019) .

</p>

</div>



<div id="info_a_edge5_18" class= "paragraph">

<h3>paragraph source</h3>

<p>it was reported that 27 of the first 41 infected patients had been exposed to the huanan seafood market [18] . thus, it was believed that the new coronavirus originated from the huanan seafood market in wuhan and spread from animal hosts to humans in the process of wildlife trade, transportation, slaughter, and trade. bats have the most variety of coronaviruses in their bodies and are the hosts of many kinds of coronaviruses, such as the sars-covand the mers-cov [19] . the sars-cov and the mers-cov are considered highly pathogenic, and it is very likely that the sars-cov was transmitted from bats to palm civets and the mers-cov was transmitted from bats to dromedary camels and finally to humans [20, 21] . given the high sequence similarity between the sars-cov-2 and the sars-like bat covs from hipposideros bats in china, the natural host of the sars-cov-2 may be the hipposideros bat. the discovery that pangolin coronavirus genomes have 85.5% to 92.4% sequence similarity to sars-cov-2 suggests pangolins should be considered as possible hosts in the emergence of sars-cov-2 [22] .

</p>

</div>



<div id="info_a_edge5_19" class= "paragraph">

<h3>paragraph source</h3>

<p>three emerging, highly pathogenic human covs are sars-cov, middle east respiratory syndrome (mers)-cov, and covid-19 virus, which was previously named 2019-ncov by the world health organization (who), and is also known as hcov-19 or sars-cov-2 [1] . atypical pneumonia (sars) was first reported from guangdong province, china in late 2002. sars caused a global pandemic in 2003 with approximately 10% (774/8098) case fatality rate (cfr) [2] . sars-cov has not circulated in humans since 2004. mers-cov was first reported from saudi arabia in 2012 and has continued to infect humans with limited human-to-human transmission, leading to a cfr of approximately 34.4% (858/2494) in 27 countries, according to the most recent who report i . both sars-cov and mers-cov are zoonotic viruses. they use bats as their natural reservoirs and transmit from bats to intermediate hosts (e.g., palm civets for sars-cov, dromedary camels for mers-cov), leading to infection in humans [2, 3] .

</p>

</div>



<div id="info_a_edge6_20" class= "paragraph">

<h3>paragraph source</h3>

<p>humans are nowadays very often in contact with pets or farm animals such as cattle, pigs, poultry, and horses. although there are many infectious diseases typically found in people working with livestock, according to the microorganisms-driven genetic selection theory humans should have less to fear from livestock than from wild animals regarding the risk of deadly zoonotic diseases since they shared the same ecosystem with livestock for millennia. this does not mean humans should not remain vigilant (this is the reason why milk pasteurization exists and why meat must be cooked before consumption) and not worry about pathogens abnormally present in their livestock because of the interconnectedness of different ecosystems. despite veterinary figure 3 </p>

</div>



<div id="info_a_edge7_21" class= "paragraph">

<h3>paragraph source</h3>

<p>we next examined how the sars-cov-2 codon pair usage compares with other coronaviruses and to its current host. codon pair data inherently contain the codon usage data and therefore are better suited than codon usage data for this type of comparison. as expected, sars-cov-2 codon pair usage closely resembles the codon usage of the coronoviridae family, while it is quite distinct from the codon pair usage of the human genome (table 1) . bat (chiroptera) and pangolin (pholidota) from which the virus may have been transmitted to humans, as well as dog (canis lupus familiaris) to which the virus is feared may be transmitted next, were included in the analysis. we find that these species have a similar codon usage when compared with human; therefore, viral tropism cannot be inferred based on codon usage data alone (table 1) . since sars-cov-2 infects bronchial epithelial cells and type ii pneumocytes and our recent findings show that transcriptomic tissue-specific codon pair usage can vary greatly from genomic codon pair usage [24] , we also examined the transcriptomic codon pair usage of the lung and how it compares with the sars-cov-2 codon pair usage. rather surprisingly, the codon usage in the lung was more distinct from sars-cov-2 codon usage than the homo sapiens genomic codon usage. the transcriptome codon pair usage of kidney and small intestine, tissues that are also susceptible to the infection, are similarly distant from sars-cov-2 (table 1) .

</p>

</div>



<div id="info_a_edge7_22" class= "paragraph">

<h3>paragraph source</h3>

<p>both mers-cov and sars-cov are zoonotic pathogens and are believed to have been transmitted from a natural host, possibly bats, to humans through intermediate mammalian hosts 12, 13 . the key determinant of host specificity is the envelope-located trimeric spike (s) glycoprotein, which can be further cleaved by host proteases into an n-terminal s1 subunit and a membrane-bound c-terminal s2 region 14 . the cleaved s protein remains non-convalently associated in the metastable pre-fusion conformation. after virus endocytosis by the host cell, a second cleavage is generated, which is mediated by endolysosomal proteases (s2 0 cleavage site), allowing membrane fusion activation to occur. in the s1 subunit, the receptor binding domain (rbd, also called the c terminal domain, ctd) is localized in the c-terminal region, spanning b200 amino acids, and structural studies have revealed that the rbd consists of two subdomains: the core and external subdomains [14] [15] [16] [17] . in the s2 subunit, the heptad repeat (hr) regions are also well characterized [18] [19] [20] , and as expected, the hr1 and hr2 of mers-cov fold into an intra-hairpin helical structure that can assemble trimerically into a six-helix bundle (a trimer of the hr1/hr2 heterodimer), demonstrating a classical type i membrane fusion process 21 . peptide inhibitors have been designed targeting these hr regions and been proven to be effective in vitro and in vivo 18, 19, [22] [23] [24] . these studies have provided insight into the characteristics of mers-cov and sars-cov s components; however, the overall s protein structures of these two highly pathogenic covs remain to be investigated. this will further enhance our understanding of s protein function and subsequent design of broadly neutralizing antibodies and vaccine immunogens.

</p>

</div>



<div id="info_a_edge7_23" class= "paragraph">

<h3>paragraph source</h3>

<p>in december 2019, covid-19 pandemic emerged in wuhan china. it became a global health problem in a very short time because of it contact-transferred behaviours 1, 2 . covid-19 is caused by a virus known as severe acute respiratory syndrome coronavirus 2 or also sars-cov-2 3 . a large family of these virus are causing different kind diseases such as cold, middle east respiratory syndrome and severe acute respiratory syndrome. the new addition to coronavirus family covid-19 is discovered in 2019, which has never been detected in humans. covid-19 is known as zoonotic disease because it is transferred from animal to human such as bat 4 , similar to sars-cov virus which is contaminated through cat and mers-cov from dromedary 5 . it is presumed that respiratory transmission and physical contact is cause to spread covid-19 rapidly. according to world economic forum , people with no symptoms or mild symptoms are the main reason of spreading the epidemic. one our of four subject shows no symptoms of covid-19, although he/she is suffering with the disease 6 .

</p>

</div>



<div id="info_a_edge7_24" class= "paragraph">

<h3>paragraph source</h3>

<p>during the 1980s, several important findings were reported. the coronavirus genomic rna is transcribed into a set of subgenomic mrnas that encode viral proteins. using the model mhv, these subgenomic rnas were shown to each contain a leader sequence derived from the 59 end of the genome (lai et al., 1982) , initially by rna fingerprinting methods. these nested subgenomic rnas came to define a larger superfamily of nidoviruses ("nidus" means nest) including several other virus families. later it was found that subgenomic mrnas are transcribed from negative strand rnas that were synthesized from the full-length genomic rna (wu and brian, 2010) . this is a unique mechanism involving noncontiguous transcription of the genome, presumed to occur by a process involving viral polymerase jumping from one part of the genome template to another. this leads to a high rate of recombination for coronaviruses, which may play a role in viral evolution and interspecies infections. however, the ancestral bat virus of sars-cov-2, the species from which it was transferred to humans, and the sequence changes leading to the evolution of this highly pathogenic virus have yet to be identified.

</p>

</div>



<div id="info_a_edge7_25" class= "paragraph">

<h3>paragraph source</h3>

<p>in fact, it is a novel virus which most likely recently passed from bat to human and then almost unknown to human immune system. besides, as the other coronaviruses, the covid-19 seems to modify rapidly during spreading, further avoiding immune defences (2) . however, the most clear and troubling difference of the novel coronavirus covid-19 compared to others alfa and beta coronaviruses is the higher mortality rate, that today reached a value of over 3%

</p>

</div>



<div id="info_a_edge7_26" class= "paragraph">

<h3>paragraph source</h3>

<p>presently coronavirus is created a health emergency to the world population and became a pandemic [2] , [3] , [4] . initially, this virus is transferred from the bat animal to the human [5] .

</p>

</div>



<div id="info_a_edge7_27" class= "paragraph">

<h3>paragraph source</h3>

<p>although many scientists believe that bats are the intermediate host, most species of bats live in tropical or subtropical rain forests and caves far from human populations, thus the probability of bats transmitting the virus directly to humans is unlikely. it is generally known that bats may spread viruses to other intermediate hosts such as wild animals or livestock, from which the viruses are then transmitted to humans. ji et al. carried out a comprehensive sequence analysis and comparison in conjunction with relative synonymous codon usage bias among different animal species based on the sars-cov-2 sequence [13] . the result suggested that sars-cov-2 appeared to be a recombinant virus between the bat coronavirus and a coronavirus of unknown origin. they considered that snake was a most probable wildlife animal reservoir, but the conclusion was controversial. guo et al. introduced vhp (virus host prediction) to predict the potential hosts of viruses by deep learning algorithm and predicted that bat coronaviruses were assigned with more similar infectivity patterns with sars-cov-2 [14] . the results illustrated that bat and mink might be two candidate reservoirs of sars-cov-2. lam et al. identified pangolinassociated coronaviruses belonging to two sublineages of sars-cov-2-related coronaviruses, including one very closely related to sars-cov-2 in the rbd by metagenomic sequencing [15] . hence, pangolins should be considered as possible intermediate hosts and should be removed from markets to prevent such zoonotic transmission.

</p>

</div>



<div id="info_a_edge7_28" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses (covs) are well-known as a major cause of the common cold in humans by mild infection in the upper respiratory tract. covs have gained attention since the outbreak of severe acute respiratory syndrome (sars) in 2002-2003 when sars-related cov (sars-cov) was identified (anderson et al. 2004) . sars-cov is the cause of human lower respiratory tract diseases such as bronchitis and pneumoniae (holmes 2003; girard et al. 2005; peiris et al. 2004 ). in 2012, one more cov was isolated in saudi arabia and named mers-cov. this virus belongs to lineage c of beta-coronaviruses (zaki et al. 2012 ). according to a world health organization (who) survey, 2143 cases of mers-cov infections were reported, and 750 people were killed in 27 countries by march 2018. in south korea, 186 people were infected by mers-cov, and 36 people died in 2015. the symptoms of the mers-cov infection in south korea included fever, cough, upper respiratory tract signs, lower respiratory tract distress with lymphopenia and elevated liver enzymes, acute respiratory distress syndrome, and multiple organ system failure (zhang et al. 2016) . mers-cov is reportedly transmitted to humans generally from bats or dromedary camels, but the exact route of infection has not yet been identified (who mers-cov research group 2013; yin and wunderink 2018; mackay and arden 2017) .

</p>

</div>



<div id="info_a_edge7_29" class= "paragraph">

<h3>paragraph source</h3>

<p>mers-cov has a wide species tropism, as the virus can replicate in a variety of mammalian cells of different origins [37] . this suggests that the virus originated from animals. as is the case with several beta-covs, bats are the common reservoir from which the virus jumped to humans through intermediate hosts [38] . mers-cov is thought to be transmitted to humans directly from bats or through an intermediate host, most likely dromedary camels [39] [40] [41] . many studies have sought to define the animal reservoir, intermediate host(s), and the possibility of human-to-human transmission. although bats are considered the ancestral reservoir of covs [42] , camels are important for the maintenance and diversification of mers covs, and they are the source of human infections. it has been estimated that camel exposure resulted in 12% of mers-cov infections [43] . a survey of mers-cov-infected individuals from 20 hospitals in saudi arabia revealed that some had direct contact with camels, while others had likely acquired the virus from infected humans [44] . the possibility of human-to-human transmission was also supported by the presence of hospitalassociated outbreaks in several countries, including saudi arabia [30, 33, [45] [46] [47] [48] .

</p>

</div>



<div id="info_a_edge7_30" class= "paragraph">

<h3>paragraph source</h3>

<p>we found that mortality rates were higher among the elderly group for both genders which was also concordant with a previous study [26] . the possible explanation for the enhanced mortality in aged patients could be the presence of senescence-associated immune vulnerability in these individuals and suboptimal immune reactivity following a systemic challenge by exposure to mers-cov natural infection. other gulf countries show a few sporadic cases which may be due to the missing data that still have to be set straight or it could possibly be due to small size of camel populations is wide spread across vast geographical region. there may be still other factors involved that remain hidden at present but contribute significantly to the survival, transmission and pathogenesis of this relatively newly identified pathogen in this region of the world. notably, it appears as if coronaviruses are able to cause serious viral infections when transferred from their reservoir (wild bat) host to the human host as observed previously for ebola virus as well which was transmitted from wild bats to humans in africa. actually, many of mers-cov cases are initiated in rural areas and following hospitalization, further cases were reported. the majority of the mers-cov outbreak cases took place at health facilities; index cases are very crucial and they raise the question of the route of transmission of this zoonotic virus. this warrants caution that strict healthcare protocols and guidelines need to be followed and practiced by health care personnel in order to prevent the new outbreaks of mers-cov.

</p>

</div>



<div id="info_a_edge7_31" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses are viruses that circulate among animals and are named because of the crownlike spikes (protein spikes) that protrude from their surface resembling the sun's corona. the first transmission of this type of virus from animals to humans happened in 2002 in the guangdong province of china which resulted in sars (severe acute respiratory syndrome). bats were thought to be the potential source of this virus. a novel coronavirus (ncov) was identified in early january 2020 which was traced to the severe pneumonic outbreak of an undocumented cause in early december 2019 in the city of wuhan, china. similar to sars, bats are seen as the potential source from which this virus has spread to humans. due to rapid spread of the virus within a short time globally resulting in health emergencies in a number of countries, who declared this as a pandemic on 11 th march 2020. initially the virus was named 2019-ncov but later was named sars-cov-2 due to its "genetic relationship to the sars-cov-1 virus"(medscape, 2020) the entire human population is at potential risk as being a new virus nobody has prior immunity to it. there is no vaccine and no specific treatment for the disease and is highly transmissible. epidemiological estimation at present is that on an average, one infected person will infect between two to three other people. the spread has been from respiratory droplets and from contaminated surfaces. the world is facing a common challenge; how to control the spread of and what can be the effective interventions to control mortality. the early examples from china suggested the most effective method to control the spread of the virus is through lockdowns and social distancing measures. the example of south korea suggested testing as the major component of the mitigation measures. some studies pointed out the importance of hand hygiene and face masks in controlling the virus. with new hotspots emerging, the number of new cases and those not able to recover are raising new concerns every day. risk and uncertainty behind this disease control has generated a global concern for health, economy, and for persons at large. the alarming spread of the virus has shocked people across the world pushing among others researchers to understand the virus-its structure, transmission, replication mechanism, latency, etc. and promising interventions that can effectively control it. extensive global efforts are undertaken to develop vaccine and drug. this is unsurprisingly leading to huge volume of research activity within a short period of time increasing at an exponential rate. as the recent editorial published in the lancet highlights "the whole-genome sequence of sars-cov-2 had been obtained and shared widely by mid-january, a feat not possible at such speed in previous infectious disease outbreaks" the editorial points out the importance of the need for development of effective diagnostics, therapeutics and vaccine for the virus. examining from the dimensions database as of 22 april, it was found that 1633 clinical trials are being conducted on the virus and 171 policy documents have been published so far. the number of research papers, clinical trials at different phases within such a short period is unprecedented and shows the intensive efforts of the global research community to understand the different aspects of this disease and address it. seven patents have also been granted. it is important to capture insights of influential research and innovation from this ongoing activity for policy makers and research scholars from cross-disciplinary areas to build up further on this valuable repository. societal impact and what aspects are of concern to the people at large are difficult to capture. one useful method would be from online trends surrounding this disease that would indicate to some extent the key issues that are influencing the society at large. the present study is motivated by this and applies tools and techniques of scientometrics to uncover insights from research papers. scientometrics applies various mathematical and statistical techniques to capture insights of research activity from research papers and patents and other published sources including online sources (altmetrics) by constructing various types of indicators. citation based analysis is a prominent method to capture academically significant and theoretically relevant material (see for example glanzel, 2003) . keeping in view the research activity in this area started primarily with the outbreak of this disease, impact captured through citations would not give a correct picture as citations takes time to accrue. this is true for research paper as well as patent citations. citations that influence current research activity would however be useful to construct the present knowledge base. one of the useful method that can do so is based on cocitation analysis which captures frequency with which two documents are cited together (small, 1973) . co-citation establishes an intellectual relationship with earlier literature in a field/subfield/area of research; strength of relationship based on frequency of co-citation pairs. the rise of social networking websites like twitter, facebook etc. provides researchers a wider scope to share their scholarly publications. altmetrics allows to track and capture online impact of scholarly research and thus broadly indicates papers that are influencing the research community. to put it in a proper perspective, one can borrow from william (2017), "altmetrics are measurements of how people interact with a given scholarly work". altmetrics or article level metrics according to das and mishra (2014) is a "new trendsetter" to measure "impact of scientific publication and their social outreach to intended audience". it reflects "a scholarly article's popularity, usage, acceptance and availability" by using an altmetric score. google trends which was launched in 2006, primarily shows how frequently a particular search term is entered in comparison with all other search terms in different regions and languages (google, 2017). in google trends level of interest in a topic is approximated using search volume of google. sullivan (2016) estimated searches on google trends reached 2 trillion in 2016! thus, this is one of the most significant source of data if it is properly analysed. one of the most influential study was by ginsberg et al. (2006) which showed that google trends traced and predicted the spread of influenza earlier than the centers for disease control and prevention. jun et al. (2018) provides a good assessment of research studies in the past decade which have utilized google trends. they highlight the diverse fields in which this has been used for, from merely describing and diagnosing research trends to forecasting changes. according to mavragani et al. (2018) "google trends shows the changes in online interest for time series in any selected term in any country or region over a selected time period, for example, a specific year, several years, 3 weeks, 4 months, 30 days, 7 days, 4 hours, 1 hour, or a specified time-frame." they argue that as the internet penetration is increasing web based search activity has become a valid indicator of public behaviour. the paper positions itself in this direction; applying various tools and techniques of scientometrics, altmetrics and google trends to draw meaning from the huge volume of research papers and online activity surrounding this pandemic. the study attempts to answer the following research questions:

</p>

</div>



<div id="info_a_edge7_32" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses that are closely related to sars-cov are typically found in bats (li et al. 2005) . there is evidence that sars-cov originated in bats in china and reached humans most probably after jumping to an intermediate host, the civet (paguma larvata) (reviewed by wang and eaton 2007) . similarly, mers-cov is endemic in dromedary camels in the middle east, from which it can be transmitted to humans, but it may also have originated in bats (cui et al. 2019) . the mers epidemic is ongoing, and as of december 2019, 2468 cases have been reported (baharoon and memish 2019) and the mortality rate has been as high as 42% (mohd et al. 2016) . at the other end of the virulence spectrum are coronavirus species that cause common colds in humans with relatively mild symptoms (reviewed by su et al. 2016) .

</p>

</div>



<div id="info_a_edge7_33" class= "paragraph">

<h3>paragraph source</h3>

<p>the existence of corona virus is not new, but this time it has been spread from animal (bat) to human. the occurrence of having such epidemic had already been predicted last year in china [2] . the first spread of this novel corona virus was started from wuhan, china on december, 2019. now, almost all the parts of the world are exposed to this virus and since the inflectibility of this virus is large, a huge number of people have already become victim of it. in the lockdown situation, all the countries are trying to identify the infected people and isolate them. so, how many people will be more affected by this virus, is a million dollar question. for the countries like italy, usa, spain, india etc., the number of new cases are increasing day by day even after the declaration of lockdown by the respective government.

</p>

</div>



<div id="info_a_edge7_36" class= "paragraph">

<h3>paragraph source</h3>

<p>one of the most difficult issues concerned reports that u.s. officials were investigating the possibility the coronavirus was secretly 'manufactured' and/or 'escaped' from a chinese lab in wuhan, specifically the wuhan institute of virology. there is no scientific evidence to support these theories. the sequencing and analysis of coronavirus genomes have already dispelled these rumors and instead demonstrate that bats are the likeliest source, suggesting that covid-19 was created by nature, not humans. at its molecular level, the viral genome most closely resembles an isolate that already exists in horseshoe bats in hunan province. supported by several studies, bats have an unusually high capacity to harbor viruses and have been linked to past outbreaks, including sars, mers and ebola. the virus may have spread from bats to an intermediary animal before infecting humans; this remains unclear. the fact that the earliest cases of covid-19 were linked to a live animal market in wuhan that sold exotic species only bolsters these observations.

</p>

</div>



<div id="info_a_edge7_37" class= "paragraph">

<h3>paragraph source</h3>

<p>c ovid-19 is currently a global pandemic impacting around 212 countries and has already claimed the lives of more than 79,385 individuals worldwide with confirmed cases globally now standing at around 1,356,780. 1 experts across the world warn that the disease, which originated in the wuhan district of china in the december of 2019, will only propagate more and the present numbers are likely to inflate further. the sudden outbreak of covid-19 shook the scientific fraternity, and scientists across the world are working tirelessly to understand the virus and its properties in order to design intervention strategies to combat the disease. so far, we have been able to understand that covid-19 is caused by a virus known as sars-cov-2, which is a single-stranded rna virus (ssrna) with a genome size of 29903 bp. sars-cov-2 belongs to the same beta-coronavirus clade of the previously reported sars-cov and mers-cov and bears sequence similarity to sars-cov. 2 in fact, not only do they bear similarity in sequences, but also the entry point of both the viruses in humans is through the same receptor recognition. the virus sars-cov-2 has been speculated to have been transmitted from bats to human considering the fact that evidence of a similar coronavirus, ratg13, has been reported in bats. 3 sars-cov-2 is made up of three structural proteins, namely, the spike (s), envelope (e), and membrane (m), which makes up the viral envelope, and the nucleocapsid containing the rna genome. it is the spike protein that carries out the frontline action for the virus by performing the initial receptor recognition with the human angiotensin-converting enzyme-2 receptor (hace2). 3 it is this interaction of hace2 with the viral spike protein that is central to the viral infection.

</p>

</div>



<div id="info_a_edge7_38" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<div id="info_a_edge8_39" class= "paragraph">

<h3>paragraph source</h3>

<p>the life-threatening pneumonia outbreak started as an epidemic within wuhan city, china. on december 31, 2019, the wuhan municipal health commission reported critically ill cases of viral pneumonia. later, the international committee on taxonomy of viruses (ictv) identified the virus as sars-cov-2 [26] . the chinese government then mandated a quarantine to control the epidemic stage and to determine the effectiveness of a quarantine against the virus [27] . however, sars-cov-2 was transmitted rapidly, quickly crossing borders of various countries. thus, the number of established cases climbed quickly, causing the who to announce covid-19 a global pandemic disease on march 11, 2020. initially, china had the most confirmed cases, followed by north america and europe. italy and the us, however, surpassed china in terms of the number of established cases. thus, at this stage of vigorous spread, it is essential to determine the mechanism of viral infection to control and treat the disease. until now, it was believed that bat, especially rhinolophus affinis, was the natural host of the virus [7] . it is still uncertain how this virus was transmitted from bat to human, although some research has suggested that pangolin (manis javanica) could be the intermediate host [14, 28, 29] . the aggressive intervention steps were taken by governments, such as isolating and quarantining latent patients, have reduced the contact rate and effectively decreased the peak for covid-19 cases.

</p>

</div>



<div id="info_a_edge9_40" class= "paragraph">

<h3>paragraph source</h3>

<p>a large number of molecular biologic and sequencing studies established that the coronaviruses share a common genetic organization and mode of replication. infected cells contain a set of 6 to 9 virus-specific polyadenylated rnas that make up a 3 co-terminal nested set, with the largest member of this set being the genome rna and the remaining rnas representing subgenomic mrnas. the subgenomic rnas encode the structural proteins enumerated above in a characteristic order: he (if present)-s-e-m-n. in addition to these structural genes, coronavirus subgenomic rnas encode accessory proteins that are not essential for viral replication. they generally modulate cellular responses to infection and are specific for each coronavirus subgroup. the replicase proteins are encoded in the 5 two-thirds of the genome. this region contains two very long partially overlapping open reading frames that are translated by a ribosomal frame shifting mechanism. the 750-kda primary translation product is cotranslationally autocatalytically cleaved into 16 nonstructural proteins (nsps) that encode the machinery needed for rna replication and transcription. many of the coronaviruses are associated with respiratory or enteric diseases, although some viruses replicate and cause disease at other sites, including the liver, spleen, and central nervous system. most of the coronaviruses are relatively species specific, only infecting their natural host and cell cultures derived from that host or a closely related host species. however, there are several examples of coronaviruses jumping species, with sars-cov being the most well known example. sars-cov is thought to have originated in bats, where sars-like covs have been identified, and then transmitted to civets and raccoon dogs in wild animal markets in southern china, from which they were subsequently transmitted to humans (lau et al., 2010) . during this chain of transmission, several key adaptations to the new host occurred (li et al., 2005a,b) . ccov and fecov provide another exception to the high degree of species specificity of coronaviruses, in that both viruses are able to grow in feline cells. phylogenetic analysis of the group ia coronaviruses, ccov, fecov, and tgev, suggest a complex set of recombination events has given rise to these viruses from a common ancestor, likely explaining the ability of both ccov and fecov to infect feline cells (wesseling et al., 1994; lorusso et al., 2008) . the species specificity of coronaviruses is controlled by the ectodomain of the s protein, and can be altered by creating chimeric s proteins that contain the ectodomain of a distantly related cov (kuo et al., 2000) .

</p>

</div>



<div id="info_a_edge10_110" class= "paragraph">

<h3>paragraph source</h3>

<p>nucleotide sequence analysis of some human enteric viruses has indicated a high degree of sequence similarity with animal strains. notably, hepatitis e virus-related sequences have been detected in pigs (meng, 2000; van der poel et al., 2001; banks et al., 2004) and birds (huang et al., 2002) . zoonotic infections may occur either through direct transmission, suspected for hepatitis e virus (hev; reyes, 1993) and caliciviruses (humphrey et al., 1984) , or through incidental co-infection of a host with animal and human viruses, resulting in the mixing of genes and generation of novel variants (recombination/reassortment; unicomb et al., 1999) . recombination has been demonstrated as a mechanism for rapid expansion of diversity for noroviruses and rotaviruses, but it is likely to be a common feature of the rna viruses involved (jiang et al., 1999; unicomb et al., 1999; lopman et al. 2004) . viruses related to the human rotaviruses, astroviruses, noroviruses, sapoviruses, and hev circulate in several animal species, providing a huge reservoir for virus diversity (shirai et al., 1985; meng et al., 1997; van der poel et al., 2001; huang et al., 2002; oliver et al., 2006) . to corroborate these hypotheses, animal viruses have been recently characterised in shellfish samples from the market either in europe or in the us (dubois et al., 2004; costantini et al., 2006) . sars, reported in november 2002 (ksiazek et al., 2003 , is an example of an emerging disease. the primary mode of transmission of the sars coronavirus appears to be direct mucous membrane contact with infectious respiratory droplets and/or through exposure to fomites. several coronaviruses are known to spread by the faecaloral route, but there is no current evidence that this mode of transmission plays a key role in the transmission of sars, although there is a considerable shedding of the virus in stools (tsang, 2003) . another emerging pathogen of concern is the avian influenza h5n1 virus, highly pathogenic among birds, that in some cases has been transmitted from birds to humans. most cases of h5n1 infection in humans to date have occurred as a result of direct contact with poultry or with surfaces and objects contaminated by their faeces. however, concern has recently been expressed about the potential for transmission of the virus to humans through water and sewage, although no definitive cases have been reported to date (who, 2006, http://www.who.int/ water_sanitation_health/emerging/h5n1background.pdf) .

</p>

</div>



<div id="info_a_edge10_111" class= "paragraph">

<h3>paragraph source</h3>

<p>the common human influenza virus (h3n2) is highly contagious but rarely lethal. avian flu in chickens (h5n1) is a particularly virulent type that can kill rapidly and causes widespread organ damage. fortunately, it is not easily transmitted from birds to humans, and more importantly, human to human spread is poor. however, swapping genetic material, should an individual be co-infected with both, might produce a hybrid h5 (avian flu) n2 (human flu) virus with devastating effects. dna sequencing of the viral genome from various outbreaks has shown that the virus continues to mutate. this has implications for pathogenicity, as well as antiviral drug resistance, and having the right vaccine ready if needed. in this unpredictable environment, the value of rapid nat diagnostics is crucial to detect early cases and for public health surveillance. the genes of the virus that caused the 1918 pandemic have been studied to better understand what makes an influenza virus virulent and capable of producing a pandemic [35] . (h1n1, 1918) : the virus from this pandemic, which killed about 40 million people, had not been isolated. without a virus little research was possible, then the viral rna sequence was determined using material from archival tissue, including formalin-fixed autopsy material. the sequence itself did not provide clues for why the spanish influenza virus was so virulent, and so the next step was to reconstruct the viral coding segments and clone them into plasmids. individual genes from the h1n1 1918 virus were then introduced into a common laboratory viral strain and pathogenicity sought. although the h and n glycoproteins were factors in the virulence of this virus, it was also shown that one of the rna polymerase subunits known as pb1 was involved. another subunit (pb2) was then found to be important for viral transmissibility [35] . swine influenza (h1n1, later called h1n1(09), 2009): after the appearance and then rapid disappearance of sars (box 6.4), followed by the concerns regarding the possibility of a h5n1 pandemic that did not occur (so far), this infection attracted a lot of publicity and provoked considerable fear when it emerged in china and then hong kong in 2003. sars subsequently spread to many countries, producing around 700 deaths in the first half of 2003. this was at one time described as the first pandemic of the 21st century, but it never progressed beyond an epidemic because of effective public health measures effected by mid 2003 [36] . the social and economic impacts of this infection were considerable, including major disruptions to international travel. sars was shown to be caused by a novel coronavirus (cov) which was thought to have crossed the species barrier, although the animal reservoir for sars took a while to find. it is now thought to be:

</p>

</div>



<div id="info_a_edge10_112" class= "paragraph">

<h3>paragraph source</h3>

<p>most emerging infectious diseases are zoonoses, the pathogens of which are transmitted between humans and animals. the 2009 pandemic h1n1 influenza virus spread worldwide through reassortment that exchanged a gene segment between pigs and humans [1] . recently, cases of influenza a virus h7n9 transmitted from birds to humans have been reported [2] . the 2003 severe acute respiratory syndrome (sars) outbreak originated from the transmission of a novel bat coronavirus [3] . for the sporadically endemic ebola virus, bats are suspected to be the natural reservoir, but this is still controversial [4] . vector-borne zoonoses caused by transmission of viruses through mosquitoes and ticks have also become a public health concern. the 1999 outbreak of west nile virus (wnv) that occurred in new york was caused by the transmission of the wnv among birds, horses and humans via mosquitoes [5] . similarly, severe fever with thrombocytopenia syndrome (sfts) was found to be due to a virus transmitted by ticks [6] .

</p>

</div>



<div id="info_a_edge10_113" class= "paragraph">

<h3>paragraph source</h3>

<p>avian flu h5n1 virus causes outbreaks of serious disease in poultry and wild birds in countries around the world. although it is not easily transmitted to humans, more than 500 human cases have been reported since 2003 in parts of asia, africa, eastern europe, and the middle east, with a case mortality rate of about 60%. transmission of avian flu h5n1to humans usually involves direct contact with infected birds, but if the virus mutates and becomes more easily spread from human to human (e.g., airborne by the respiratory route), this could start an influenza pandemic or worldwide outbreak of disease. avian influenza h7n9 virus continues to cause severe outbreaks of disease on poultry farms in china, in other asian countries, and in the us. so far, human cases reported from asia have not led to sustained human-to-human transmission. considering the possibility of an avian flu pandemic in the future, iata in cooperation with the cdc developed guidelines for airline personnel meeting passengers arriving from known outbreak areas, as well as for airline cleaning and maintenance crews who may potentially be exposed. all guidelines emphasize basic hygiene practices to prevent becoming ill, such as hand washing with soap and water or an alcohol-based hand gel, separation of any ill passengers and providing them with a surgical mask or tissues to cover the mouth and nose, use of disposable gloves when handling blood or body fluids, and prompt reporting of the illness to public health authorities. specialized personal protective equipment such as respirators or gowns is not recommended except in direct patient-care situations, although the use of surgical masks when around symptomatic individuals in public areas may be considered. specialized passenger screening techniques such as thermal screening to identify febrile passengers when pre-boarding that were utilized during the sars epidemic in some international locations are not being recommended at this time.

</p>

</div>



<div id="info_a_edge10_114" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<div id="info_a_edge11_115" class= "paragraph">

<h3>paragraph source</h3>

<p>the emergence of severe acute respiratory syndrome virus (sars)-cov-2 has shown once again that coronaviruses (covs) in animals are potential sources for epidemics in humans. porcine deltacoronavirus (pdcov) is an emerging enteropathogen of swine with a worldwide distribution. here, we implemented and described an approach to analyze the epidemiology of pdcov following its emergence in the pig population. we performed an integrated analysis of full genome sequence data from 21 newly-sequenced viruses, along with comprehensive epidemiological surveillance data collected globally over the last 15 years. we found four distinct phylogenetic lineages of pdcov, which differ in their geographic circulation patterns. interestingly, we identified more frequent intra- and inter-lineage recombination and higher virus genetic diversity in the chinese lineages compared with the us lineage where pigs are raised in different farming systems and ecological environments. most recombination breakpoints are located in the orf1ab gene rather than in genes encoding structural proteins. we also identified five amino acids under positive selection in the spike protein suggesting a role for adaptive evolution. according to structural mapping, three positively selected sites are located in the n-terminal domain of the s1 subunit, which is the most likely involved in binding to a carbohydrate receptor, while the other two are located in or near the fusion peptide of the s2 subunit and thus might affect membrane fusion. finally, our phylogeographic investigations highlighted notable south-north transmission as well as frequent long-distance dispersal events in china that could implicate human-mediated transmission. our findings provide new insights into the evolution and dispersal of pdcov that contribute to our understanding of the critical factors involved in coronaviruses emergence. significance: coronaviruses (covs) exhibit a pronounced propensity for interspecies transmission as illustrated by important emerging viruses in humans such as severe acute respiratory syndrome virus (sars)-cov and middle east respiratory syndrome virus (mers)-cov, as well as the recent sars-cov-2 that is causing a major human pandemic. a better understanding of the evolutionary process leading to viral emergence may help unravel the principles of cross-species transmission and therefore assist in preventing future introductions into new host populations. here, we examined the evolution and spread of an emerging coronavirus that likely transmitted from birds to pigs and caused sustained epidemics in asia and the united states. we integrated viral full genome sequence analysis and comprehensive surveillance data to examine the evolution of emerging porcine deltacoronavirus (pdcov), and in particular the factors driving ongoing circulation and recent cases in china. we found evidence for adaptive evolution of this emerging virus in the absence of vaccine immune pressure, with important amino acids under positive selection, as well as frequent inter- and intra-lineage recombination events in the asian lineages. breeding patterns and ecological environment could play an important role in swine coronavirus evolution, leading to differences in recombination and population diversity. using phylogeographic approaches, we further shed light on the dispersal history and dynamics of the china lineage of pdcov. our study provides a deeper understanding of the geographic dispersal of genetic lineages of pdcov in china and suggests that southern china acted as a source of transmission. modeling how the virus spread in china will allow the prioritization of areas for swine coronavirus surveillance in the future. these findings have important implications for preventing the emergence of coronaviruses in livestock and for controlling their spread in asian countries.

</p>

</div>



<div id="info_a_edge11_116" class= "paragraph">

<h3>paragraph source</h3>

<p>recently, many researchers have focused on bats as a major reservoir of novel viruses (54) . several novel viruses have been identified and/or isolated from bats, and some are related to human-pathogenic viruses, such as sars-cov (55) and mers-cov (56) . however, other animal species are also potential novel virus reservoirs; for example, lau et al. reported that the deltacoronavirus porcine cov hku15 (porcov hku15) can be transmitted from birds to pigs (57) . in addition, hcov oc43 is known to originate from bovine cov (58) . therefore, not only bats but also other animal species should be investigated for their potential as reservoirs or sources of emerging viruses. indeed, we found that the canine-derived a72 cell line permitted replication of type i fcov (fig. 3) . although the host receptors of type i fcov remain unclear, use of our reporter virus for fcov infection may provide insights on viral host jumping and emergence of novel viruses.

</p>

</div>



<div id="info_a_edge11_117" class= "paragraph">

<h3>paragraph source</h3>

<p>although the earliest detection of pdcov was in 2004 in mainland china, it is difficult to decipher the evolutionary relationships of pdcovs from different countries and to definitely infer the origin country of pdcov at this point with limited sequence data. how- fig. 4 . phylogenetic analysis of 50 global pdcov complete genome sequences. the tree was constructed using the distance-based neighbor-joining method of the software mega6.06. bootstrap analysis was carried out on 1000 replicate data sets, and values are indicated adjacent to the branching points. the us, korean, chinese, and thailand pdcovs are indicated by black, red, blue and purple font color, respectively. ever, the currently available sequence data support the inference that pdcov in south korea was from u.s. although the routes of introducing pdcov from u.s. to south korea have not been identified. not only for pdcov, other infectious diseases such as ped have emerged or reemerged in multiple countries in recent years. without knowing introduction routes of the pathogens between countries, it still remains a serious challenge to efficiently control spread of the emerging or reemerging diseases between countries or continents. how dcov originated in or was transmitted to domestic pigs remains unclear. in 2007, dong et al. (2007) reported a previously unrecognized coronavirus lineage detected from wild asian leopard cats and chinese ferret badgers from samples collected in southern china in 2006 . in 2009 , woo et al. (2009b reported identification of three novel avian coronaviruses in wild birds (bulbul cov hku11, thrush cov hku12, and munia cov hku13). these three novel avian covs and covs identified from asian leopard cats and chinese ferret badgers all belong to the newly established deltacoronavirus genus (woo et al., 2010 (woo et al., , 2012 . in 2012, identification of seven additional novel deltacoronaviruses in pigs and birds were reported (woo et al., 2012) . among deltacoronaviruses, pdcov is most genetically similar to covs in asian leopard cats based on available helicase, spike and nucleocapsid gene sequences (woo et al., 2012) . it is unknown how dcovs were transmitted between small mammals (such as asian leopard cats and chinese ferret badgers) and domestic pigs. a model of cov evolution has been proposed in which bats are the gene source of alphacoronavirus and betacornoavirus whereas birds are the gene source of gammacoronavirus and deltacoronavirus (woo et al., 2009a (woo et al., , 2012 . it is likely that dcovs are transmitted from wild birds to small mammals and/or domestic pigs. more surveillance and sequence data are needed in order to understand interspecies transmission of dcovs between wild birds and small mammals, between wild birds and domestic pigs, and/or between small mammals and domestic pigs.

</p>

</div>



<div id="info_a_edge12_119" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses are classified phylogenetically into four genera: alphacoronavirus, betacoronavirus, gammacoronavirus, and deltacoronavirus. alphacoronaviruses include human coronaviruses hcov-229e and hcov-nl63, porcine transmissible gastroenteritis virus (tgev), feline coronavirus (fcov), and ccov. four human coronaviruses hcov-oc43, sars-cov, hcov-hku1, mers-cov and bovine coronavirus (bcov), as well as crcov, are betacoronaviruses, but belong to different subgroups. phylogenetically, ccov is closely related to hcov-229e and hcov-nl63, while crcov is close to bcov and hcov-oc43. it has been suggested that crcov might share the same ancestor in this clade, or originated from a host transfer from bovine to canine (erles et al., 2003) .

</p>

</div>



<div id="info_a_edge13_63" class= "paragraph">

<h3>paragraph source</h3>

<p>middle east respiratory syndrome (mers) was first discovered in 2012 and has since caused more than 1700 cases and 700 deaths worldwide. 1,2 mers is caused by mers-coronavirus, which can be transmitted from camels to humans and from humans to other humans. 3, 4 the vast majority of cases to date have been reported out of saudi arabia, where mers is endemic due to frequent interactions between humans and dromedary camels. 2, 5 that said, the world health organization (who) estimates that about 75% of cases in saudi arabia are due to human-to-human transmission, although the vast majority of cases cause no secondary infections. [6] [7] [8] [9] worldwide mers-coronavirus transmission appears to be amplified in nosocomial (healthcare) settings, resulting in greater numbers of infections than would be otherwise expected given only direct (i.e., human-to-human) household or communitywide transmission. [7] [8] [9] [10] [11] this was exemplified during the 2015 mers outbreak in south korea, which was initiated by a local businessman returning home from a visit to the middle east and resulted in nearly 200 nosocomial cases soon thereafter ( figure 1 ). 12 throughout the entirety of the outbreak, the south korean government provided a wealth of publicly available information on case demographics, including case contact details crucial for reconstructing networks of infectivity. however, it remains unclear whether or not certain types of cases were more likely to cause direct (i.e., human-to-human) secondary transmissions than others, and if so, why.

</p>

</div>



<div id="info_a_edge13_64" class= "paragraph">

<h3>paragraph source</h3>

<p>m iddle east respiratory syndrome coronavirus (mers-cov) is a zoonotic viral respiratory illness caused by a novel betacoronavirus, which was first reported in saudi arabia in 2012. 1 since, mers-cov cases have been reported across the arabian peninsula, with occasional cases exported among travelers to other continents. 2 dromedary camels are a major reservoir host, in which the virus causes negligible disease. 3 the virus can spread from dromedary camels to humans, through direct or indirect contact, causing significant morbidity and mortality. 4 the clinical spectrum ranges from asymptomatic illness to septic shock and multiorgan failure. 5 as of january 2019, a total of 2298 laboratoryconfirmed human cases of mers-cov from 27 countries have been reported, including 811 associated deaths giving a fatality rate of 35.2%. eighty percent of cases have been reported from countries of the world health organization eastern mediterranean region. 2 human-to-human transmission of mers-cov has been described in several closed clusters of cases, including a small family cluster of mild disease. 5 two major healthcare-associated outbreaks due to superspreading events led to massive numbers of cases and excessive morbidity and mortality in several countries. 2, 6 currently, there has been no evidence of sustained human-to-human transmission. 2, 6 in oman, the first laboratory-confirmed case of mers-cov was reported in june 2013. 7, 8 up to march 2018, a total of 11 laboratory-confirmed cases have been reported sporadically. 5 ten (91%) cases were males with a mean age of 5217.7 years (range: 20-75 years). nine cases (82%) were primary and two cases (18%) were secondary. patients were predominately from the north al batinah governorate (average rate: 1 per 100 000 population) with a fatality rate of 9%. eight cases (73%) reported domestic camel exposure. uncontrolled diabetes mellitus was the most common comorbidity in eight cases (73%). no secondary cases were reported among healthcare workers. seroprevalence studies revealed mers-cov exposure among all sampled domestic camels across oman. 9, 10 between 27 january and 12 february 2019, a total of 13 additional human cases of laboratoryconfirmed mers-cov using real-time polymerase chain reaction (rt-pcr) were reported in oman. eight (61%) and four (39%) cases were residents of north al batinah and a'south sharqiyah governorates, respectively. 11 the first cluster of five females (four from the same family), reported on 27 and 28 january 2019 were residents of north al batinah. their mean age was 4210.8 years (range: 30-59 years). three of the five cases were probably secondary cases exposed to the index case who reported a history of direct contact with camels. none of the other cases had contact with camels. however, four cases resided on a farm where camels were kept, and the fifth case had a history of contact with one of the patient's with mers-cov at the hospital. two of the five cases died (40% fatality rate). uncontrolled diabetes mellitus and hypertension were the most common comorbidities reported among three of the cases.

</p>

</div>



<div id="info_a_edge13_65" class= "paragraph">

<h3>paragraph source</h3>

<p>multiple lines of evidence implicate dromedary camels in the emergence and transmission of mers-cov. firstly, mers-cov antibodies are highly prevalent in dromedary camels from across the arabian peninsula, north africa and eastern africa ( figure 1 and table 1 ). [29] [30] [31] 34, [69] [70] [71] mers-cov antibodies were detected in stored camel sera dating as far back as early 1990s, suggesting that mers-cov may have been circulating in dromedaries for over 20 years before it was first recognized as a cause of human infection. 32, 71, 72 the prevalence of mers-cov antibodies is significantly higher in older camels compared with those aged two years or less. 31, 32, 73, 74 interestingly, despite the high overall prevalence of mers-cov seropositivity in dromedary camels from kenya, no mers-cov antibodies were detected in those from the north eastern region where dromedaries have been raised largely in isolation from those in the rest of the country. 71 similarly, all dromedaries raised at the dubai central veterinary research laboratory, which had no contact with other camels, were seronegative for mers-cov. 75 also noteworthy is that 15 of 105 dromedary camels from the canary islands were seropositive for mers-cov. although the majority of camels in the herd were born and bred in the canary islands, the group included three camels that were imported from morocco. 29 in contrast, no mers-cov antibodies were detected in dromedary camels from australia, canada, the united states of america, germany, the netherlands or japan. 29, 34, 72, 75, 76 the high prevalence of mers-cov seropositivity in africa and the middle east suggests that animal movement has facilitated the transmission and circulation of mers-cov amongst dromedary camels in these regions. mers-cov antibodies have neither been found in mongolian or dutch bactrian camels, nor in south american camelids such as lamas, alpacas and guanacos. 29, 77 the second line of evidence is the reported detection of mers-cov by rt-pcr in oro-nasal and faecal samples from dromedary camels in multiple locations in the arabian peninsula ( figure 1 and table 2 ). 32, 70, 73, 74, [78] [79] [80] [81] the four of 110 dromedary camels in which mers-cov rna was detected in egypt were all imported from sudan or ethiopia for slaughter. 82 of particular concern is the particularly high rate of mers-cov rna detection in camels presented for slaughter in eastern saudi arabia and qatar; the latter was in the vicinity of a market in doha to which two prior human cases were linked. 78, 79 overall, mers-cov is more commonly detected and in higher viral loads in nasal swabs than in faecal samples. 32, 73, 74 another notable observation is that sequences from mers-cov strains isolated several months apart within the same location are often identical to one another, indicating that mers-cov circulation in dromedary herds is very stable. 73, 74 interestingly, the prevalence of mers-cov rna shedding is significantly higher in juvenile than in adult camels that had prior evidence of mers-cov seropositivity, indicating that animal re-infection is possible. 73, 74, 79, 83 the third level of evidence is the demonstration of active mers-cov infection in dromedary camels through documented rises in anti-mers-cov antibody titres. 73, 74, 85, 86 mers-cov was also detected by rt-pcr in symptomatic camels. 73, 74, 78, 83, 85, 86 dromedaries with active mers-cov infection exhibited symptoms such muco-purulent nasal and lachrymal discharge, cough, sneezing, fever and loss of appetite. 73, 78, 85, 86 it was initially argued that detection of mers-cov in camels by rt-pcr is not necessarily evidence of shedding of infectious virus and thus transmissibility of mers-cov between dromedaries and humans. 87, 88 however, several groups have since been able to isolate viable mers-cov in cell cultures of nasal and faecal samples from dromedary camels. 73, 74, 80, 84, 89 moreover, the potential infectiousness of mers-cov recovered from dromedary camels was evident by its capability to cause ex-vivo infection in human respiratory cells and human hepatoma cells (huh-7) . 80, 90 successful mers-cov cultures usually coincide with corresponding high viral loads in the same specimens. 73, 80, 82, 84 the fifth level of evidence is the successful experimental mers-cov infection of dromedary camels with resultant mild clinical infection manifesting as fever and rhinorrhea. 91 three mers-cov seronegative adult dromedary camels were inoculated via intranasal, intra-tracheal camels. 32, 73, 74, 78 in one prospective study, mers-cov detection by rt-pcr in dromedary camels was highest in the period between the months of november and january, coinciding with the dromedaries' calving season. 78 this, along with the observed increased incidence of human mers-cov infection during the period between march and may, suggests that juvenile camels are an important source of new infections in camels and potentially hum ans. 73, 74, 78 mers-cov rna prevalence data also indicate that animal movement may facilitate the introduction of mers-cov into herds of dromedaries. for example, phylogenetic analysis of mers-cov strains that were detected in a herd of dromedary camels in dubai showed that they were closely related to those circulating in eastern saudi arabia, where some of the camels had recently grazed. 74 in another example, a group of camels imported from oman was screened by rt-pcr on arrival in uae and was found positive for mers-cov. 83 moreover, mers-cov strains isolated from dromedary camels in eastern saudi arabia were phylogenetically related to those isolated from camels several hundred kilometres away in buraidah, central saudi arabia. 73 most significantly, phylogenetic analyses of partial and whole mers-cov genomes from dromedary camels show that they are clustered within human isolates, supporting possible camel-human intertransmission. 32, 70, 73, 74, 78, 80, 82, 84 rt-pcr was positive in co-worker were diagnosed with laboratory-confirmed mers-cov infection. mers-cov was detected by rt-pcr in five of their 14 camels. 89 alignment of six mers-cov genomic fragments, covering 4.2 kb of the mers-cov genome, from the camels and the human cases showed that they differed by one nucleotide each in orf1a and orf4b regions. 89 in november 2013, a 43-year-old man died in a hospital in jeddah, saudi arabia, with severe mers-cov infection. he had owned a herd of nine dromedary camels, some of which were reported to have had recent respiratory symptoms. mers-cov was detected by pcr and culture of nasal specimens from one camel. 85, 86 partial mers-cov sequence of 4.6 kb from the camel and the linked human isolates differed in only two positions. 85 whole mers-cov genome sequences obtained from viral cultures of the human and camel isolates were 100% identical. 86 importantly, 4-fold rise in mers-cov antibody titres was documented in the camels, indicating that active mers-cov infection was probably circulating in the dromedary herd. 85, 86 later, rising mers-cov antibodies were documented in the patient, suggesting that mers-cov infection was transmitted from the camels to the human and not vice versa. 86 in may 2015, mers-cov was detected in eight asymptomatic dromedary camels at entry into uae from oman. two asymptomatic men, aged 29 and 33 years, who were in contact with the camels were found to be positive for mers-cov rna in their respiratory samples. 83 partial sequences of mers-cov spike, orf3-4a and nucleocapsid regions from the human and linked camels were and conjunctival routes with a total dose of 10 7 50% tissue culture infective dose (tcid 50 ) of mers-cov. clinical symptoms appeared within 2 days and persisted for up to 2 weeks. submucosal inflammation and necrosis were evident in the upper and lower respiratory tracts, but not the alveoli. mers-cov antibodies were detected within 14 days of inoculation. shedding of infectious mers-cov in oral and nasal secretions, as determined by plaque assay titres, continued for up 5 and 7 days, respectively. mers-cov was detectable by quantitative pcr for up to 35 days. 91 under certain condition, mers-cov can survive on plastic and steel surfaces for up to 30 hours. 92 moreover, mers-cov rna can be detected in milk expressed from mers-cov infected camels. 93 whether mers-cov is secreted in camel milk or it is contaminated from the animal's other body secretions is probably immaterial as viable mers-cov can be recovered from dromedary camel milk stored at 22 degrees celsius for up to 48 hours. 94 it is therefore possible to envisage that in the absence of appropriate precautions, the environment surrounding a mers-cov infected camel can become extensively contaminated with viable, potentially infectious virus.

</p>

</div>



<div id="info_a_edge13_66" class= "paragraph">

<h3>paragraph source</h3>

<p>a 2018 publication noted that hcov-229e, a primarily non-lethal coronavirus responsible for a proportion of upper and lower respiratory tract infections, can be transmitted from camels to humans similar to mers transmission, thus implicating dromedary camels as a possible reservoir of coronaviruses (corman et al. 2018) . woo et al. (2014) identified a novel betacoronavirus uae-hku23 or dromedary camel coronavirus dccov in the united arab emirates that was detected in 52% of tested camel serum samples.

</p>

</div>



<div id="info_a_edge13_67" class= "paragraph">

<h3>paragraph source</h3>

<p>coronavirus infections are endemic and ubiquitous in feline populations. two viral types, type 1 and 2, are distinguished and both of them could well sustain themselves in the cat reservoir [15, 43] . both have been shown to have worldwide distribution, with the seropositivity rate up to 90% among animal shelter populations and in multi-cat households [20, 44] . the majority of natural infections are caused by type 1 fcovs, while in the field type 2 fcovs are less common and mainly occur in asia [15, 28, [45] [46] [47] . covs are generally considered to be host-specific; however, cross-species transmission does occur which may lead to incidental infections like the spillover of mers-cov from dromedary camel to humans, where humans function as an incidental and ultimately dead-end host [4] . but covs might also adapt to the new host exemplified by the animal origin of all four endemic human covs (hcov-oc43, hcov-nl63, hcov-229e and hcov-hku1) [8] [9] [10] [11] . whereas in cats infections with fcov are well recognized, studies regarding possible natural infections with other animal and human coronaviruses are lacking to the best of our knowledge. knowing the genetic variability of coronaviruses and the use of orthologous receptors by non-feline covs, studies on cross-species transmission are desirable. this may provide insight regarding whether cross-species transmission does occur. in the present study we used the highly immunogenic s1 antigens to screen cat sera for the presence of antibodies against feline and non-feline coronaviruses, as a first indication of possible infections with these viruses.

</p>

</div>



<div id="info_a_edge13_68" class= "paragraph">

<h3>paragraph source</h3>

<p>although sars and mers have a higher mortality rate, sars-cov-2 infection spreads much more rapidly . as of may 2020, more than 4,000,000 confirmed infections were reported in 216 countries, including over 300,000 deaths (who, 2020) . mers-cov, sars-cov, and sars-cov2 were suggested to originate from bats and most likely serve as a reservoir host for these viruses (ge, et al., 2013; haagmans, et al., 2014; li, et al., 2005; memish, et al., 2013; zhou, et al., 2020) . detailed investigations of the zoonotic origin of human covs indicate that sars-cov was transmitted from palm civets to humans and mers-cov from dromedary camels to humans (guan, et al., 2003; haagmans, et al., 2014; kan, et al., 2005) . however, the intermediate host for zoonotic transmission of sars-cov-2, linked to a wet animal market in wuhan, is still under investigation (walls, et al., 2020; ye, et al., 2020) .

</p>

</div>



<div id="info_a_edge13_69" class= "paragraph">

<h3>paragraph source</h3>

<p>the actual emergency in the context of the sars-cov-2 crisis is successful drug repurposing, [34] for ab initio drug discovery and its lengthy phases will take 10 + years and cannot contribute to a rapid jugulation of the current outbreak -vaccine development being the rational alternative. drug repurposing of known antivirals does (unfortunately) not require high-throughput calculations or screen-ing because approved antivirals are preciously few. [25] clearly, mankind allotted sufficient research resources only against threatening, widely spread and persistent viruses. until 2020, coronaviruses did not reunite all the three conditions. for example, the middle-east respiratory syndrome (mers) did not vanish but has no pandemic potential as it only spreads from camel to human: it is de facto a neglected disease, which at only 70 entries in chembl did not even make it into the pool of herein studied chemical spaces. if the long and costly drug discovery campaign would have been pursued at high priority, although the sars threat disappeared and mers is confined to remote rural regions of the world, we would have certainly been in a better situation to address sars-cov-2 on the basis of a larger and better suited portfolio of repurposeable antivirals. following long-time goals, not driven by immediate danger but by the rational decision to anticipate further threats, has never been a strength of homo sapiens -although it is the only mammal as successful in spreading over terra as viruses and bacteria.

</p>

</div>



<div id="info_a_edge13_71" class= "paragraph">

<h3>paragraph source</h3>

<p>the human coronaviruses are known major pathogens of respiratory diseases that are highly infectious and communicable. for instance, the sars-cov can be transmitted directly from civets to humans; the mers-cov can be transmitted from dromedary camels to human. there is also evidence of human to human transmission. as earlier stated, sars-cov-2 possesses certain similar characteristics as sars-cov. when an individual comes in contact with a sars-cov-2, the virus targets major routes where it can easily propagate to get to target cells and mediate infection. coronaviruses are respiratory viruses; thus, their first major target is the epithelial cells of the respiratory tract where they bind to receptors (the ace-2) [12] . by binding to the ace-2 expressed on the respiratory epithelial cells, the sars-cov/sars-cov-2 not only mediates infection but also facilitates their replication and transmission [2, 12, 15, 16] . this marks the genesis of the pathology of the virus.

</p>

</div>



<div id="info_a_edge13_72" class= "paragraph">

<h3>paragraph source</h3>

<p>the coronavirus outbreaks had significant impacts on the previously small and cohesive coronavirus field. the discovery that a coronavirus is the etiologic agent of the sars epidemic shocked coronavirologists, and we all began to work on the new sars-cov using all we had learned previously. as is now well known, but was not clear at the time, sars disappeared after 6 or 7 mo with 8,069 infections and 774 deaths. as the virus disappeared, the number of people working on coronaviruses diminished. following the sars epidemic, two other human coronaviruses were identified, nl63 (croup, bronchiolitis; pyrc et al., 2004 ) and hku1 (pneumonia; lau et al., 2006) . another notable finding was the discovery that bats were the hosts for many sars-like coronaviruses (li et al., 2005; lau et al., 2005) . when the more deadly mers-cov emerged in 2012, this rekindled the widespread interest in coronaviruses. however, there were important differences in the emergence of mers-cov, as it was found to have a reservoir in the intermediate animal, the camel, and continues to this day to spread from camel to human. maybe this should have warned us that coronaviruses may emerge in different ways and spread with different patterns, as we have observed from sars-cov-2.

</p>

</div>



<div id="info_a_edge13_73" class= "paragraph">

<h3>paragraph source</h3>

<p>because of the resemblance of sars-cov-2 to sars-like coronaviruses, bats are likely to act as natural reservoir hosts of the progenitor of sars-cov-2 (andersen et al., 2020) . it is established that sars-cov was transmitted from palm civet to humans (wang et al., 2006) and mers-cov from dromedary camels to humans, yet the zoonotic origin of sars-cov-2 is still unclear. the genome sequence analysis shows that sars-cov-2 has the closest relative sars-like slcov/ratg13, found in horseshoe bat (rhinolophus affinis) from yunnan, china (zhou et al., 2020) . sars-cov-2 has 96.2% overall genome sequence identity to sars-like coronavirus slcov/ratg13 (zhou et al., 2020) . therefore, the natural reservoir of sars-cov-2 could be the horseshoe bat. recent evidence suggests pangolins as host candidates (lam et al., 2020) , however, the host pangolins are not firmly determined. not all bat sars-like cov can infect humans. for example, bat coronavirus, swine acute diarrhoea syndrome coronavirus (sads-cov) found in 2017 (zhou et al., 2018) , caused millions of piglet deaths, but no human cases. we may postulate that sars-cov-2 evolved from its progenitor through mutations and evolutionary fitness in the host-shift and adapting. sars-cov-2 are fast-evolving pathogens that continuously undertake mutations in the generations of infection of the host (yin, 2020; wang et al., 2020; korber et al., 2020) . this fact suggests that sars-cov-2 had evolved mutations in critical proteins and genome structures prior to establishing human infection. to understand how sars-cov-2 jumps from animals to adaptively infect humans is important to the surveillance of virus evolution and diversity, therefore ultimately controlling the covid-19 pandemic and preventing future sars-like outbreaks.

</p>

</div>



<div id="info_a_edge13_74" class= "paragraph">

<h3>paragraph source</h3>

<p>another novel human coronavirus, mers-cov was isolated from a patient with acute pneumonia in 2012 in saudi arabia. as of march 2015, mers-cov infection has led to 1084 cases and 439 deaths reported in multiple countries. the fatality of mers-cov is considerably higher than sars-cov, approaching 30%. mers-cov may be originating from bats due to its high sequence homology to the bat virus. it is speculated the virus spreads from bats to human via dromedary camel (fig. 21.11 ). almost all cases have been linked to saudi arabia. although human-to-human transmission seems to be inefficient, it has been shown to spread between people who are in close contact.

</p>

</div>



<div id="info_a_edge13_75" class= "paragraph">

<h3>paragraph source</h3>

<p>cases have been detected in health care workers, like with sars (severe acute respiratory syndrome) and mers (middle east respiratory syndrome)-corona virus outbreaks (grant et al., 2019; wang et al., 2020) . sars caused a major outbreak in the beginning of 2000s mostly in asia, the source was civet cats. mers transmits from camels to humans in middle east. the source for both may be originally bats (cui et al., 2019) . the novel sars-cov-2 is closely phylogenetically related to sars coronavirus with 89% similarity. the genetic code for sars-cov-2 was sequenced and published in a record time and relation to other coronaviruses known in days. the authorities in china have taken action in a timely manner under the pressure from (social) media and with the help of world health organization.

</p>

</div>



<div id="info_a_edge13_76" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses (covs) are well-known as a major cause of the common cold in humans by mild infection in the upper respiratory tract. covs have gained attention since the outbreak of severe acute respiratory syndrome (sars) in 2002-2003 when sars-related cov (sars-cov) was identified (anderson et al. 2004) . sars-cov is the cause of human lower respiratory tract diseases such as bronchitis and pneumoniae (holmes 2003; girard et al. 2005; peiris et al. 2004 ). in 2012, one more cov was isolated in saudi arabia and named mers-cov. this virus belongs to lineage c of beta-coronaviruses (zaki et al. 2012 ). according to a world health organization (who) survey, 2143 cases of mers-cov infections were reported, and 750 people were killed in 27 countries by march 2018. in south korea, 186 people were infected by mers-cov, and 36 people died in 2015. the symptoms of the mers-cov infection in south korea included fever, cough, upper respiratory tract signs, lower respiratory tract distress with lymphopenia and elevated liver enzymes, acute respiratory distress syndrome, and multiple organ system failure (zhang et al. 2016) . mers-cov is reportedly transmitted to humans generally from bats or dromedary camels, but the exact route of infection has not yet been identified (who mers-cov research group 2013; yin and wunderink 2018; mackay and arden 2017) .

</p>

</div>



<div id="info_a_edge13_77" class= "paragraph">

<h3>paragraph source</h3>

<p>mers-cov has a wide species tropism, as the virus can replicate in a variety of mammalian cells of different origins [37] . this suggests that the virus originated from animals. as is the case with several beta-covs, bats are the common reservoir from which the virus jumped to humans through intermediate hosts [38] . mers-cov is thought to be transmitted to humans directly from bats or through an intermediate host, most likely dromedary camels [39] [40] [41] . many studies have sought to define the animal reservoir, intermediate host(s), and the possibility of human-to-human transmission. although bats are considered the ancestral reservoir of covs [42] , camels are important for the maintenance and diversification of mers covs, and they are the source of human infections. it has been estimated that camel exposure resulted in 12% of mers-cov infections [43] . a survey of mers-cov-infected individuals from 20 hospitals in saudi arabia revealed that some had direct contact with camels, while others had likely acquired the virus from infected humans [44] . the possibility of human-to-human transmission was also supported by the presence of hospitalassociated outbreaks in several countries, including saudi arabia [30, 33, [45] [46] [47] [48] .

</p>

</div>



<div id="info_a_edge13_78" class= "paragraph">

<h3>paragraph source</h3>

<p>several coronaviruses infect the human respiratory tract, and usually cause mild disease; however, the beta coronaviruses severe acute respiratory syndromeassociated coronavirus (sars-cov) and middle east respiratory syndrome-associated coronavirus (mers-cov) cause severe zoonotic respiratory disease. sars emerged in 2002 in guangdong province, china, and caused an epidemic leading to 8096 cases and 774 deaths globally in more than 25 countries across five continents, but was contained through public health interventions. mers-cov transmits from dromedary camels to humans, sometimes leading to clusters of human-to-human transmission, especially within health-care facilities. to date, within health-care facilities, with 2519 cases, with 866 deaths across 27 countries, have been confirmed as of january, 2020. 1 in december, 2019, the novel coronavirus sars-cov-2 caused an outbreak of respiratory illness (coronavirus disease 2019; in wuhan, china. within 5 months, the disease burden and fatalities have surpassed both sars and mers, with more than 2 million confirmed cases and more than 150 000 deaths reported globally, as of april 19, 2020. 2 who declared this outbreak a pandemic on march 11, 2020. although the virus appears to be more transmissible than either sars or mers, disease severity is variable-from asymptomatic to fatal-and case fatality appears to be substantially lower than both sars and mers. 3 modes of transmission and pathogenesis have been key knowledge gaps. the virus is assumed to be primarily transmitted by large respiratory droplets, but there has been no direct evidence for this hypothesis. identifying the organs and cell types that are permissive to implantation and virus replication will help to understand the portals by which the infection can be established. to our knowledge, to date there has been only one autopsy study reporting a patient dying after testing positive for sars-cov-2 using needle core samples and one histopatho logical study on two patients undergoing lobectomy. 4, 5 although these studies have provided information on the histopathology of covid-19, they did not include characterisation of sars-cov-2 virus trop ism by immunohistochemistry, and therefore do not shed light on virus tropism in the early stages of the infection process. 4, 5 thus, data from experimental infection of ex-vivo cultures of the respiratory tract are needed.

</p>

</div>



<div id="info_a_edge13_79" class= "paragraph">

<h3>paragraph source</h3>

<p>previously, the severe acute respiratory syndrome-coronavirus (sars-cov) and the middle east respiratory syndrome-coronavirus (mers-cov) have been known to affect humans. outbreaks of respiratory disease caused by these viruses seem to have originated in animals before moving into other hosts like humans. mers-cov was found to be transmitted from arabian camels to humans, whereas sars-cov was transmitted from civet cats to humans. sars-cov-2 seems to have originated from bats and first reports of cases were from wuhan, hubei province in china, suggesting an animal-to-person spread from a live animal market. the virus then spread outside hubei and subsequently, to the rest of the world via human transmission. several countries have now reported community spread. the world health organization (who) declared coronavirus disease as a pandemic on march 11, 2020 [2] .

</p>

</div>



<div id="info_a_edge13_80" class= "paragraph">

<h3>paragraph source</h3>

<p>abstract coronaviruses (covs) are enveloped, positive-strand rna viruses with characteristic spike glycoproteins that project outward like the rays of the sun (corona  latin for crown), when visualized by electron microscopy. cov are classified, together with the toroviruses, in the family coronaviridae and the order nidovirales. all nidoviruses have a common genome organization and generate a nested set (nido  latin for nest) of 3 co-terminal mrnas. covs have been isolated from a variety of species, including birds, livestock, domestic animals, and humans. cov infections can cause respiratory, gastrointestinal, and neurologic disease, depending on the strain of the virus and the site of infection. importantly, covs have been shown to cross species barriers and have emerged from animal reservoirs to infect humans and cause severe disease. the cov responsible for an outbreak of severe acute respiratory disease (sars-cov) in 200203 likely originated as a bat coronavirus which, during replication in an intermediate host (such as the palm civet), evolved to be able to infect humans efficiently. sars-cov infected over 8000 people with approximately 10% mortality rate before it was controlled by public health measures of isolation of infected individuals and contacts. middle east respiratory syndrome cov (mers-cov), first reported in 2012, is likely transmitted from camels to humans with potentially fatal consequences. to date, there are no approved vaccines or direct acting antiviral drugs to combat coronavirus infections in humans. the emergence or re-emergence of covs from animal reservoirs is a potential concern for public health.

</p>

</div>



<div id="info_a_edge13_81" class= "paragraph">

<h3>paragraph source</h3>

<p>the covid-19 outbreak is caused by a newly identified member of a large family of viruses known as coronaviruses (name deriving from the latin word corona, meaning crown) . according to the world health organization (who) [1] this family of viruses causes illnesses ranging from the common cold to more severe diseases such as severe acute respiratory syndrome (sars) and the middle east respiratory syndrome (mers). these viruses were originally transmitted from animals to people. sars, first recognized in mid-march 2003 [2] , was transmitted from civet cats to humans, while mers, first detected in saudi arabia in 2012 [3] , moved to humans from a type of camel. the novel coronavirus, identified by chinese authorities on january 7 is a new strain that has not been previously identified in humans. the sars-cov-2 virus, as it has been named by the who, seems to be closely related to bat and pangolin coronaviruses, but a final confirmation has not yet been given [4] . people may catch covid-19 by close contact with infected people, via small droplets produced when they cough, sneeze, talk, or by touching a contaminated surface and then their face. according to the who, symptoms of infection include fever, cough, shortness of breath and breathing difficulties. notwithstanding the fact that infected patients can also be asymptomatic, in the more severe cases, the virus can lead to pneumonia, multiple organ failure and death. the number of fatalities from the new coronavirus has overwhelmingly overtaken the toll of the sars outbreak but its percentage remains considerably lower compared to the 10 percent of sars and 30 percent of mers.

</p>

</div>



<div id="info_a_edge13_82" class= "paragraph">

<h3>paragraph source</h3>

<p>infection with mers-cov, in its initial description, resembled "sars-like" illness (chan et al., 2015a,b) . further analysis of the epidemiological, virological, and clinical aspects of mers-cov and sars-cov revealed important differences between the two viruses. identifying unique aspects of mers-cov helped to explain how the epidemic evolved and the steps that could be taken to prevent its spread (chan et al., 2015a,b) . serological studies have indicated that most dromedary camels in africa and the middle east, but not other animals such as sheep, goats, and cows, were seropositive for mers-cov . moreover, seroprevalence in dromedary camels appears to vary, with high rates reported in animals from egypt, ethiopia, nigeria, and sudan and lower rates in animals from tunisia (ali et al., 2017) . intriguingly, dromedaries from australia, canada, the united states, germany, netherlands, and japan have been reported to be seronegative for mers-cov (omrani et al., 2015) . importantly, population-based seroepidemiologic studies indicated that the seroprevalence of the virus was several folds higher in people who have been exposed to camels compared to those in the general population ( accumulating serologic and molecular evidence indicates that the virus in dromedaries is genetically similar to mers-cov in humans, supporting the notion that dromedary camels could be the potential source of infection to human memish et al., 2014a,b,c; sabir et al., 2015) . indeed, mers-cov antibodies have been isolated in dromedary camels across the arabian peninsula, north africa, and eastern africa dating from as far back as the 1990s (milne-price et al., 2014; omrani et al., 2015) . this finding suggests that mers-cov may have been circulating in dromedaries for over 20 years before it was first recognized as a cause of human infection (aly et al., 2017) . in a recent study, a fatal case of mers-cov infection was reported in an individual who had direct contact with a dromedary camel (azhar et al., 2014) . sequence analysis of the virus isolated from the case and the camel was identical, clearly indicating that mers-cov can indeed be transmitted from camels to human (azhar et al., 2014) . it appears that active infection with release of the virus in nasal secretions, particularly during the incubation period, is important for the transmission of the virus to humans . where and how the camels acquired the infection remains unknown. it has been hypothesized that bats could be the potential source (fig. 8.3 ) (anthony et al., 2017; mohd et al., 2016; omrani et al., 2015) . indeed, mers-cov-like viruses have been identified in certain species of bats (anthony et al., 2017; woo et al., 2006) . the bats are present in most parts of the world and often infected with various zoonotic viruses. thus it is plausible that at some point in the past, camels acquired the infection from bats, leading to a sustained infection in the camel population (fig. 8.3) . mers-cov rna has been identified in the milk, nasal secretion, and feces of dromedary camels (omrani et al., 2015) . since camels and humans are often in close contact, particularly in the arab gulf states, humans would be at increased risk of contracting the virus from actively infected animals (mackay and arden, 2015; reusken et al., 2015) . indeed, mers-cov seropositivity in shepherds and those working in slaughterhouses in saudi arabia has been reported to be an order of magnitude higher than in the general population (arabi et al., 2017) . although possible, no evidence currently exists to support the transmission of mers-cov from bats to humans directly. what is certain is that transmission of the virus can occur from camels to humans, but the process is still not fully understood (al hammadi et al., 2015; memish et al., 2014a,b,c) . one possibility is that some species of covs from camels and humans could recombine leading to the emergence of a new virus that can infect both, camels and human (sabir et al., 2015) .

</p>

</div>



<div id="info_a_edge13_83" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses that are closely related to sars-cov are typically found in bats (li et al. 2005) . there is evidence that sars-cov originated in bats in china and reached humans most probably after jumping to an intermediate host, the civet (paguma larvata) (reviewed by wang and eaton 2007) . similarly, mers-cov is endemic in dromedary camels in the middle east, from which it can be transmitted to humans, but it may also have originated in bats (cui et al. 2019) . the mers epidemic is ongoing, and as of december 2019, 2468 cases have been reported (baharoon and memish 2019) and the mortality rate has been as high as 42% (mohd et al. 2016) . at the other end of the virulence spectrum are coronavirus species that cause common colds in humans with relatively mild symptoms (reviewed by su et al. 2016) .

</p>

</div>



<div id="info_a_edge13_84" class= "paragraph">

<h3>paragraph source</h3>

<p>middle east respiratory syndrome coronavirus (mers-cov) is a major emerging zoonotic infectious disease. since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. to date, there are no efficacious drugs and vaccines against mers-cov, increasing its potential to cause a public health emergency. a critical step in the life cycle of mers-cov is the fusion with the host cell with its spike (s) protein as main determinant of viral entry. proteolytic cleavage of s exposes its fusion peptide (fp), which initiates membrane fusion. previous studies on the related severe acute respiratory syndrome coronavirus (sars-cov) fp have shown that calcium (ca2+) plays an important role for fusogenic activity via a ca2+ binding pocket with conserved glutamic acid (e) and aspartic acid (d) residues. sars-cov and mers-cov fp share a high sequence homology and here, we investigated whether ca2+ is required for mers-cov fusion by substituting e and d residues in the mers-cov fp with neutrally charged alanines. upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudo-particles (pps) on host cells without and with ca2+. our results demonstrate that intracellular ca2+ enhances mers-cov wt pps infection by approximately two-fold and that e891 is a crucial residue for ca2+ interaction. electron spin resonance revealed that this enhancement could be attributed to ca2+ increasing mers-cov fp fusion-relevant membrane ordering. intriguingly, isothermal calorimetry titration showed that mers-cov fp binds one ca2+, as opposed to sars-cov fp which binds two. our data suggests that there are significant differences in fp-ca2+ interactions of mers-cov and sars-cov fp despite their high sequence similarity and that the number of ca2+ ions interacting with the fp has implications on the fusion dynamics of the virus.

</p>

</div>



<div id="info_a_edge13_85" class= "paragraph">

<h3>paragraph source</h3>

<p>while there were intensive studies on coronavirus research over the past 5 years, limited reports are available on the pathogenesis of mers-cov. recent post-mortem histopathological studies revealed that the localization of viral particles in the pulmonary and extra pulmonary tissue of a 33-year-old male patient of t cell lymphoma, who acquired mers-cov infection (ng et al., 2016; alsaad et al., 2018) . the histopathological analysis of the biopsies were collected from lung, brain, liver, heart, kidney, and skeletal muscle demonstrated that there were pulmonary diffuse alveolar damage, necrotizing pneumonia, acute kidney injury, portal and lobular hepatitis and myositis with muscle atrophic changes. further, the viral particles were found to be localized in the pulmonary macrophages, pneumocytes, renal proximal figure 3 </p>

</div>



<div id="info_a_edge13_86" class= "paragraph">

<h3>paragraph source</h3>

<p>the middle east respiratory syndrome (mers) is caused by a zoonotic respiratory pathogen, coronavirus, which causes a nonspecific respiratory illness that was first reported in saudi arabia in 2012 [1] . following reports of mers-cov infections from the ara-bian peninsula, cases were likewise reported from travelers visiting other continents [2] . dromedary camels, in which the virus does not cause disease, are believed to be the major host reservoir [3, 4] . the virus can spread from dromedary camels to humans through direct or indirect contact, causing significant morbidity and mortality [4] . the clinical spectrum ranges from asymptomatic illness to septic shock, multi-organ failure and death in severe cases [5] . evidence suggests that the average incubation period in an infected human host is 5.5-6.5 days with a maximum of 10-14 days [6] .

</p>

</div>



<div id="info_a_edge13_87" class= "paragraph">

<h3>paragraph source</h3>

<p>middle east respiratory syndrome coronavirus (mers--cov) is an emerging zoonotic agent that was first isolated in 2012 from a patient hospitalized in saudi arabia [1] , and has since infected over 2200 people with a 36% case fatality ratio [2] . after an incubation period of 2-14 days [3] , the virus causes a disease (middle east respiratory syndrome, or mers) characterized by fever, cough, and shortness of breath, which commonly leads to pneumonia and respiratory failure [4] . the virus circulates silently in dromedary camels, the only known reservoir species and zoonotic source of spillover to humans [5] . however, not all primary human cases have documented exposure to dromedaries or their products, such as milk and meat. although human-to-human community-acquired infections have not been documented, there is evidence that asymptomatic infections of mers-cov exist and could be a source of community transmission [6] . zoonotic spillover from dromedary camels to humans has been documented in the arabian peninsula [7] . subsequent secondary cases can occur after unprotected contact with family members and within healthcare facilities once the primary case seeks medical assistance [8] . while the sizes of mers-cov outbreaks have decreased thanks to improved infection control in healthcare settings in affected countries, cases continue to be reported regularly, especially in saudi arabia, where surveillance is strong [9] . in order to further reduce cases and prevent human outbreaks, a better understanding of zoonotic transmission of mers-cov is needed. a deeper understanding of the epidemiology of primary human cases can inform evidence-based interventions at the level of the community at the animal-human interface.

</p>

</div>



<div id="info_a_edge14_120" class= "paragraph">

<h3>paragraph source</h3>

<p>in december 2019, covid-19 pandemic emerged in wuhan china. it became a global health problem in a very short time because of it contact-transferred behaviours 1, 2 . covid-19 is caused by a virus known as severe acute respiratory syndrome coronavirus 2 or also sars-cov-2 3 . a large family of these virus are causing different kind diseases such as cold, middle east respiratory syndrome and severe acute respiratory syndrome. the new addition to coronavirus family covid-19 is discovered in 2019, which has never been detected in humans. covid-19 is known as zoonotic disease because it is transferred from animal to human such as bat 4 , similar to sars-cov virus which is contaminated through cat and mers-cov from dromedary 5 . it is presumed that respiratory transmission and physical contact is cause to spread covid-19 rapidly. according to world economic forum , people with no symptoms or mild symptoms are the main reason of spreading the epidemic. one our of four subject shows no symptoms of covid-19, although he/she is suffering with the disease 6 .

</p>

</div>



<div id="info_a_edge14_121" class= "paragraph">

<h3>paragraph source</h3>

<p>previously, the severe acute respiratory syndrome-coronavirus (sars-cov) and the middle east respiratory syndrome-coronavirus (mers-cov) have been known to affect humans. outbreaks of respiratory disease caused by these viruses seem to have originated in animals before moving into other hosts like humans. mers-cov was found to be transmitted from arabian camels to humans, whereas sars-cov was transmitted from civet cats to humans. sars-cov-2 seems to have originated from bats and first reports of cases were from wuhan, hubei province in china, suggesting an animal-to-person spread from a live animal market. the virus then spread outside hubei and subsequently, to the rest of the world via human transmission. several countries have now reported community spread. the world health organization (who) declared coronavirus disease as a pandemic on march 11, 2020 [2] .

</p>

</div>



<div id="info_a_edge15_122" class= "paragraph">

<h3>paragraph source</h3>

<p>in the few last years, covs of the dog have undergone a genetic evolution mainly through accumulation of point mutations and deletions in some genomic regions rather than through recombination events. changes in the group-specific genes located between orfs 2 and 4 have been postulated to be responsible for increased virulence of ccov type ii strains (sanchez-morgado et al., 2004; decaro et al., 2007d) . point mutations in the s gene have been also suggested to be involved in the emergence of pantropic ccov (decaro et al., 2007d) . crcov emerged likely as host variant of a bcov strain that was able to spread from cattle to dogs . a different origin could be hypothesised for ccov type i. based on the close genetic relatedness to fcov type i in the s gene, a potential recombinant origin of this ccov genotype had been suggested (pratelli et al., 2003a) . however, the recent identification of orf3 in the genome of ccov type i should lead to reconsider its origin. in fact, since remnants of orf3 were found in some type ii ccovs, the most likely scenario is that ccov has lost this gene during its evolution, probably because orf3 is not indispensable for viral replication (lorusso et al., 2007) . consequently, an ancestral carnivore cov could be thought to have generated both ccov genotypes and fcov type i as well.

</p>

</div>



<div id="info_a_edge16_41" class= "paragraph">

<h3>paragraph source</h3>

<p>after the sars epidemic, we started to focus on cov biodiversity, genomics and phylogeny and built up an evolutionary map of cov evolution. before 2003, there were less than 10 covs with complete genomes available, which include two human covs, human coronavirus 229e (hcov-229e) and human coronavirus oc43 (hcov-oc43). by september 2012, the number of covs with complete genomes sequenced had tripled. it includes two additional human covs, human coronavirus nl63 (hcov-nl63) and human coronavirus hku1 (hcov-hku1). 2,3 traditionally, covs were classified into groups 1, 2 and 3. in 2011, the coronavirus study group of the international committee for taxonomy of viruses has re-classified these three groups of covs as three genera, alphacoronavirus, betacoronavirus and gammacoronavirus; and we have discovered a fourth genus of cov, deltacoronavirus, which includes at least nine avian covs and a porcine coronavirus hku15. 4, 5 within the betacovs, they are further subclassified into group a, including hcov-hku1, hcov-oc43, bovine coronavirus (bcov), sable antelope coronavirus, giraffe coronavirus, equine coronavirus, porcine hemagglutinating encephalomyelitis virus, murine hepatitis virus, rat coronavirus and rabbit coronavirus hku14 (rbcov hku14); 6 group b, including the human and civet sars-related covs (sarsr-cov) and sars-related rhinolophus bat coronavirus (sarsr-rh-batcov); 7,8 group c, including tylonycteris bat coronavirus hku4 (ty-batcov hku4) and pipistrellus bat coronavirus hku5 (pi-batcov hku5) we discovered in 2006; 9,10 and group d, including rousettus bat coronavirus hku9 (ro-batcov hku9). 10, 11 in addition to ty-batcov hku4 and pi-batcov hku5, other group c bat betacovs should also be present, but their complete genome sequences are not available. 12, 13 based on the covs discovered, we have constructed a model of cov evolution, with evidence supporting that bat covs are the gene source of alphacovs and betacovs and avian covs are the gene source of gammacovs and deltacovs. 5 all these works have laid down an evolutionary map for rapid phylogenetic and bioinformatics analyses of hcov-emc. the diversity of covs is a result t of the infidelity of rdrp which make cov genomes especially plastic, a high frequency of homologous rna recombination due to their unique random template switching during rna replication, and their large genomes. in addition to biodiversity, a number of natural recombination and possible interspecies jumping events has also been documented in betacovs. 6, 11, [14] [15] [16] [17] [18] for group a betacovs, molecular clock analysis has shown that hcov-oc43 is a relatively recent zoonotic virus of bovine origin that emerged in around 1890 likely from bovine-to-human transmission. 17 we have also recently discovered rbcov hku14, closely related to other members of the species betacoronavirus 1 including hcov-oc43 and bcov, with recombination events that may have played a role in interspecies transmission of these hcov-oc43related viruses between human, cattle, rabbits, swine and horses. 6 despite having circulated in humans for more than a century, hcov-oc43 is also found to be continuously evolving, with the recent emergence of a novel genotype due to natural recombination. 15 for group b betacovs, sarsr-cov is believed to be transmitted from civet to humans, although it is the horseshoe bat that was likely the primary host. 7, 8 civet sarsr-cov was also likely a recombinant virus arising from different strains of sarsr-rh-batcov from different geographical locations in china. 14, 16 although no interspecies transmission events have been documented in group d betacovs, we have also identified recombination events between different ro-batcov hku9 strains from different bat individuals, which may have allowed for the generation of different genotypes. 11 while these findings supported that betacovs have the propensity to recombine and cause interspecies transmission, such events were unknown in group c betacovs. as hcov-emc is most closely related to ty-batcov hku4 and pi-batcov hku5, it would be important to study their genetic relatedness, which may provide clues on whether bats are the possible origin as in sarsr-cov.

</p>

</div>



<div id="info_a_edge16_42" class= "paragraph">

<h3>paragraph source</h3>

<p>covid-19 is the third known zoonotic coronavirus disease 3 sars is a zoonosis caused by sars-cov, which has infected 8096 humans, including 774 deaths (mortality rate 9.6%), in at least 29 countries 4 . another highly pathogenic coronavirus, mers-cov, has been reported in 27 countries with reported viral infection and 858 associated deaths (mortality rate 34.4%). 5 research indicates that sars-cov was transmitted from civet cats to humans and mers-cov was transmitted from dromedary camels to humans. however, the intermediate host of sars-cov-2 has not been identified 6, 7, 8 .

</p>

</div>



<div id="info_a_edge16_43" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses (cov) are a large family of viruses that cause sickness ranging from the common cold to more severe diseases such as middle east respiratory syndrome (mers-cov) and severe acute respiratory syndrome (sars-cov). coronaviruses caused diseases in mammals and birds, a zoonotic virus that is transmitted between animals and people. detailed investigations found that sars-cov was transmitted from civet cats to humans and mers-cov from dromedary camels to humans. when a virus spread circulating animal populations affect human population, this is termed as spillover event. it is speculated that the 2019 novel coronaviurs was originated in bats and was transmitted in humans, possibly with pangloin as an intermediatory hosts. coronavirus disease (covid-19) is a new strain that was discovered in 2019 (not previously recognized in humans). unfortunately, covid-19 is now spreading from humans to humans.

</p>

</div>



<div id="info_a_edge16_44" class= "paragraph">

<h3>paragraph source</h3>

<p>although sars and mers have a higher mortality rate, sars-cov-2 infection spreads much more rapidly . as of may 2020, more than 4,000,000 confirmed infections were reported in 216 countries, including over 300,000 deaths (who, 2020) . mers-cov, sars-cov, and sars-cov2 were suggested to originate from bats and most likely serve as a reservoir host for these viruses (ge, et al., 2013; haagmans, et al., 2014; li, et al., 2005; memish, et al., 2013; zhou, et al., 2020) . detailed investigations of the zoonotic origin of human covs indicate that sars-cov was transmitted from palm civets to humans and mers-cov from dromedary camels to humans (guan, et al., 2003; haagmans, et al., 2014; kan, et al., 2005) . however, the intermediate host for zoonotic transmission of sars-cov-2, linked to a wet animal market in wuhan, is still under investigation (walls, et al., 2020; ye, et al., 2020) .

</p>

</div>



<div id="info_a_edge16_45" class= "paragraph">

<h3>paragraph source</h3>

<p>a newly enteric coronavirus, pdcov, was identified in the usa in 2014, and this coronavirus caused clinical signs in swine similar to those of pedv (wang et al., 2014a) . in this study, a novel peav (peav-gd-ch/2017) strain was identified from suckling piglets with diarrhea, and this strain shares high identities with the other two peav strains that were previously reported (gong et al., 2017; pan et al., 2017) . these novel peavs are most identical to bat-hku2, with 95% nucleotide identity, and have the same genome organization and trs motif for each orf. the greatest difference between peav and bat-hku2 is their s proteins, which share 85% amino acid identity each other, a value much lower compared with those of other proteins (table 1) . this difference is caused by amino acid mutations in the s protein, particularly in the ntd in the s1 subunit, which has been proven to be the key factor determining issue tropism and the host range of coronaviruses (lu et al., 2015) . in addition to its low amino acid identity with the s protein of hku2-like coronavirus, it shares low amino acid identity (lower than 28%) with s proteins of all known coronaviruses. thus, clarifying the origin of the s proteins of peav and hku2-like coronavirus is important for determining the origin and evolutionary history of these coronaviruses. a previous study showed that the extreme ntd in the s1 subunit of peav is structurally similar to that of nl63, while the rest of the s1 subunit is structurally similar to that of mhv (pan et al., 2017) . in addition, a short peptide in the s protein of bat-hku2 was found to be homologous to a corresponding peptide within the receptor-binding motif (rbm) in the s1 subunit of sars-cov (lau et al., 2007) . we also analyzed the arginine (position 545) at cleavage sites s1/s2 of peav and found that it is conserved in several beta-covs in this study. moreover, the phylogenetic tree based on the s protein presents a uniform evolutionary relationship; these bat-hku2like coronaviruses cluster with a newly identified rat-cov, lrnv, which represents a novel species of coronaviruses. all of these strains form a distinct lineage and cluster with beta-covs but are separate from all four known subgroups of beta-covs (fig. 3) , which may indicate that these strains are part of a novel subgroup of beta-covs. these results suggest that peav and hku2-like coronaviruses may have some relations with beta-covs and most likely resulted from recombination with the backbone of alpha-cov and the s gene from an unrecognized beta-cov. another large difference between peav and bat-hku2 is n protein, which share about 93.9% amino acid identity, as n protein is a multifunctional protein for coronaviruses, which is involving in virus replication, budding and pathogenesis et al. (mcbride et al., 2014) . while the role of these mutations in n protein between peav and bat-hku2 should further investigate. the origin and emergence time of a newly emerged pathogen are important issues to answer to determine the evolutionary history and plan methods of prevention for these new pathogens. for example, previous sars research reported that the interspecies transfer of sarslike coronaviruses from bats to the amplifying host (e.g., civet) occurred in 1998 and that interspecies transfer from civet to humans occurred in 2002 (chinese, 2004; hon et al., 2008; lau et al., 2010; song et al., 2005) . these results provide insight into the origin and evolutionary history of sars coronavirus. the origin and divergence time of other coronaviruses have also been estimated previously; the divergence time of bovine-cov and hcov-oc43 could be dated back to the end of the 19th century to the beginning of the 20th century and was estimated at 1910 (vijgen et al., 2006) . the tmrca for all pdcov strains was reported at 1991, approximately 24 years before pdcov was identified (woo et al., 2017) . in this study, we also addressed the emergence time and evolutionary history of peav and of the other coronaviruses based on the rdrp gene by bayesian analysis. in anticipation, the mean divergence time of bovine-cov and hcov-oc43 was estimated at 1914, and the mean tmrca of human and civet sarsr-cov was estimated at 2001 in our analysis (fig. 4) . these results are highly consistent with those of a previous report discussed above and further indicate that our analysis is unbiased. the emergence time of peav was estimated at 1926 (95% hpd, 1864 to 1984) based on our analysis, which indicates that peav is not newly emerged and may have circulated in swine herds for several decades since interspecies transmission from bat to swine occurred. in addition, these hku2-like coronaviruses have a common ancestor with hcov-nl63 and hcov-229e, and the divergence time was estimated at 277, which indicates that these hku2-like coronaviruses have a long evolutionary history. the mean tmrca for alpha-cov, beta-cov, and gamma-cov, as well as those of all coronaviruses estimated in this study, were later compared with those of a previous report (woo et al., 2012) . nevertheless, the mean tmrca coincides with the regions with 95% hpd to each other. the evolutionary rate of different coronaviruses was estimated previously; 4.3  10 4 substitutions per site per year was estimated for hcov-oc43 (vijgen et al., 2005) , and the mean evolutionary rate for group 1b coronaviruses was estimated to be 3  10 4 substitutions per site per year (pyrc et al., 2006) . in addition, the evolutionary rate for all coronaviruses was estimated to be 1.3  10 4 substitutions per site per year (woo et al., 2012) , which is estimated to be 1.93  10 4 (95% hpd, 1.27  10 4 to 3.57  10 4 ) nucleotide substitutions per site per year for the rdrp gene in this study, and all of these results are comparable to each other.

</p>

</div>



<div id="info_a_edge16_46" class= "paragraph">

<h3>paragraph source</h3>

<p>the human coronaviruses are known major pathogens of respiratory diseases that are highly infectious and communicable. for instance, the sars-cov can be transmitted directly from civets to humans; the mers-cov can be transmitted from dromedary camels to human. there is also evidence of human to human transmission. as earlier stated, sars-cov-2 possesses certain similar characteristics as sars-cov. when an individual comes in contact with a sars-cov-2, the virus targets major routes where it can easily propagate to get to target cells and mediate infection. coronaviruses are respiratory viruses; thus, their first major target is the epithelial cells of the respiratory tract where they bind to receptors (the ace-2) [12] . by binding to the ace-2 expressed on the respiratory epithelial cells, the sars-cov/sars-cov-2 not only mediates infection but also facilitates their replication and transmission [2, 12, 15, 16] . this marks the genesis of the pathology of the virus.

</p>

</div>



<div id="info_a_edge16_47" class= "paragraph">

<h3>paragraph source</h3>

<p>in the case of sars-cov, wild animals such as palm civets, sold as a delicacy on chinese 'wet markets', were initially identified as the immediate source of the virus [16] , but from 2005 insectivorous rhinolophid bats came into focus as the original reservoir, from where the virus was possibly transmitted to civets and other market species and from them to humans [17, 18] (see [19] for a recent review on bat coronaviruses). however, it took until 2013 to discover a bat coronavirus that is more than 95% identical to sars-cov and uses the same receptor on the surface of host cells, the angiotensin-converting enzyme 2 (ace2) [20] . in the case of mers coronavirus (mers-cov), bats were again suspected to be the reservoir as a few coronaviruses with high sequence similarity to mers-cov were discovered in african and european bats [21, 22] , but in recent months the picture has changed somewhat and dromedary camels are now the main suspects of being the reservoir from where the zoonotic transmission into the human population originates [23, 24] .

</p>

</div>



<div id="info_a_edge16_48" class= "paragraph">

<h3>paragraph source</h3>

<p>it is believed that a novel coronavirus, severe acute respiratory syndrome coronavirus (sars-cov), was passed from palm civets to humans and caused the epidemic of sars in 2002 to 2003. the major species barriers between humans and civets for sars-cov infections are the specific interactions between a defined receptor-binding domain (rbd) on a viral spike protein and its host receptor, angiotensin-converting enzyme 2 (ace2). in this study a chimeric ace2 bearing the critical n-terminal helix from civet and the remaining peptidase domain from human was constructed, and it was shown that this construct has the same receptor activity as civet ace2. in addition, crystal structures of the chimeric ace2 complexed with rbds from various human and civet sars-cov strains were determined. these structures, combined with a previously determined structure of human ace2 complexed with the rbd from a human sars-cov strain, have revealed a structural basis for understanding the major species barriers between humans and civets for sars-cov infections. they show that the major species barriers are determined by interactions between four ace2 residues (residues 31, 35, 38, and 353) and two rbd residues (residues 479 and 487), that early civet sars-cov isolates were prevented from infecting human cells due to imbalanced salt bridges at the hydrophobic virus/receptor interface, and that sars-cov has evolved to gain sustained infectivity for human cells by eliminating unfavorable free charges at the interface through stepwise mutations at positions 479 and 487. these results enhance our understanding of host adaptations and cross-species infections of sars-cov and other emerging animal viruses.

</p>

</div>



<div id="info_a_edge16_49" class= "paragraph">

<h3>paragraph source</h3>

<p>a novel coronavirus (2019-ncov) from wuhan, china, has recently caused over 500 confirmed cases of human infections and at least 17 deaths in china (https://www .cdc.gov/coronavirus/novel-coronavirus-2019.html). there are also numerous confirmed cases of 2019-ncov infections in other countries including the united states. many of the symptoms caused by 2019-ncov, such as acute respiratory syndrome, are similar to those caused by severe acute respiratory syndrome coronavirus (sars-cov). sars-cov emerged in 2002 to 2003 and transmitted among humans, causing over 8,000 confirmed cases of human infections and about 800 deaths (1) (2) (3) (4) . it briefly reemerged in 2003 to 2004, with 4 confirmed cases of mild human infections and no human-to-human transmission (5) (6) (7) . sars-cov has also been isolated from animals and been adapted to lab cell culture (5, (8) (9) (10) (11) . it is believed that bats and palm civets were the natural and intermediate reservoirs for sars-cov, respectively, and that sars-cov transmitted from palm civets to humans in an animal market in southern china (12) (13) (14) . it has been reported that 2019-ncov also infected humans in an animal market in wuhan, although the animal source of the outbreak is currently unknown. moreover, it has been confirmed that 2019-ncov has the capacity to transmit from human to human.

</p>

</div>



<div id="info_a_edge16_50" class= "paragraph">

<h3>paragraph source</h3>

<p>because of the resemblance of sars-cov-2 to sars-like coronaviruses, bats are likely to act as natural reservoir hosts of the progenitor of sars-cov-2 (andersen et al., 2020) . it is established that sars-cov was transmitted from palm civet to humans (wang et al., 2006) and mers-cov from dromedary camels to humans, yet the zoonotic origin of sars-cov-2 is still unclear. the genome sequence analysis shows that sars-cov-2 has the closest relative sars-like slcov/ratg13, found in horseshoe bat (rhinolophus affinis) from yunnan, china (zhou et al., 2020) . sars-cov-2 has 96.2% overall genome sequence identity to sars-like coronavirus slcov/ratg13 (zhou et al., 2020) . therefore, the natural reservoir of sars-cov-2 could be the horseshoe bat. recent evidence suggests pangolins as host candidates (lam et al., 2020) , however, the host pangolins are not firmly determined. not all bat sars-like cov can infect humans. for example, bat coronavirus, swine acute diarrhoea syndrome coronavirus (sads-cov) found in 2017 (zhou et al., 2018) , caused millions of piglet deaths, but no human cases. we may postulate that sars-cov-2 evolved from its progenitor through mutations and evolutionary fitness in the host-shift and adapting. sars-cov-2 are fast-evolving pathogens that continuously undertake mutations in the generations of infection of the host (yin, 2020; wang et al., 2020; korber et al., 2020) . this fact suggests that sars-cov-2 had evolved mutations in critical proteins and genome structures prior to establishing human infection. to understand how sars-cov-2 jumps from animals to adaptively infect humans is important to the surveillance of virus evolution and diversity, therefore ultimately controlling the covid-19 pandemic and preventing future sars-like outbreaks.

</p>

</div>



<div id="info_a_edge16_51" class= "paragraph">

<h3>paragraph source</h3>

<p>as the ctr of the s 1 subunit contains the protein rbd for most covs, also mutations in this part of s have been associated with changes in the virus' tropism. perhaps the most well-known example of viral cross-species transmission involves the sars-cov. studies support a transmission model in which a sars-like cov was transmitted from rhinolophus bats to palm civets, which subsequently transmitted the palm civet-adapted virus to humans at local food markets in southern china . according to this model, sars-like viruses adapted to both the palm civet and human host, which was reflected in the rapid viral evolution observed for these viruses within these species . two-amino acid substitutions within the rbd were elucidated that are of relevance for binding to the ace2 proteins of palm civets and humans (li et al., 2005b qu et al., 2005) . from these studies it appears that due to strong conservation of ace2 between mammalian species only a few amino acid alterations within the rbd are needed to change coronavirus host species tropism. indeed serial passage of sars-covs in vitro or in vivo can rapidly lead to adaptation to new host species (roberts et al., 2007) . sars-like viruses isolated from bats displayed major differences including a deletion in the ace2 rbm compared to human sars-cov (drexler et al., 2010; ren et al., 2008) and as a consequence were unable of using human ace2 as an entry receptor (becker et al., 2008) . however, recently a novel sarslike btcov was identified, which could use ace2 of rhinolophus bats, palm civets as well as of humans as a functional receptor . these findings not only provide further evidence that bats are indeed the natural reservoir for sars-like covs, but also that these bat coronaviruses can directly include human ace2 in their receptor repertoire. the detection of sequences of sars-cov-like viruses in palm civets and raccoon dogs (guan et al., 2003; tu et al., 2004) therefore probably reflects the unusually wide host range of these viruses. a similar promiscuous receptor usage is also observed for mers-cov which binds to dpp4 of many species (barlan et al., 2014; eckerle et al., 2014; falzarano et al., 2014; m uller et al., 2012; van doremalen et al., 2014) as indicated earlier.

</p>

</div>



<div id="info_a_edge16_52" class= "paragraph">

<h3>paragraph source</h3>

<p>until now, a total of seven human covs have been identified, namely hcov-229e, hcov-nl63, hcov-oc43, hcov-hku1, sars-cov, mers-cov, and, recently, 2019-ncov (1-3, 6-8). the former two human covs belong to the genus -cov, and the latter five human covs belong to the genus -cov. three recently identified human covs, that is, sars-cov, mers-cov, and 2019-ncov, have been recognized as zoonotic viruses, which transmit between animals and human. recent studies revealed that sars-cov was transmitted from civet cats to humans, mers-cov from dromedary camel, and 2019-ncov (probably) from pangolin (1, 2, (6) (7) (8) (9) . recent investigations indicated that bats were most probably the natural reservoir of sars-cov, mers-cov, and 2019-ncov (1, 6, (9) (10) (11) . genome sequence analysis revealed that 2019-ncov was distinct from sars-cov (about 79% identity) and mers-cov (about 50% identity) yet more closely related to sars-like-covs (about 88% identity) in bats (10, 11) .

</p>

</div>



<div id="info_a_edge16_53" class= "paragraph">

<h3>paragraph source</h3>

<p>the covid-19 outbreak is caused by a newly identified member of a large family of viruses known as coronaviruses (name deriving from the latin word corona, meaning crown) . according to the world health organization (who) [1] this family of viruses causes illnesses ranging from the common cold to more severe diseases such as severe acute respiratory syndrome (sars) and the middle east respiratory syndrome (mers). these viruses were originally transmitted from animals to people. sars, first recognized in mid-march 2003 [2] , was transmitted from civet cats to humans, while mers, first detected in saudi arabia in 2012 [3] , moved to humans from a type of camel. the novel coronavirus, identified by chinese authorities on january 7 is a new strain that has not been previously identified in humans. the sars-cov-2 virus, as it has been named by the who, seems to be closely related to bat and pangolin coronaviruses, but a final confirmation has not yet been given [4] . people may catch covid-19 by close contact with infected people, via small droplets produced when they cough, sneeze, talk, or by touching a contaminated surface and then their face. according to the who, symptoms of infection include fever, cough, shortness of breath and breathing difficulties. notwithstanding the fact that infected patients can also be asymptomatic, in the more severe cases, the virus can lead to pneumonia, multiple organ failure and death. the number of fatalities from the new coronavirus has overwhelmingly overtaken the toll of the sars outbreak but its percentage remains considerably lower compared to the 10 percent of sars and 30 percent of mers.

</p>

</div>



<div id="info_a_edge16_54" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses mainly cause respiratory and gastrointestinal tract infections and are genetically classified into four major genera: alphacoronavirus, betacoronavirus, gammacoronavirus, and deltacoronavirus (li, 2016) . the former two genera primarily infect mammals, whereas the latter two predominantly infect birds (tang et al., 2015) . six kinds of human covs have been previously identified. these include hcov-nl63 and hcov-229e, which belong to the alphacoronavirus genus; and hcov-oc43, hcov-hku1, severe acute respiratory syndrome coronavirus (sars-cov), and middle east respiratory syndrome coronavirus (mers-cov), which belong to the beta-coronavirus genus (tang et al., 2015) . coronaviruses did not attract worldwide attention until the 2003 sars pandemic, followed by the 2012 mers and, most recently, the 2019-ncov outbreaks (china cdc, 2020; song et al., 2019) . sars-cov and mers-cov are considered highly pathogenic (cui et al., 2019) , and it is very likely that both sars-cov and mers-cov were transmitted from bats to palm civets (guan et al., 2003) or dromedary camels (drosten et al., 2014) , and finally to humans (cui et al., 2019) .

</p>

</div>



<div id="info_a_edge16_55" class= "paragraph">

<h3>paragraph source</h3>

<p>a large number of molecular biologic and sequencing studies established that the coronaviruses share a common genetic organization and mode of replication. infected cells contain a set of 6 to 9 virus-specific polyadenylated rnas that make up a 3 co-terminal nested set, with the largest member of this set being the genome rna and the remaining rnas representing subgenomic mrnas. the subgenomic rnas encode the structural proteins enumerated above in a characteristic order: he (if present)-s-e-m-n. in addition to these structural genes, coronavirus subgenomic rnas encode accessory proteins that are not essential for viral replication. they generally modulate cellular responses to infection and are specific for each coronavirus subgroup. the replicase proteins are encoded in the 5 two-thirds of the genome. this region contains two very long partially overlapping open reading frames that are translated by a ribosomal frame shifting mechanism. the 750-kda primary translation product is cotranslationally autocatalytically cleaved into 16 nonstructural proteins (nsps) that encode the machinery needed for rna replication and transcription. many of the coronaviruses are associated with respiratory or enteric diseases, although some viruses replicate and cause disease at other sites, including the liver, spleen, and central nervous system. most of the coronaviruses are relatively species specific, only infecting their natural host and cell cultures derived from that host or a closely related host species. however, there are several examples of coronaviruses jumping species, with sars-cov being the most well known example. sars-cov is thought to have originated in bats, where sars-like covs have been identified, and then transmitted to civets and raccoon dogs in wild animal markets in southern china, from which they were subsequently transmitted to humans (lau et al., 2010) . during this chain of transmission, several key adaptations to the new host occurred (li et al., 2005a,b) . ccov and fecov provide another exception to the high degree of species specificity of coronaviruses, in that both viruses are able to grow in feline cells. phylogenetic analysis of the group ia coronaviruses, ccov, fecov, and tgev, suggest a complex set of recombination events has given rise to these viruses from a common ancestor, likely explaining the ability of both ccov and fecov to infect feline cells (wesseling et al., 1994; lorusso et al., 2008) . the species specificity of coronaviruses is controlled by the ectodomain of the s protein, and can be altered by creating chimeric s proteins that contain the ectodomain of a distantly related cov (kuo et al., 2000) .

</p>

</div>



<div id="info_a_edge16_56" class= "paragraph">

<h3>paragraph source</h3>

<p>for example, lyssaviruses such as hendra are periodically transmitted from bats to horses and then to humans who contact the infected horse. similarly, sars-cov was shown to be transmitted from bats to civet cats and from them to humans. to date, a virus identical to sars-cov-2 has not been identified in bats or any other non-human species, making its origins unclear. to date, the most closely related coronavirus to sars-cov-2, is the bat coronavirus, batcov ratg1, which has 96% whole-genome identity to sars-cov-2. 50 hence other genetic factors could underlie the apparent lack of susceptibility of dogs to covid-19 clinical infection.

</p>

</div>



<div id="info_a_edge16_57" class= "paragraph">

<h3>paragraph source</h3>

<p>the sequence of mers-cov was determined to be closely related to other betacoronaviruses isolated from bats and therefore a bat origin has been proposed early after genomic characterization [3] [4] [5] [6] [7] [8] . however, transmission of mers-cov to humans was suspected to occur via an intermediate mammalian host, since the majority of human middle east respiratory syndrome (mers) patients did not state any direct contact to bats prior to disease onset 6, 9 . similarly, severe acute respiratory syndrome coronavirus (sars-cov), a betacoronavirus of the lineage b, originated from bats 10 and spread from palm civets to humans in 2002/2003 11 .

</p>

</div>



<div id="info_a_edge16_58" class= "paragraph">

<h3>paragraph source</h3>

<p>mechanisms of viral emergence vet. res. (2010) 41:38 and re-emergences of human viral diseases, that have amounted to about one per year during the last decades, have been associated with rna viruses that display active recombination or reassortment. as examples, several recent poliomyelitis outbreaks have been related to viruses generated by recombination between attenuated vaccine polioviruses and other circulating enteroviruses [75] . as an historical event, the coronavirus mouse hepatitis virus appears to have acquired its hemagglutinin-esterase gene by recombination with an influenza type c virus [46] . also, the alphavirus western equine encephalitis virus probably resulted from an ancient recombination between a sindbis-like and a easter-equine encephalitis-like virus (reviewed in [87] ). the emergence of the sars coronavirus around 2002 probably involved the transfer of the virus from a bat to a carnivore (probably a civet cat); contact with civets may have facilitated the human outbreak. the human ace2 receptor-binding sequences used by the human sars coronavirus may have been acquired by recombination with a preexisting group 1 human coronavirus [45] . in this, and in the other cases in which recombination played a role in the generation of a new pathogen, selection of virus variants for a more efficient replication in the new host (for receptor use or other) probably took place [91] . genome segment reassortment is a major evolutionary force for some viruses with a segmented genome, the most studied case being reassortments in the influenza type a viruses. reassortants that have acquired the hemagglutinin and/or neuraminidase genes from viruses of some animal reservoirs (notably birds or swine) have been associated with the major human influenza pandemics such as the spanish influenza of 1918 (h1n1 virus), the hong kong influenza of 1968 (h3n2 virus) or the pandemics of 2009 (h1n1 virus) [35, 66, 74] . such viruses acquired a selective advantage relative to previously circulating influenza viruses because the new hemagglutinin and/or neuraminidase surface antigens were not recognized (or were only poorly recognized) by the neutralizing antibodies present in the human population. the acquisition of new genome segments that produce a change in antigenic specificity is termed antigenic shift. the gradual antigenic modification undergone by the virus as it circulates in the human population is termed antigenic drift. the latter is linked to point mutations that affect the antigenic determinants, and can be regarded as the fitness adjustment by mutation as the virus explores new environments (some of which include anti-influenza antibodies) following a reassortment event. drastic and gradual versions of antigenic variation are not exclusive of influenza virus, and they have been amply recognized as a major factor in the adaptation of viruses to host organisms endowed with an immune system [48, 67] . viruses are undoubtedly subjected to selective constraints additional to the several components of an immune response, and some of them probably have not even been identified. multiple constraints contribute to shape the genomic compositions of the viral populations as they complete successive life cycles. mutation, recombination and reassortment can produce new viral forms, a minority of which might be competent to replicate in a new environment. there is evidence that a true recombinant hemagglutinin might have been one of the virulence determinants of the influenza pandemics of 1918 [28] .

</p>

</div>



<div id="info_a_edge16_59" class= "paragraph">

<h3>paragraph source</h3>

<p>t he ability of coronaviruses (covs) to infect multiple species and to rapidly change through recombination presents a continuing human health threat. the epidemic of severe acute respiratory syndrome (sars) during 2003-2004 during which a cov transmitted from bats to civet cats and then to humans demonstrated this potential (reviewed in [1, 2] ). recently, a novel human betacoronavirus (betacov) was found to be associated with at least 13 human infections, 7 of which were fatal (3) (4) (5) (6) (7) . one of the viruses, emc/2012, has been sequenced, and its sequence similarity to several bat covs suggested an animal origin, but a definitive bat species of origin has not yet been identified (3) . additional genome sequences from this virus are needed to aid diagnostics, monitor population dynamics, identify the animal source, and characterize mechanisms of pathogenesis. the large size (30,000 nt) and high variability of cov rna genomes present a challenge for sequencing.

</p>

</div>



<div id="info_a_edge16_60" class= "paragraph">

<h3>paragraph source</h3>

<p>a novel coronavirus (cov), severe acute respiratory syndrome coronavirus (sars-cov), caused a worldwide epidemic of sars with a fatality rate of 9.6% in 2002/03 and later reemerged and resulted in infection of four individuals with full recovery in the winter of 2003/04 [1] [2] [3] [4] [5] . sars-cov has been demonstrated to be a zoonotic disease that evolved in palm civet and human hosts. the global outbreak that occurred in 2002/03 and the cluster of 2003/04 sars cases were the result of two independent zoonotic transfers from palm civets to humans [6] [7] [8] [9] . although palm civets were identified as the hosts involved in human transmission, evidence suggested the existence of another precursor reservoir. indeed bats, predominantly horseshoe bats, were later found to be a natural reservoir of sars-like-covs, and harbor more diverse viruses than any other hosts [10] [11] [12] [13] [14] . variants of sars-like-covs circulating in bats may cross the species barrier again and this threat is enhanced by the large numbers of bats that often congregate, their broad geographic distribution and their ability to travel long distances. diversity of host range and variant immune pressures within the natural reservoir or intermediate hosts are likely to continue to drive sars-cov evolution.

</p>

</div>



<div id="info_a_edge16_61" class= "paragraph">

<h3>paragraph source</h3>

<p>the severe acute respiratory syndrome [sars] first appeared in southern china [guangdong province] in november 2002, as an atypical community pneumonia [1] . within a year, the world health organization [who] reported 8096 cases with 774 deaths [9. 6% fatality] in >30 countries from five continents, and the outbreak was declared a pandemic infection [1] . in the elderly and subjects with significant comorbid illness, the mortality rate was up to 50%. investigations revealed that sars was due to a novel coronavirus [sars-cov] which was circulating among wild game animals in wet markets of southern china. the palm civet, a wild feline, was considered the amplification host that transmitted the virus to humans from occupational contact and handling by consumers during the preparation for consumption as a delicacy [2, 3] . sars-cov mainly spread from human to human by respiratory droplets or by contact of mucosae with contaminated fomites. spread of sars-cov from mainland china to hong kong and other countries was from interpersonal transmission in healthcare facilities, homes, workplaces, and public transports [1] .

</p>

</div>



<div id="info_a_edge16_62" class= "paragraph">

<h3>paragraph source</h3>

<p>three emerging, highly pathogenic human covs are sars-cov, middle east respiratory syndrome (mers)-cov, and covid-19 virus, which was previously named 2019-ncov by the world health organization (who), and is also known as hcov-19 or sars-cov-2 [1] . atypical pneumonia (sars) was first reported from guangdong province, china in late 2002. sars caused a global pandemic in 2003 with approximately 10% (774/8098) case fatality rate (cfr) [2] . sars-cov has not circulated in humans since 2004. mers-cov was first reported from saudi arabia in 2012 and has continued to infect humans with limited human-to-human transmission, leading to a cfr of approximately 34.4% (858/2494) in 27 countries, according to the most recent who report i . both sars-cov and mers-cov are zoonotic viruses. they use bats as their natural reservoirs and transmit from bats to intermediate hosts (e.g., palm civets for sars-cov, dromedary camels for mers-cov), leading to infection in humans [2, 3] .

</p>

</div>



<div id="info_a_edge17_125" class= "paragraph">

<h3>paragraph source</h3>

<p>the three horses inoculated with 1 ml of the vaccine did not exhibit any adverse reaction during this study. in contrast, two out of the three horses inoculated with 2 ml of the vaccine exhibited swelling at the inoculation site after the second vaccination. none of the horses developed a fever after the vaccinations. administration of more than 2 ml of the vaccine to horses would likely increase the risk of adverse reactions. as described above, a significant increase in antibody titers against ecov was observed from 14 dpi irrespective of whether 1 or 2 ml was administered. additionally, the differences in antibody titers against ecov at each dpi from 14 dpi were less than twofold between horses inoculated with 1 and 2 ml. these results suggest that inoculation of 1 ml is suitable for horses as well as for cattle. although horse no. 5 had no detectable antibodies against ecov before vaccination, the horse already had antibodies against bcov (table 1) . that horse was born and had been kept at a farm that reared cows before coming to our facility. in saudi arabia, dromedary camel coronavirus hku23 was detected in apparently healthy horses kept at facilities that reared camels, sheep, goats, and chickens [5] . hku23, which is closely related to bcov, is endemic in camels of the middle east [16] and the hku23-positive horses frequently came into contact with camels and other animals [5] . hku23 may have been transmitted from infected camels to those horses. horse no. 5 in the present study may have also been in contact with infected cows, and bcov may have been transmitted from infected cows to the horse. however, because there is no epidemiological information, it is unknown whether there were in fact bcov-infected cows at the farm or whether horse no. 5 had shown any clinical signs.

</p>

</div>



<div id="info_a_edge18_126" class= "paragraph">

<h3>paragraph source</h3>

<p>in areas where white-tailed deer and meningeal worm are endemic, meningeal worm prophylaxis with monthly, parenteral administration of avermectins has been recommended. 7 documentation of the development of anthelminthic resistance because of meningeal worm prophylaxis has led some farms to pursue drug-free meningeal worm prophylaxis, which may be instituted when designing housing and husbandry recommendations. meningeal worm is transmitted from white-tailed deer to camelids through arboreal snails. [7] [8] [9] as such, deer-proof fencing, introduction of guinea fowl, and the use of molluscicides can minimize transmission. removal of organic matter, in the form of leaf or wood piles, and establishment of pasture perimeters using limestone or gravel also inhibits snail and slug habitation in camelid pastures. 10 more complete assessments of parasite control measures are covered in the articles ''gastrointestinal parasitology'' by ballweber and ''neurological diseases'' by whitehead and bedenice elsewhere in this issue.

</p>

</div>



<div id="info_a_edge19_127" class= "paragraph">

<h3>paragraph source</h3>

<p>in areas where white-tailed deer and meningeal worm are endemic, meningeal worm prophylaxis with monthly, parenteral administration of avermectins has been recommended. 7 documentation of the development of anthelminthic resistance because of meningeal worm prophylaxis has led some farms to pursue drug-free meningeal worm prophylaxis, which may be instituted when designing housing and husbandry recommendations. meningeal worm is transmitted from white-tailed deer to camelids through arboreal snails. [7] [8] [9] as such, deer-proof fencing, introduction of guinea fowl, and the use of molluscicides can minimize transmission. removal of organic matter, in the form of leaf or wood piles, and establishment of pasture perimeters using limestone or gravel also inhibits snail and slug habitation in camelid pastures. 10 more complete assessments of parasite control measures are covered in the articles ''gastrointestinal parasitology'' by ballweber and ''neurological diseases'' by whitehead and bedenice elsewhere in this issue.

</p>

</div>



<div id="info_a_edge20_124" class= "paragraph">

<h3>paragraph source</h3>

<p>niaid [14] and cdc [15] also publish lists of pathogens of priority which comprise bacteria and viruses, but categorize these into three groups depending on pathogenicity, accessibility and the availability of vaccines and therapies. all the viruses listed by the who also occur on these lists, alongside bacterial pathogens of concern. one of these is yersinia pestis, causative of bubonic and pneumonic plague, which is recognized by all three bodies (who, cdc, niaid) as a current priority following the exceptionally large and serious outbreak between september 2017 to april 2018 in madagascar [16] , where the disease is endemic. in addition, niaid recognizes the added threat to human health posed by the acquisition of amr by pathogens and who has also published a priority list [17] of bacterial species for which r&d is required to develop new antibiotics (table 2) . globally, there are many regions of endemic disease which are maintained by reservoirs of zoonotic pathogens in the local wild animal species. these pathogens comprise viral, bacterial and rickettsial species and some have complex lifecycles, infecting an environmental (e.g. standing water) or animal reservoir and then being transmitted either by direct contact with man or indirectly, via an insect (e.g. mosquito) or mammalian (e.g. bat, civet, camel) vector, to man, from which they may be spread by human-tohuman transmission. (fig. 1 ). an example of a serious and widespread bacterial infection is mosquito-transmitted plasmodium falciparum, causing malaria in many tropical regions. malaria causes significant morbidity and co-morbidity in these regions in which it is endemic. due to the complex life cycle of the causative bacterium, it has been challenging to achieve a vaccine for malaria. the most advanced candidate is the rts,s/as01 vaccine [18] , which is undergoing a pilot implementation in three countries in sub-saharan africa, with a view to determining its impact on disease prior to a wider implementation [19] . despite the availability of approved vaccines [20, 21] , typhoid fever and cholera remain significantly debilitating enteric diseases in regions of the world where hygienic living conditions are poor and there is little access to health care. both infections are caused by bacteria (salmonella typhi and vibrio cholera, respectively) which exist in contaminated water and food. tuberculosis (tb) is a respiratory infection which is widespread globally. caused by mycobacterium tuberculosis, and transmitted to man from zoonotic reservoirs (badgers, cattle) with a high potential for subsequent human-to-human transmission, tb is prevalent in susceptible individuals living in overcrowded conditions [22] . although the bcg vaccine has been in routine use for many years, variable efficacy has been reported, depending on region of use [23, 24] with 60-80% reported in the united kingdom but lower levels in equatorial countries. the emergence of multi-drugresistant tb (mdr tb) in recent years raises the bar for treatment of this disease and makes a high level of vaccine efficacy even more important [25] . examples of zoonotic reservoirs which maintain endemic viral diseases are numerous and are summarized in table 3 . in particular, viral endemic disease is caused by the coronaviruses (mers and sars) in saudi arabia and the eastern mediterranean countries; and by the lassa arenavirus, which is endemic in west african countries and caused a serious outbreak of lassa fever in nigeria in 2018 [26] ; other viruses which are endemic and cause viral haemorrhagic fever include dengue, which is widespread in tropical and subtropical regions worldwide [27] , the filoviruses (ebola and marburg) which have a zoonotic reservoir in bats in sub-saharan africa [28] ; and yellow fever virus, which is prevalent in africa, central and south america and the caribbean [29] . interestingly, the same mosquito (aedes aegypti) which spreads yellow fever virus also transmits the dengue, chikungunya and zika viruses [30] . chikungunya virus has a widespread distribution in africa, asia, india and south america and with occasional cases in europe, and causes a debilitating, but rarely fatal, disease [31] . zika virus emerged in brazil in 2015 [32] , although it was first detected in monkeys in uganda as early as 1947 [33] , and the first documented human case occurred in nigeria in 1954 [34] . the large brazilian outbreak of zika viral disease culminated in 2017, with thousands of cases reported [35] . rift valley fever virus (rvfv) is another zoonotic virus which is endemic in sub-saharan africa and primarily infects cattle, sheep and goats, but can be transmitted to man by mosquito bite to cause an acute fever [36] . an example of a bacterium which has been classed as a re-emerging pathogen and which is endemic in global regions is y. pestis, causative of plague. bubonic [37] [38] [39] . over thousands of years, y. pestis has evolved away from the enteric yersinia species to become a lethal flea-transmitted bacterium [40] . it is transmitted to man, typically from a zoonotic reservoir in infected rats or other rodents (e.g. ground squirrels or prairie dogs), by flea-bite to cause bubonic plague [41] (fig. 2 ). if not detected and treated, this can develop into either septicemic plague or the most serious form of all, a secondary pneumonic plague. in turn, individuals with pneumonic plague can transmit this by aerosol droplet to others, to establish a primary pneumonic plague infection. each year, a few cases of plague are also reported in the southwestern united states, where the disease has been endemic in the rodent population since the late 1800s [42] . as well as infecting the rodent population, infected fleas can spread y. pestis to other wildlife species (e.g. rabbits/hares or, rarely, domesticated animals [43] ) which, in turn, raises the potential of transmission to man by inhalation to cause a primary pneumonic plague. in endemic areas, outbreaks of plague are often associated with seasonal environmental changes, causing rodents to stray closer to human habitation. the plague outbreak in madagascar during 2017/18 was particularly serious, with an estimated 2671 cases and 239 deaths (89% fatality rate) [44] . this outbreak was approximately sixfold greater than usual, with an unusual predominance of pneumonic, rather than bubonic plague.

</p>

</div>



<div id="info_a_edge21_123" class= "paragraph">

<h3>paragraph source</h3>

<p>viruses related to simian virus 5 (parainfluenza virus 5) have been detected in humans, but the role of cpiv as a human pathogen is controversial. reoviruses have a broad host range, and human infection has been associated with enteritis and respiratory disease; mrv-3 infection was associated with meningitis in a child.64 thus they represent a potential zoonosis. although adenoviruses, influenza viruses, coronaviruses, and paramyxoviruses cause respiratory disease in humans, there is no evidence that cav-2, civ, cdv, chv-1, or crcov can be transmitted to humans. human influenza viruses such as h1n1 can cause disease in dogs and may have the potential to spread from infected dogs back to humans.case examplesignalmentwinnie, a 10-year old male neutered miniature schnauzer from sacramento, cahistorywinnie was brought to his local veterinarian because of a 1-day history of serous ocular and nasal discharge, and sneezing. fever (103.5f or 39.7c) was documented on physical examination, and enrofloxacin (2 mg/kg po q24h), bacitracin-neomycin-polymyxin ophthalmic ointment (both eyes q8h), and diphenhydramine (2.2 mg/kg po q12h) were prescribed, without clinical improvement. a day later, he became lethargic, inappetent, and polydipsic, and the owners described continuous nasal discharge, and a moist cough. winnie was brought to the uc davis vmth for a second opinion. he had been vaccinated regularly for cdv, cav-2, cpv, rabies, bordetella bronchiseptica, and cpiv. he was recently cared for in a boarding facility for a week and had been home for the past 11 days. he also regularly visited dog parks. there were two other miniature schnauzers at home, and both were currently well.current medicationsenrofloxacin, 2 mg/kg po q24hother medical historyincreased activity of serum alp was noted at a senior care visit 2 months before the onset of respiratory signs.physical examinationbody weight9.4 kggenerallethargic but hydrated. t = 100.8f (38.2c), hr = 104 beats/min, respiratory rate = 12 breaths/min, mucous membranes pink, crt = 1 to 2 s.eyes, ears, nose, and throatsevere chemosis, hyperemia, and mucoid ocular discharge were present bilaterally. there was also profuse bilateral serous nasal discharge and decreased nasal airflow, hypersalivation, and right tonsillar enlargement.musculoskeletalbody condition score 5/9.respiratorystertorous respiratory noises were noted. increased respiratory effort with a normal respiratory rate was also present. auscultation revealed referred wheezing noises in all lung fields.gastrointestinaltense abdomen, hepatomegaly was detected on abdominal palpation.all other systemsno abnormalities were detected.ophthalmologic examinationpupillary light reflexes (plrs) (direct and consensual) were brisk and complete. there was no evidence of anisocoria. menace response, dazzle, and palpebral reflexes were all complete. globe position and movements were normal bilaterally (ou). periorbital palpation and globe retropulsion were also normal. the dog behaved as if sighted. the eyelids were normal. both nictitans were hyperemic and edematous, and there was moderate conjunctival hyperemia, chemosis, and mucoid ocular discharge ou. bilateral iris atrophy and lens nuclear sclerosis were also present. the cornea, anterior chamber, vitreous, and dilated fundic examination were normal ou. in the left eye (os), a pinpoint incipient anterior cortical cataract was identified just ventral to the center of the lens. in the right eye (od), a linear pigment deposition on the anterior lens capsule was present at the 5 oclock position.schirmer tear test8 mm/min ouintraocular pressure16 mm hg os, 17 mm hg odfluorescein stainmultiple dendritic ulcers were identified at the ventral peripheral cornea od (see figure 17-2).laboratory findingscbchct 53.5% (40-55%)mcv 67.7 fl (65-75 fl)mchc 35.3 g/dl (33-36 g/dl)1 nucleated red cell/100 wbcswbc 14,600 cells/l (6000-13,000 cells/l)neutrophils 10,804 cells/l (3000-10,500 cells/l)band neutrophils 438 cells/llymphocytes 1752 cells/l (1000-4000 cells/l)monocytes 1460 cells/l (150-1200 cells/l)eosinophils 0 cells/l (0-1500 cells/l)platelets 216,000 platelets/l (150,000-400,000 platelets/l)a few highly reactive lymphocytes were noted.serum chemistry profilesodium 142 mmol/l (145-154 mmol/l)potassium 6.1 mmol/l (3.6-5.3 mmol/l)chloride 102 mmol/l (105-116 mmol/l)bicarbonate 25 mmol/l (16-26 mmol/l)phosphorus 6.7 mg/dl (3.0-6.2 mg/dl)calcium 9.5 mg/dl (9.9-11.4 mg/dl)bun 20 mg/dl (8-31 mg/dl)creatinine 0.7 mg/dl (0.5-1.6 mg/dl)glucose 80 mg/dl (60-104 mg/dl)total protein 5.7 g/dl (5.4-7.4 g/dl)albumin 2.2 g/dl (2.9-4.2 g/dl)globulin 3.5 g/dl (2.3-4.4 g/dl)alt 564 u/l (19-67 u/l)ast 116 u/l (21-54 u/l)alp 1024 u/l (15-127 u/l)gamma gt 10 u/l (0-6 u/l)cholesterol 195 mg/dl (135-345 mg/dl)total bilirubin 0.4 mg/dl (0-0.4 mg/dl).urinalysissgr 1.009; ph 8.5, negative for protein, bilirubin, hemoprotein, glucose; rare wbc/hpf, 0 rbc/hpf, 0-3 hyaline casts/hpf, few amorphous crystals, moderate degenerated cells and amorphous debris in the sediment.imaging findingsthoracic and cervical radiographsa mild to moderate bronchial and interstitial pattern was identified and was most prominent in the caudal lung fields. cervical radiographs were unremarkable. the pulmonary changes were considered nonspecific, but more significant than those expected due to age.abdominal ultrasoundthe cranial pole of the right adrenal was mildly to moderately enlarged. the liver appeared sonographically normal. a 3 mm stone was identified in the urinary bladder.microbiologic testinga nasal swab specimen was submitted for real-time pcr panel for canine respiratory pathogens, which included chv-1, civ, cdv, cpiv, cav-2, and b. bronchiseptica. the results were available 3 days later, and the specimen was positive for chv-1 dna.diagnosisupper respiratory disease and keratitis associated with chv-1 infection.treatmentwinnie was hospitalized in isolation and treated with 0.9% nacl (30 ml/hr iv), nebulization for 15 min q8h, clavulanic acid-amoxicillin (13.5 mg/kg po q12h), and lubricating eye ointment (1 inch ou q4h). clinical improvement occurred a day later, electrolyte abnormalities normalized, and the dog was discharged with instructions to administer topical idoxuridine (0.1% ophthalmic solution, 2 drops ou q4-6h). unfortunately a caretaker of the dog did not regularly administer the idoxuridine. at a recheck 5 weeks after the dog was first seen, the owner reported 80% improvement in winnies respiratory signs, but he continued to sneeze approximately six times a day and was pawing at his eyes. examination revealed persistent conjunctivitis, and a fluorescein stain showed large, superficial areas of uptake over the central cornea ou (see figure 17-2). a schirmer tear test (stt) was 13 mm/min os and 8 mm/min od. a cbc was normal, and the serum biochemistry profile showed persistently increased alp (835 u/l) and ggt activities (12 u/l) and improvement in the serum alt activity (124 u/l). the owner was instructed to administer idoxuridine for an additional 2 weeks, as well as lubricating ointment, bacitracin-neomycin ophthalmic ointment (q8h ou), and l-lysine gel (2 ml [400 mg] q12h po), separating the ophthalmic medications with administration of the idoxuridine first. at a recheck 2 weeks later, all signs had resolved and there was no fluorescein uptake. stt results were 14 mm/min os and 10 mm/min od. the idoxuridine was discontinued. winnie had another episode of respiratory signs 4 months later, at which time the stt results were 15 mm/min ou. treatment with lubricating eye ointment and l-lysine was continued.commentsthis was an unusual case of ocular and respiratory herpesviral infection. the early detection of dendritic ulcers allowed a connection to be made between the clinical signs and positive pcr assay result. the negative pcr assay results for other respiratory pathogens did not rule out the possibility of co-infections, because shedding of these organisms may have ceased or been at undetectable levels. although idoxuridine was used to treat this dog, topical cidofovir may also have been effective.50 approximately 2 years later, mild serous nasal discharge and depigmentation of the planum nasale developed. a biopsy of the nasal planum showed epitheliotropic lymphoma. the dog was treated with multiagent chemotherapy for over a year without relapse of herpetic keratitis.

</p>

</div>



<div id="info_a_edge22_88" class= "paragraph">

<h3>paragraph source</h3>

<p>covid-19 is the third known zoonotic coronavirus disease 3 sars is a zoonosis caused by sars-cov, which has infected 8096 humans, including 774 deaths (mortality rate 9.6%), in at least 29 countries 4 . another highly pathogenic coronavirus, mers-cov, has been reported in 27 countries with reported viral infection and 858 associated deaths (mortality rate 34.4%). 5 research indicates that sars-cov was transmitted from civet cats to humans and mers-cov was transmitted from dromedary camels to humans. however, the intermediate host of sars-cov-2 has not been identified 6, 7, 8 .

</p>

</div>



<div id="info_a_edge22_89" class= "paragraph">

<h3>paragraph source</h3>

<p>in december 2019, covid-19 pandemic emerged in wuhan china. it became a global health problem in a very short time because of it contact-transferred behaviours 1, 2 . covid-19 is caused by a virus known as severe acute respiratory syndrome coronavirus 2 or also sars-cov-2 3 . a large family of these virus are causing different kind diseases such as cold, middle east respiratory syndrome and severe acute respiratory syndrome. the new addition to coronavirus family covid-19 is discovered in 2019, which has never been detected in humans. covid-19 is known as zoonotic disease because it is transferred from animal to human such as bat 4 , similar to sars-cov virus which is contaminated through cat and mers-cov from dromedary 5 . it is presumed that respiratory transmission and physical contact is cause to spread covid-19 rapidly. according to world economic forum , people with no symptoms or mild symptoms are the main reason of spreading the epidemic. one our of four subject shows no symptoms of covid-19, although he/she is suffering with the disease 6 .

</p>

</div>



<div id="info_a_edge22_90" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses (cov) are a large family of viruses that cause sickness ranging from the common cold to more severe diseases such as middle east respiratory syndrome (mers-cov) and severe acute respiratory syndrome (sars-cov). coronaviruses caused diseases in mammals and birds, a zoonotic virus that is transmitted between animals and people. detailed investigations found that sars-cov was transmitted from civet cats to humans and mers-cov from dromedary camels to humans. when a virus spread circulating animal populations affect human population, this is termed as spillover event. it is speculated that the 2019 novel coronaviurs was originated in bats and was transmitted in humans, possibly with pangloin as an intermediatory hosts. coronavirus disease (covid-19) is a new strain that was discovered in 2019 (not previously recognized in humans). unfortunately, covid-19 is now spreading from humans to humans.

</p>

</div>



<div id="info_a_edge22_91" class= "paragraph">

<h3>paragraph source</h3>

<p>middle east respiratory syndrome (mers) is a zoonotic viral disease that can be transmitted from dromedaries to human beings. more than 1500 cases of mers have been reported in human beings to date. although mers has been associated with 30% case fatality in human beings, mers coronavirus (mers-cov) infection in dromedaries is usually asymptomatic. in rare cases, dromedaries may develop mild respiratory signs. no mers-cov or antibodies against the virus have been detected in camelids other than dromedaries. mers-cov is mainly acquired in dromedaries when they are less than 1 year of age, and the proportion of seropositivity increases with age to a seroprevalence of 100% in adult dromedaries. laboratory diagnosis of mers-cov infection in dromedaries can be achieved through virus isolation using vero cells, rna detection by real-time quantitative reverse transcriptase-pcr and antigen detection using respiratory specimens or serum. rapid nucleocapsid antigen detection using a lateral flow platform allows efficient screening of dromedaries carrying mers-cov. in addition to mers-cov, which is a lineage c virus in the betacoronavirus (betacov) genus, a lineage b betacov and a virus in the alphacoronavirus (alphacov) genus have been detected in dromedaries. dromedary cov uae-hku23 is closely related to human cov oc43, whereas the alphacov has not been detected in human beings to date.

</p>

</div>



<div id="info_a_edge22_92" class= "paragraph">

<h3>paragraph source</h3>

<p>until now, a total of seven human covs have been identified, namely hcov-229e, hcov-nl63, hcov-oc43, hcov-hku1, sars-cov, mers-cov, and, recently, 2019-ncov (1-3, 6-8). the former two human covs belong to the genus -cov, and the latter five human covs belong to the genus -cov. three recently identified human covs, that is, sars-cov, mers-cov, and 2019-ncov, have been recognized as zoonotic viruses, which transmit between animals and human. recent studies revealed that sars-cov was transmitted from civet cats to humans, mers-cov from dromedary camel, and 2019-ncov (probably) from pangolin (1, 2, (6) (7) (8) (9) . recent investigations indicated that bats were most probably the natural reservoir of sars-cov, mers-cov, and 2019-ncov (1, 6, (9) (10) (11) . genome sequence analysis revealed that 2019-ncov was distinct from sars-cov (about 79% identity) and mers-cov (about 50% identity) yet more closely related to sars-like-covs (about 88% identity) in bats (10, 11) .

</p>

</div>



<div id="info_a_edge22_93" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses mainly cause respiratory and gastrointestinal tract infections and are genetically classified into four major genera: alphacoronavirus, betacoronavirus, gammacoronavirus, and deltacoronavirus (li, 2016) . the former two genera primarily infect mammals, whereas the latter two predominantly infect birds (tang et al., 2015) . six kinds of human covs have been previously identified. these include hcov-nl63 and hcov-229e, which belong to the alphacoronavirus genus; and hcov-oc43, hcov-hku1, severe acute respiratory syndrome coronavirus (sars-cov), and middle east respiratory syndrome coronavirus (mers-cov), which belong to the beta-coronavirus genus (tang et al., 2015) . coronaviruses did not attract worldwide attention until the 2003 sars pandemic, followed by the 2012 mers and, most recently, the 2019-ncov outbreaks (china cdc, 2020; song et al., 2019) . sars-cov and mers-cov are considered highly pathogenic (cui et al., 2019) , and it is very likely that both sars-cov and mers-cov were transmitted from bats to palm civets (guan et al., 2003) or dromedary camels (drosten et al., 2014) , and finally to humans (cui et al., 2019) .

</p>

</div>



<div id="info_a_edge22_94" class= "paragraph">

<h3>paragraph source</h3>

<p>it was reported that 27 of the first 41 infected patients had been exposed to the huanan seafood market [18] . thus, it was believed that the new coronavirus originated from the huanan seafood market in wuhan and spread from animal hosts to humans in the process of wildlife trade, transportation, slaughter, and trade. bats have the most variety of coronaviruses in their bodies and are the hosts of many kinds of coronaviruses, such as the sars-covand the mers-cov [19] . the sars-cov and the mers-cov are considered highly pathogenic, and it is very likely that the sars-cov was transmitted from bats to palm civets and the mers-cov was transmitted from bats to dromedary camels and finally to humans [20, 21] . given the high sequence similarity between the sars-cov-2 and the sars-like bat covs from hipposideros bats in china, the natural host of the sars-cov-2 may be the hipposideros bat. the discovery that pangolin coronavirus genomes have 85.5% to 92.4% sequence similarity to sars-cov-2 suggests pangolins should be considered as possible hosts in the emergence of sars-cov-2 [22] .

</p>

</div>



<div id="info_a_edge22_95" class= "paragraph">

<h3>paragraph source</h3>

<p>three emerging, highly pathogenic human covs are sars-cov, middle east respiratory syndrome (mers)-cov, and covid-19 virus, which was previously named 2019-ncov by the world health organization (who), and is also known as hcov-19 or sars-cov-2 [1] . atypical pneumonia (sars) was first reported from guangdong province, china in late 2002. sars caused a global pandemic in 2003 with approximately 10% (774/8098) case fatality rate (cfr) [2] . sars-cov has not circulated in humans since 2004. mers-cov was first reported from saudi arabia in 2012 and has continued to infect humans with limited human-to-human transmission, leading to a cfr of approximately 34.4% (858/2494) in 27 countries, according to the most recent who report i . both sars-cov and mers-cov are zoonotic viruses. they use bats as their natural reservoirs and transmit from bats to intermediate hosts (e.g., palm civets for sars-cov, dromedary camels for mers-cov), leading to infection in humans [2, 3] .

</p>

</div>



<div id="info_a_edge23_131" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<div id="info_a_edge24_130" class= "paragraph">

<h3>paragraph source</h3>

<p>niaid [14] and cdc [15] also publish lists of pathogens of priority which comprise bacteria and viruses, but categorize these into three groups depending on pathogenicity, accessibility and the availability of vaccines and therapies. all the viruses listed by the who also occur on these lists, alongside bacterial pathogens of concern. one of these is yersinia pestis, causative of bubonic and pneumonic plague, which is recognized by all three bodies (who, cdc, niaid) as a current priority following the exceptionally large and serious outbreak between september 2017 to april 2018 in madagascar [16] , where the disease is endemic. in addition, niaid recognizes the added threat to human health posed by the acquisition of amr by pathogens and who has also published a priority list [17] of bacterial species for which r&d is required to develop new antibiotics (table 2) . globally, there are many regions of endemic disease which are maintained by reservoirs of zoonotic pathogens in the local wild animal species. these pathogens comprise viral, bacterial and rickettsial species and some have complex lifecycles, infecting an environmental (e.g. standing water) or animal reservoir and then being transmitted either by direct contact with man or indirectly, via an insect (e.g. mosquito) or mammalian (e.g. bat, civet, camel) vector, to man, from which they may be spread by human-tohuman transmission. (fig. 1 ). an example of a serious and widespread bacterial infection is mosquito-transmitted plasmodium falciparum, causing malaria in many tropical regions. malaria causes significant morbidity and co-morbidity in these regions in which it is endemic. due to the complex life cycle of the causative bacterium, it has been challenging to achieve a vaccine for malaria. the most advanced candidate is the rts,s/as01 vaccine [18] , which is undergoing a pilot implementation in three countries in sub-saharan africa, with a view to determining its impact on disease prior to a wider implementation [19] . despite the availability of approved vaccines [20, 21] , typhoid fever and cholera remain significantly debilitating enteric diseases in regions of the world where hygienic living conditions are poor and there is little access to health care. both infections are caused by bacteria (salmonella typhi and vibrio cholera, respectively) which exist in contaminated water and food. tuberculosis (tb) is a respiratory infection which is widespread globally. caused by mycobacterium tuberculosis, and transmitted to man from zoonotic reservoirs (badgers, cattle) with a high potential for subsequent human-to-human transmission, tb is prevalent in susceptible individuals living in overcrowded conditions [22] . although the bcg vaccine has been in routine use for many years, variable efficacy has been reported, depending on region of use [23, 24] with 60-80% reported in the united kingdom but lower levels in equatorial countries. the emergence of multi-drugresistant tb (mdr tb) in recent years raises the bar for treatment of this disease and makes a high level of vaccine efficacy even more important [25] . examples of zoonotic reservoirs which maintain endemic viral diseases are numerous and are summarized in table 3 . in particular, viral endemic disease is caused by the coronaviruses (mers and sars) in saudi arabia and the eastern mediterranean countries; and by the lassa arenavirus, which is endemic in west african countries and caused a serious outbreak of lassa fever in nigeria in 2018 [26] ; other viruses which are endemic and cause viral haemorrhagic fever include dengue, which is widespread in tropical and subtropical regions worldwide [27] , the filoviruses (ebola and marburg) which have a zoonotic reservoir in bats in sub-saharan africa [28] ; and yellow fever virus, which is prevalent in africa, central and south america and the caribbean [29] . interestingly, the same mosquito (aedes aegypti) which spreads yellow fever virus also transmits the dengue, chikungunya and zika viruses [30] . chikungunya virus has a widespread distribution in africa, asia, india and south america and with occasional cases in europe, and causes a debilitating, but rarely fatal, disease [31] . zika virus emerged in brazil in 2015 [32] , although it was first detected in monkeys in uganda as early as 1947 [33] , and the first documented human case occurred in nigeria in 1954 [34] . the large brazilian outbreak of zika viral disease culminated in 2017, with thousands of cases reported [35] . rift valley fever virus (rvfv) is another zoonotic virus which is endemic in sub-saharan africa and primarily infects cattle, sheep and goats, but can be transmitted to man by mosquito bite to cause an acute fever [36] . an example of a bacterium which has been classed as a re-emerging pathogen and which is endemic in global regions is y. pestis, causative of plague. bubonic [37] [38] [39] . over thousands of years, y. pestis has evolved away from the enteric yersinia species to become a lethal flea-transmitted bacterium [40] . it is transmitted to man, typically from a zoonotic reservoir in infected rats or other rodents (e.g. ground squirrels or prairie dogs), by flea-bite to cause bubonic plague [41] (fig. 2 ). if not detected and treated, this can develop into either septicemic plague or the most serious form of all, a secondary pneumonic plague. in turn, individuals with pneumonic plague can transmit this by aerosol droplet to others, to establish a primary pneumonic plague infection. each year, a few cases of plague are also reported in the southwestern united states, where the disease has been endemic in the rodent population since the late 1800s [42] . as well as infecting the rodent population, infected fleas can spread y. pestis to other wildlife species (e.g. rabbits/hares or, rarely, domesticated animals [43] ) which, in turn, raises the potential of transmission to man by inhalation to cause a primary pneumonic plague. in endemic areas, outbreaks of plague are often associated with seasonal environmental changes, causing rodents to stray closer to human habitation. the plague outbreak in madagascar during 2017/18 was particularly serious, with an estimated 2671 cases and 239 deaths (89% fatality rate) [44] . this outbreak was approximately sixfold greater than usual, with an unusual predominance of pneumonic, rather than bubonic plague.

</p>

</div>



<div id="info_a_edge25_96" class= "paragraph">

<h3>paragraph source</h3>

<p>humans are nowadays very often in contact with pets or farm animals such as cattle, pigs, poultry, and horses. although there are many infectious diseases typically found in people working with livestock, according to the microorganisms-driven genetic selection theory humans should have less to fear from livestock than from wild animals regarding the risk of deadly zoonotic diseases since they shared the same ecosystem with livestock for millennia. this does not mean humans should not remain vigilant (this is the reason why milk pasteurization exists and why meat must be cooked before consumption) and not worry about pathogens abnormally present in their livestock because of the interconnectedness of different ecosystems. despite veterinary figure 3 </p>

</div>



<div id="info_a_edge26_97" class= "paragraph">

<h3>paragraph source</h3>

<p>the three horses inoculated with 1 ml of the vaccine did not exhibit any adverse reaction during this study. in contrast, two out of the three horses inoculated with 2 ml of the vaccine exhibited swelling at the inoculation site after the second vaccination. none of the horses developed a fever after the vaccinations. administration of more than 2 ml of the vaccine to horses would likely increase the risk of adverse reactions. as described above, a significant increase in antibody titers against ecov was observed from 14 dpi irrespective of whether 1 or 2 ml was administered. additionally, the differences in antibody titers against ecov at each dpi from 14 dpi were less than twofold between horses inoculated with 1 and 2 ml. these results suggest that inoculation of 1 ml is suitable for horses as well as for cattle. although horse no. 5 had no detectable antibodies against ecov before vaccination, the horse already had antibodies against bcov (table 1) . that horse was born and had been kept at a farm that reared cows before coming to our facility. in saudi arabia, dromedary camel coronavirus hku23 was detected in apparently healthy horses kept at facilities that reared camels, sheep, goats, and chickens [5] . hku23, which is closely related to bcov, is endemic in camels of the middle east [16] and the hku23-positive horses frequently came into contact with camels and other animals [5] . hku23 may have been transmitted from infected camels to those horses. horse no. 5 in the present study may have also been in contact with infected cows, and bcov may have been transmitted from infected cows to the horse. however, because there is no epidemiological information, it is unknown whether there were in fact bcov-infected cows at the farm or whether horse no. 5 had shown any clinical signs.

</p>

</div>



<div id="info_a_edge27_98" class= "paragraph">

<h3>paragraph source</h3>

<p>coronaviruses (covs) are a group of viruses that co-infect humans and other vertebrate animals. cov infections affect the respiratory, gastrointestinal, liver, and central nervous systems of humans, livestock, birds, bats, mice, and many other wild animals [1,2,3]. for example, severe acute respiratory syndrome (sars) in 2002 and the middle east respiratory syndrome (mers) in 2012 were both coronaviruses that transmitted from animals to humans [4,5]. the source of unexplained pneumonia was first discovered in wuhan in dec, 2019, and sars-cov-2, a new coronavirus, was isolated from the respiratory epithelium of patients. it belongs to a new evolutionary branch within the cov. on feb. 11th, 2020, the new coronavirus was officially renamed sars-cov-2 from 2019-ncov [6]. the disease caused by sars-cov-2 was called coronavirus disease 2019 (covid-19) [7]. according to the data released by the national health commission of the peoples republic of china, sars-cov-2 was most likely transmitted from wild bats to humans, and all the above three covs can transmit from person to person [8,9,10]. sars-cov-2 shares a highly similar gene sequence and behavior pattern with sars-cov [11]. this paper summarized the similarities and differences between sars-cov-2 and sars-cov, both of which cause major disease outbreaks in china and worldwide, which will provide comprehensive reference for epidemic prevention.

</p>

</div>



<div id="info_a_edge28_100" class= "paragraph">

<h3>paragraph source</h3>

<p>there are four possible routes of mers-cov transmission in humans and camels, namely camel-to-human, human-tohuman, camel-to-camel, and human-tocamel ( figure 1 ). the former three routes have been well accepted [3, 4] . in the sabir et al. study [1] , the high prevalence again suggests that mers-cov is enzootic in dromedary camels and could be transmitted among camels. however, the humanto-camel transmission is theoretically possible but largely ignored [5] , because of the obsession of animals as sources of human viruses. recent studies reveal influenza a virus can be transmitted from human to a range of animal species (especially swine), a phenomenon known as reverse zoonosis [6] . for another cov, severe acute respiratory syndrome-associated coronavirus (sars-cov), reverse zoonosis has been reported in cats and pigs [7] . when inspecting sabir et al.'s phylogenetic trees of mers-cov (figures 1, s2 , and s3 in [1] ), we found many viruses of camel origin fell within the diversity of human viruses. there are two possible explanations for this phylogenetic pattern: (i) it is artifact due to sampling bias, if many camel viral strains are not sampled. if this is the case, we must miss substantial diversity of mers-cov and should expand our surveillance in camels to fully capture the mers-cov diversity. (ii) mers-cov could be transmitted from humans to camels, which means that humans serve as a source of mers-cov for camels. the role of camels in generating and maintaining mers-cov diversity has been repeatedly emphasized [1] . if human-to-camel transmission occurs, it requires us to refine our concept of the definition of a 'breeding ground' for mers-cov, which has important practical implications for developing vaccination strategies and control measures.

</p>

</div>



<div id="info_a_edge29_103" class= "paragraph">

<h3>paragraph source</h3>

<p>the references were selected by the authors from their knowledge of published work. zoonotic, transmission. for example, genetic analyses have suggested that hiv infection was transmitted to human beings from a common subspecies of chimpanzee (pan troglodytes troglodytes). 5 another wellpublicised example of zoonotic transmission is severe acute respiratory syndrome, caused by the transmission of a new coronavirus to people, initially in southern china and probably from small mammals, which are the natural hosts of the severe-acute-respiratory-syndrome coronavirus. 6 after the virus first appeared in human beings, there was an explosive short epidemic. since clinical symptoms preceded the peak of transmission of this virus, outbreaks were restricted by quarantine measures in china, canada, and other countries. many epidemiologists predict similar types of outbreaks in the future with emerging viruses.

</p>

</div>



<div id="info_a_edge30_99" class= "paragraph">

<h3>paragraph source</h3>

<p>horseshoe bats ( rhinolophus ) are hypothesized to be the reservoir for the coronavirus that causes sars. when a related virus was found in bats, it was presumed to have been transmitted from bats to civets caged in proximity in asian animal markets. unfortunately, the hypothesis was presented as fact (fenton et al. 2006 ) , triggering an onslaught of sensational media warnings of dangerous bats (mckie 2005 ) . because horseshoe bats are seldom sold in animal markets, the initially hypothesized route of transmission is unlikely. furthermore, janies et al. ( 2008 ) concluded that though horseshoe bats possibly were involved in the early evolution of sars, it most recently appears to have spread from humans to civets and pigs.

</p>

</div>



<div id="info_a_edge31_104" class= "paragraph">

<h3>paragraph source</h3>

<p>is the (which was not peer-reviewed) the copyright holder for this preprint . https://doi.org/10.1101/2020.04.11.20061143 doi: medrxiv preprint of cases even doubled in some cases relative to the full-lockdown configuration. this increase in cases can be attributed to the increased mobility of infections over larger distances enabled by the few human agents still moving across the population in partial lockdowns, while also preserving the mosquitoes' advantage in being able to infect more humans. this result suggests that additional care must be taken to see that essential workers who continue to be mobile in lockdowns do not carry mosquito-borne diseases, and that others who may carry such infections are not allowed to move around. we further analyzed how the impact of lockdowns varies across three mosquito-borne diseases: chikungunya, dengue and malaria. we found that dengue outbreaks are more severe than chikungunya in any situation even though they share the same vector, aedes aegypti. this is because the dengue virus is transmissible from humans to mosquitoes even during late intrinsic incubation stage and early recovered stage. this in turn allows movement of infections by means of humans who are infectious but not showing symptoms (ironically in a manner similar to covid-19). we also observed that the effects of lockdowns are less pronounced on malaria outbreaks, as they are spread by female anopheles mosquitoes which are generally active during the night, a time where most humans are stationary with or without lockdowns.

</p>

</div>



<div id="info_a_edge32_101" class= "paragraph">

<h3>paragraph source</h3>

<p>in 2002, a new infectious disease was found in the southern part of china which had high mortality and morbidity and was termed ''severe acute respiratory syndrome'' (sars). it affected 8096 people and led 774 patients to death around the world, within a very short span of time [1] . several studies have shown that sars is caused by a new member of the coronavirus family. the sars coronavirus (sars-cov) was supposed to be transmitted from wild animals [2] [3] [4] . this hypothesis was then supported by the identification of a sars-cov-like virus in civet cats, sold in markets in south china. this isolate had more than 99% sequence identity with sars-cov, indicating that the virus was recently transferred from animals to human beings [5] . however, according to a recent report there is no direct evidence that the civet cat virus is the origin of the sars-cov [6] . although unlikely, the possibility that sars-cov positive animals were infected from humans cannot be formally excluded, and it was indeed reported that sars-cov was transmitted from man to pig [7] . the mechanism how the pathogen broke the barrier between its natural reservoir and man is still unclear [8] .

</p>

</div>



<div id="info_a_edge32_102" class= "paragraph">

<h3>paragraph source</h3>

<p>horseshoe bats ( rhinolophus ) are hypothesized to be the reservoir for the coronavirus that causes sars. when a related virus was found in bats, it was presumed to have been transmitted from bats to civets caged in proximity in asian animal markets. unfortunately, the hypothesis was presented as fact (fenton et al. 2006 ) , triggering an onslaught of sensational media warnings of dangerous bats (mckie 2005 ) . because horseshoe bats are seldom sold in animal markets, the initially hypothesized route of transmission is unlikely. furthermore, janies et al. ( 2008 ) concluded that though horseshoe bats possibly were involved in the early evolution of sars, it most recently appears to have spread from humans to civets and pigs.

</p>

</div>



<div id="info_a_edge33_105" class= "paragraph">

<h3>paragraph source</h3>

<p>there are four possible routes of mers-cov transmission in humans and camels, namely camel-to-human, human-tohuman, camel-to-camel, and human-tocamel ( figure 1 ). the former three routes have been well accepted [3, 4] . in the sabir et al. study [1] , the high prevalence again suggests that mers-cov is enzootic in dromedary camels and could be transmitted among camels. however, the humanto-camel transmission is theoretically possible but largely ignored [5] , because of the obsession of animals as sources of human viruses. recent studies reveal influenza a virus can be transmitted from human to a range of animal species (especially swine), a phenomenon known as reverse zoonosis [6] . for another cov, severe acute respiratory syndrome-associated coronavirus (sars-cov), reverse zoonosis has been reported in cats and pigs [7] . when inspecting sabir et al.'s phylogenetic trees of mers-cov (figures 1, s2 , and s3 in [1] ), we found many viruses of camel origin fell within the diversity of human viruses. there are two possible explanations for this phylogenetic pattern: (i) it is artifact due to sampling bias, if many camel viral strains are not sampled. if this is the case, we must miss substantial diversity of mers-cov and should expand our surveillance in camels to fully capture the mers-cov diversity. (ii) mers-cov could be transmitted from humans to camels, which means that humans serve as a source of mers-cov for camels. the role of camels in generating and maintaining mers-cov diversity has been repeatedly emphasized [1] . if human-to-camel transmission occurs, it requires us to refine our concept of the definition of a 'breeding ground' for mers-cov, which has important practical implications for developing vaccination strategies and control measures.

</p>

</div>



<div id="info_a_edge34_134" class= "paragraph">

<h3>paragraph source</h3>

<p>a seminal paper published in 2008 listed 335 new human pathogens discovered between 1940 and 2004 [1] . the majority (60.3%) of these emerging infectious diseases (eid) originated from (wild) animal reservoirs, and approximately one in five was transmitted from animal reservoir hosts to humans by disease vectors (ticks, mosquitos, midges) [1] . pathogen discovery programs sampling wildlife hosts that are considered major reservoirs for eid (rodents, bats) have identified tremendous enzootic virus diversity, confirming their potential as sources of novel human pathogens [2] . since 2008, the discovery of severe fever with thrombocytopenia virus and middle east respiratory syndrome coronavirus (mers-cov), as well as unusual outbreaks of zika virus, yellow fever and ebola highlight the importance of demographic change, global travel and trade, and possibly climate change as drivers for emergence [3e7] (http://www.promedmail.org/, archive no. 20140322.2349696 2014-03 -22) . while the field is dominated by viruses from wildlife reservoirs, the q fever outbreak in the netherlands, the enterohemorrhagic escherichia coli outbreak in germany and the ongoing vast cholera outbreak in yemen are examples of bacterial pathogens that have expanded massivelydwhich is included in the definition of an eid (box 1) [8e10]. there is broad consensus that we need to prepare for more eid in the future.

</p>

</div>



<div id="info_a_edge35_132" class= "paragraph">

<h3>paragraph source</h3>

<p>toxoplasma gondii may cause self-limiting small bowel diarrhea but typically causes no disease in cats. however, transplacentally or lactationally infected kittens and immune-suppressed older cats can show severe systemic signs. 60 in a study of 100 cases with histologically confirmed toxoplasmosis, 76.7% of 86 lung tissue samples assessed had organisms present, and in the 36 cats considered to have generalized toxoplasmosis, 26 had predominantly pulmonary lesions. 59 retrospective studies of feline lrt infection have found t. gondii in 1 of 20 cats (5%) 78 and 6 of 245 cases (2.4%). 17 diagnosis is achieved by recognition of toxoplasma tachyzoites, which can be found in bal fluid 14 or fna of affected lung tissue. 219, 235 diffuse interstitial to alveolar patterns are typically described, 14,219,235 but a bronchial influence is also possible 14 (figure 30-20) . serology can aid diagnosis. 60 recognition of toxoplasma in an adult cat should instigate investigations to determine a cause of immune suppression, which is often fiv related 50,78 but can be iatrogenic (e.g., due to administration of corticosteroids or cyclosporin). 14,77 treatment is covered in detail in chapter 23 but is typically with clindamycin at 12.5 to 25 mg/kg orally every 12 hours for 2 weeks 60 ; combination therapy with pyrimethamine at 0.25 to 0.5 mg/kg orally every 12 hours may help the prognosis. 14 supportive care such as oxygen therapy maintenance of fluid and electrolyte balance is also important. successful treatment of clinical feline toxoplasmosis has been described infrequently. 14, 59, 143 heartworm despite prevalence studies demonstrating a worldwide distribution and recognizing heartworm infection in up to 18% of cats tested,* the prevalence rate being approximately 5% to 10% of that found in dogs, 84, 150, 153 * references 31, 71, 84, 107, 138, 150, 151, 155, 158, 197, 269, 285. prevalence being found similar to that of fiv and felv infection, 150, 151, 158 feline heartworm disease remains underdiagnosed in general practice. 30, 195 this is likely because clinical signs vary dramatically from no signs to sudden death and definitive diagnosis can be difficult. 20, 30, 153, 196 heartworm disease is caused by the filarioid nematode dirofilaria immitis, which is transmitted by mosquitos and for which dogs are the usual definitive host. therefore a prerequisite for heartworm infection is a climate with adequate temperature and humidity to support a viable mosquito population and allow maturation of larvae within the intermediate host. the sexually dimorphic adult heartworm mate within an infected dog's pulmonary artery, producing immature forms called microfilariae. circulating microfilariae are ingested by mosquitoes feeding on infected dogs. within mosquitoes, microfilariae transform into larval stages. there are five larval stages (l1 to l5); l1 to l3 transformations occur within the mosquito, and l3 is the form that is transmitted from mosquitos to dogs and cats. maturation from l3 to other larval stages occurs in subcutaneous tissues and peripheral veins of mammalian hosts; immature worms in peripheral veins are carried in the bloodstream to and through the heart, with adult heartworms ultimately residing at the caudal pulmonary arteries. 1 there are significant differences between feline and canine heartworm disease because the parasite is only partially adapted to the cat. the cat is susceptible to heartworm infection but more resistant than the dog. the cat represents a dead-end host because it cannot act as a reservoir for infection. the prepatent period in the cat is 7 to 8 months (1 to 2 months longer than in dogs). 169 in dogs, most juvenile worms mature into adults that can live for 5 to 7 years, whereas in cats most juvenile worms die shortly after arriving in the pulmonary arteries, instigating a severe vascular and parenchymal inflammatory response. pulmonary lesions may be long lasting. the clinical response in the cat has been termed heartworm-associated respiratory disease (hard). in a small percentage of cats, a few worms develop to mature adults that can live for 2 to 3 years. 195 cats with adult heartworms develop pulmonary changes characterized by intimal proliferation, eosinophilic endarteritis, intimal fibrosis, and disruption of the internal elastic lamina. arterial wall changes can lead to pulmonary hypertension with resultant pulmonary arterial distention. 173 damage can be caused by juvenile heartworms without development to the adult stage, resulting in increase in thickness of the pulmonary arterial wall that in turn causes occlusion. 26, 56 there have been suggestions that wolbachia pipientis, a symbiotic gram-negative intracellular bacterium harbored by d. immitis induces further pulmonary pathology. 57 a strong antibody response against wolbachia surface protein has american heartworm society, the primary reasons for heartworm testing in cats are as follows:

</p>

</div>



<div id="info_a_edge36_133" class= "paragraph">

<h3>paragraph source</h3>

<p>toxoplasma gondii may cause self-limiting small bowel diarrhea but typically causes no disease in cats. however, transplacentally or lactationally infected kittens and immune-suppressed older cats can show severe systemic signs. 60 in a study of 100 cases with histologically confirmed toxoplasmosis, 76.7% of 86 lung tissue samples assessed had organisms present, and in the 36 cats considered to have generalized toxoplasmosis, 26 had predominantly pulmonary lesions. 59 retrospective studies of feline lrt infection have found t. gondii in 1 of 20 cats (5%) 78 and 6 of 245 cases (2.4%). 17 diagnosis is achieved by recognition of toxoplasma tachyzoites, which can be found in bal fluid 14 or fna of affected lung tissue. 219, 235 diffuse interstitial to alveolar patterns are typically described, 14,219,235 but a bronchial influence is also possible 14 (figure 30-20) . serology can aid diagnosis. 60 recognition of toxoplasma in an adult cat should instigate investigations to determine a cause of immune suppression, which is often fiv related 50,78 but can be iatrogenic (e.g., due to administration of corticosteroids or cyclosporin). 14,77 treatment is covered in detail in chapter 23 but is typically with clindamycin at 12.5 to 25 mg/kg orally every 12 hours for 2 weeks 60 ; combination therapy with pyrimethamine at 0.25 to 0.5 mg/kg orally every 12 hours may help the prognosis. 14 supportive care such as oxygen therapy maintenance of fluid and electrolyte balance is also important. successful treatment of clinical feline toxoplasmosis has been described infrequently. 14, 59, 143 heartworm despite prevalence studies demonstrating a worldwide distribution and recognizing heartworm infection in up to 18% of cats tested,* the prevalence rate being approximately 5% to 10% of that found in dogs, 84, 150, 153 * references 31, 71, 84, 107, 138, 150, 151, 155, 158, 197, 269, 285. prevalence being found similar to that of fiv and felv infection, 150, 151, 158 feline heartworm disease remains underdiagnosed in general practice. 30, 195 this is likely because clinical signs vary dramatically from no signs to sudden death and definitive diagnosis can be difficult. 20, 30, 153, 196 heartworm disease is caused by the filarioid nematode dirofilaria immitis, which is transmitted by mosquitos and for which dogs are the usual definitive host. therefore a prerequisite for heartworm infection is a climate with adequate temperature and humidity to support a viable mosquito population and allow maturation of larvae within the intermediate host. the sexually dimorphic adult heartworm mate within an infected dog's pulmonary artery, producing immature forms called microfilariae. circulating microfilariae are ingested by mosquitoes feeding on infected dogs. within mosquitoes, microfilariae transform into larval stages. there are five larval stages (l1 to l5); l1 to l3 transformations occur within the mosquito, and l3 is the form that is transmitted from mosquitos to dogs and cats. maturation from l3 to other larval stages occurs in subcutaneous tissues and peripheral veins of mammalian hosts; immature worms in peripheral veins are carried in the bloodstream to and through the heart, with adult heartworms ultimately residing at the caudal pulmonary arteries. 1 there are significant differences between feline and canine heartworm disease because the parasite is only partially adapted to the cat. the cat is susceptible to heartworm infection but more resistant than the dog. the cat represents a dead-end host because it cannot act as a reservoir for infection. the prepatent period in the cat is 7 to 8 months (1 to 2 months longer than in dogs). 169 in dogs, most juvenile worms mature into adults that can live for 5 to 7 years, whereas in cats most juvenile worms die shortly after arriving in the pulmonary arteries, instigating a severe vascular and parenchymal inflammatory response. pulmonary lesions may be long lasting. the clinical response in the cat has been termed heartworm-associated respiratory disease (hard). in a small percentage of cats, a few worms develop to mature adults that can live for 2 to 3 years. 195 cats with adult heartworms develop pulmonary changes characterized by intimal proliferation, eosinophilic endarteritis, intimal fibrosis, and disruption of the internal elastic lamina. arterial wall changes can lead to pulmonary hypertension with resultant pulmonary arterial distention. 173 damage can be caused by juvenile heartworms without development to the adult stage, resulting in increase in thickness of the pulmonary arterial wall that in turn causes occlusion. 26, 56 there have been suggestions that wolbachia pipientis, a symbiotic gram-negative intracellular bacterium harbored by d. immitis induces further pulmonary pathology. 57 a strong antibody response against wolbachia surface protein has american heartworm society, the primary reasons for heartworm testing in cats are as follows:

</p>

</div>



<div id="info_a_edge37_135" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<div id="info_a_edge38_136" class= "paragraph">

<h3>paragraph source</h3>

<p>a seminal paper published in 2008 listed 335 new human pathogens discovered between 1940 and 2004 [1] . the majority (60.3%) of these emerging infectious diseases (eid) originated from (wild) animal reservoirs, and approximately one in five was transmitted from animal reservoir hosts to humans by disease vectors (ticks, mosquitos, midges) [1] . pathogen discovery programs sampling wildlife hosts that are considered major reservoirs for eid (rodents, bats) have identified tremendous enzootic virus diversity, confirming their potential as sources of novel human pathogens [2] . since 2008, the discovery of severe fever with thrombocytopenia virus and middle east respiratory syndrome coronavirus (mers-cov), as well as unusual outbreaks of zika virus, yellow fever and ebola highlight the importance of demographic change, global travel and trade, and possibly climate change as drivers for emergence [3e7] (http://www.promedmail.org/, archive no. 20140322.2349696 2014-03 -22) . while the field is dominated by viruses from wildlife reservoirs, the q fever outbreak in the netherlands, the enterohemorrhagic escherichia coli outbreak in germany and the ongoing vast cholera outbreak in yemen are examples of bacterial pathogens that have expanded massivelydwhich is included in the definition of an eid (box 1) [8e10]. there is broad consensus that we need to prepare for more eid in the future.

</p>

</div>



<div id="info_a_edge39_106" class= "paragraph">

<h3>paragraph source</h3>

<p>we next examined how the sars-cov-2 codon pair usage compares with other coronaviruses and to its current host. codon pair data inherently contain the codon usage data and therefore are better suited than codon usage data for this type of comparison. as expected, sars-cov-2 codon pair usage closely resembles the codon usage of the coronoviridae family, while it is quite distinct from the codon pair usage of the human genome (table 1) . bat (chiroptera) and pangolin (pholidota) from which the virus may have been transmitted to humans, as well as dog (canis lupus familiaris) to which the virus is feared may be transmitted next, were included in the analysis. we find that these species have a similar codon usage when compared with human; therefore, viral tropism cannot be inferred based on codon usage data alone (table 1) . since sars-cov-2 infects bronchial epithelial cells and type ii pneumocytes and our recent findings show that transcriptomic tissue-specific codon pair usage can vary greatly from genomic codon pair usage [24] , we also examined the transcriptomic codon pair usage of the lung and how it compares with the sars-cov-2 codon pair usage. rather surprisingly, the codon usage in the lung was more distinct from sars-cov-2 codon usage than the homo sapiens genomic codon usage. the transcriptome codon pair usage of kidney and small intestine, tissues that are also susceptible to the infection, are similarly distant from sars-cov-2 (table 1) .

</p>

</div>



<div id="info_a_edge39_107" class= "paragraph">

<h3>paragraph source</h3>

<p>the life-threatening pneumonia outbreak started as an epidemic within wuhan city, china. on december 31, 2019, the wuhan municipal health commission reported critically ill cases of viral pneumonia. later, the international committee on taxonomy of viruses (ictv) identified the virus as sars-cov-2 [26] . the chinese government then mandated a quarantine to control the epidemic stage and to determine the effectiveness of a quarantine against the virus [27] . however, sars-cov-2 was transmitted rapidly, quickly crossing borders of various countries. thus, the number of established cases climbed quickly, causing the who to announce covid-19 a global pandemic disease on march 11, 2020. initially, china had the most confirmed cases, followed by north america and europe. italy and the us, however, surpassed china in terms of the number of established cases. thus, at this stage of vigorous spread, it is essential to determine the mechanism of viral infection to control and treat the disease. until now, it was believed that bat, especially rhinolophus affinis, was the natural host of the virus [7] . it is still uncertain how this virus was transmitted from bat to human, although some research has suggested that pangolin (manis javanica) could be the intermediate host [14, 28, 29] . the aggressive intervention steps were taken by governments, such as isolating and quarantining latent patients, have reduced the contact rate and effectively decreased the peak for covid-19 cases.

</p>

</div>



<div id="info_a_edge39_108" class= "paragraph">

<h3>paragraph source</h3>

<p>until now, a total of seven human covs have been identified, namely hcov-229e, hcov-nl63, hcov-oc43, hcov-hku1, sars-cov, mers-cov, and, recently, 2019-ncov (1-3, 6-8). the former two human covs belong to the genus -cov, and the latter five human covs belong to the genus -cov. three recently identified human covs, that is, sars-cov, mers-cov, and 2019-ncov, have been recognized as zoonotic viruses, which transmit between animals and human. recent studies revealed that sars-cov was transmitted from civet cats to humans, mers-cov from dromedary camel, and 2019-ncov (probably) from pangolin (1, 2, (6) (7) (8) (9) . recent investigations indicated that bats were most probably the natural reservoir of sars-cov, mers-cov, and 2019-ncov (1, 6, (9) (10) (11) . genome sequence analysis revealed that 2019-ncov was distinct from sars-cov (about 79% identity) and mers-cov (about 50% identity) yet more closely related to sars-like-covs (about 88% identity) in bats (10, 11) .

</p>

</div>



<div id="info_a_edge40_118" class= "paragraph">

<h3>paragraph source</h3>

<p>in addition to rodents, we also measured the sads-cov susceptibility of cell lines from humans, monkeys, chickens, and dogs, revealing a remarkably broad spectrum of tropism (table 1 and fig. 1) . as for the ability of sads-cov to grow efficiently in human cell lines, we should not underestimate the risk that this bat-origin cov may "jump" from pigs to humans. it is noteworthy that camel workers with high rates of exposure to camel nasal and oral secretions had evidence of mers-cov infection (36) . considering that sars-cov and mers-cov originated from bats and spread from one species to another through intermediate hosts (civets and camels, respectively), sads-cov may pose a similar risk to human health through transmission from pigs or other susceptible hosts.

</p>

</div>



<div id="info_a_edge41_109" class= "paragraph">

<h3>paragraph source</h3>

<p>a large number of molecular biologic and sequencing studies established that the coronaviruses share a common genetic organization and mode of replication. infected cells contain a set of 6 to 9 virus-specific polyadenylated rnas that make up a 3 co-terminal nested set, with the largest member of this set being the genome rna and the remaining rnas representing subgenomic mrnas. the subgenomic rnas encode the structural proteins enumerated above in a characteristic order: he (if present)-s-e-m-n. in addition to these structural genes, coronavirus subgenomic rnas encode accessory proteins that are not essential for viral replication. they generally modulate cellular responses to infection and are specific for each coronavirus subgroup. the replicase proteins are encoded in the 5 two-thirds of the genome. this region contains two very long partially overlapping open reading frames that are translated by a ribosomal frame shifting mechanism. the 750-kda primary translation product is cotranslationally autocatalytically cleaved into 16 nonstructural proteins (nsps) that encode the machinery needed for rna replication and transcription. many of the coronaviruses are associated with respiratory or enteric diseases, although some viruses replicate and cause disease at other sites, including the liver, spleen, and central nervous system. most of the coronaviruses are relatively species specific, only infecting their natural host and cell cultures derived from that host or a closely related host species. however, there are several examples of coronaviruses jumping species, with sars-cov being the most well known example. sars-cov is thought to have originated in bats, where sars-like covs have been identified, and then transmitted to civets and raccoon dogs in wild animal markets in southern china, from which they were subsequently transmitted to humans (lau et al., 2010) . during this chain of transmission, several key adaptations to the new host occurred (li et al., 2005a,b) . ccov and fecov provide another exception to the high degree of species specificity of coronaviruses, in that both viruses are able to grow in feline cells. phylogenetic analysis of the group ia coronaviruses, ccov, fecov, and tgev, suggest a complex set of recombination events has given rise to these viruses from a common ancestor, likely explaining the ability of both ccov and fecov to infect feline cells (wesseling et al., 1994; lorusso et al., 2008) . the species specificity of coronaviruses is controlled by the ectodomain of the s protein, and can be altered by creating chimeric s proteins that contain the ectodomain of a distantly related cov (kuo et al., 2000) .

</p>

</div>



<div id="info_a_edge42_137" class= "paragraph">

<h3>paragraph source</h3>

<p>niaid [14] and cdc [15] also publish lists of pathogens of priority which comprise bacteria and viruses, but categorize these into three groups depending on pathogenicity, accessibility and the availability of vaccines and therapies. all the viruses listed by the who also occur on these lists, alongside bacterial pathogens of concern. one of these is yersinia pestis, causative of bubonic and pneumonic plague, which is recognized by all three bodies (who, cdc, niaid) as a current priority following the exceptionally large and serious outbreak between september 2017 to april 2018 in madagascar [16] , where the disease is endemic. in addition, niaid recognizes the added threat to human health posed by the acquisition of amr by pathogens and who has also published a priority list [17] of bacterial species for which r&d is required to develop new antibiotics (table 2) . globally, there are many regions of endemic disease which are maintained by reservoirs of zoonotic pathogens in the local wild animal species. these pathogens comprise viral, bacterial and rickettsial species and some have complex lifecycles, infecting an environmental (e.g. standing water) or animal reservoir and then being transmitted either by direct contact with man or indirectly, via an insect (e.g. mosquito) or mammalian (e.g. bat, civet, camel) vector, to man, from which they may be spread by human-tohuman transmission. (fig. 1 ). an example of a serious and widespread bacterial infection is mosquito-transmitted plasmodium falciparum, causing malaria in many tropical regions. malaria causes significant morbidity and co-morbidity in these regions in which it is endemic. due to the complex life cycle of the causative bacterium, it has been challenging to achieve a vaccine for malaria. the most advanced candidate is the rts,s/as01 vaccine [18] , which is undergoing a pilot implementation in three countries in sub-saharan africa, with a view to determining its impact on disease prior to a wider implementation [19] . despite the availability of approved vaccines [20, 21] , typhoid fever and cholera remain significantly debilitating enteric diseases in regions of the world where hygienic living conditions are poor and there is little access to health care. both infections are caused by bacteria (salmonella typhi and vibrio cholera, respectively) which exist in contaminated water and food. tuberculosis (tb) is a respiratory infection which is widespread globally. caused by mycobacterium tuberculosis, and transmitted to man from zoonotic reservoirs (badgers, cattle) with a high potential for subsequent human-to-human transmission, tb is prevalent in susceptible individuals living in overcrowded conditions [22] . although the bcg vaccine has been in routine use for many years, variable efficacy has been reported, depending on region of use [23, 24] with 60-80% reported in the united kingdom but lower levels in equatorial countries. the emergence of multi-drugresistant tb (mdr tb) in recent years raises the bar for treatment of this disease and makes a high level of vaccine efficacy even more important [25] . examples of zoonotic reservoirs which maintain endemic viral diseases are numerous and are summarized in table 3 . in particular, viral endemic disease is caused by the coronaviruses (mers and sars) in saudi arabia and the eastern mediterranean countries; and by the lassa arenavirus, which is endemic in west african countries and caused a serious outbreak of lassa fever in nigeria in 2018 [26] ; other viruses which are endemic and cause viral haemorrhagic fever include dengue, which is widespread in tropical and subtropical regions worldwide [27] , the filoviruses (ebola and marburg) which have a zoonotic reservoir in bats in sub-saharan africa [28] ; and yellow fever virus, which is prevalent in africa, central and south america and the caribbean [29] . interestingly, the same mosquito (aedes aegypti) which spreads yellow fever virus also transmits the dengue, chikungunya and zika viruses [30] . chikungunya virus has a widespread distribution in africa, asia, india and south america and with occasional cases in europe, and causes a debilitating, but rarely fatal, disease [31] . zika virus emerged in brazil in 2015 [32] , although it was first detected in monkeys in uganda as early as 1947 [33] , and the first documented human case occurred in nigeria in 1954 [34] . the large brazilian outbreak of zika viral disease culminated in 2017, with thousands of cases reported [35] . rift valley fever virus (rvfv) is another zoonotic virus which is endemic in sub-saharan africa and primarily infects cattle, sheep and goats, but can be transmitted to man by mosquito bite to cause an acute fever [36] . an example of a bacterium which has been classed as a re-emerging pathogen and which is endemic in global regions is y. pestis, causative of plague. bubonic [37] [38] [39] . over thousands of years, y. pestis has evolved away from the enteric yersinia species to become a lethal flea-transmitted bacterium [40] . it is transmitted to man, typically from a zoonotic reservoir in infected rats or other rodents (e.g. ground squirrels or prairie dogs), by flea-bite to cause bubonic plague [41] (fig. 2 ). if not detected and treated, this can develop into either septicemic plague or the most serious form of all, a secondary pneumonic plague. in turn, individuals with pneumonic plague can transmit this by aerosol droplet to others, to establish a primary pneumonic plague infection. each year, a few cases of plague are also reported in the southwestern united states, where the disease has been endemic in the rodent population since the late 1800s [42] . as well as infecting the rodent population, infected fleas can spread y. pestis to other wildlife species (e.g. rabbits/hares or, rarely, domesticated animals [43] ) which, in turn, raises the potential of transmission to man by inhalation to cause a primary pneumonic plague. in endemic areas, outbreaks of plague are often associated with seasonal environmental changes, causing rodents to stray closer to human habitation. the plague outbreak in madagascar during 2017/18 was particularly serious, with an estimated 2671 cases and 239 deaths (89% fatality rate) [44] . this outbreak was approximately sixfold greater than usual, with an unusual predominance of pneumonic, rather than bubonic plague.

</p>

</div>



<div id="info_a_edge43_138" class= "paragraph">

<h3>paragraph source</h3>

<p>the aristolochic acid hypothesis nevertheless fails to explain one of the defining features of ben, i.e., the geographical confinement. aristolochia plants are widely cultivated and are probably native in many places in europe, asia, and north america, places where no cases of ben or a ben-like disease have been documented.. also, even though many herbal remedies are used locally in the ben areas, it is unlikely that aristolochia leaves, seeds, or roots are ingested on such a widespread scale in the endemic locations (3) . the pungent repulsive odor of the plant makes it equally unlikely to be accidentally ingested, even on an endemic scale. in addition, the pelviureteric sclerosis found in one of the biopsied cases of chinese herb nephropathy reminds more of analgesic nephropathy than ben. viruses a viral etiology of ben was hypothesized based on the endemic character of the disease, the seasonal variation in the incidence and deaths from ben, and on the discovery of viral particles in the kidneys of some afflicted persons (55) . the high frequency of urothelial tumors in ben patients may also be explained by infection with slow oncogenic viruses. several viruses have been considered as possible agents: the west nile virus, papilloma viruses (bk and sv40) (56, 57) , coronaviruses (20) , and arenaviruses (58) . an extensive search for these viruses in ben patients has been unsuccessful so far. for example, the west nile virus (a mosquito-borne flavivirus that caused more than 300 cases of meningoencephalitis in the danube flood lands, including bucharest, romania, in 1996), is probably not involved in ben because only a few of the investigated ben patients exhibited antiviral antibodies. the same is true for the bk and sv40 viruses. although high titers of anti-bk and -sv40 antibodies were found in some of the ben patients, such antibodies are also frequently encountered in the general healthy population. coronaviruses transmitted from pigs in the endemic households were suggested as a possible cause for ben (20) . however, evidence supporting this hypothesis is lacking. more likely candidates for a viral etiology are rodent-borne viruses such as those responsible for hemorrhagic fever (hf) syndromes. some of these viruses (puumala, hantaan, and dobrava in the balkans) can cause more or less severe hf with renal syndrome in rural or urban areas. the classic hantaan disease in rural china (epidemic hf) and in korea (korean hf) has a seasonal pattern of incidence, being most common in spring or fall, possibly in relation to agricultural practices and rodent density (59) . infection in humans is acquired through aerosols of rodent urine; the disease is not transmitted from human to human. some of the features of hantaan hf are seen in ben-disease endemicity, incidence in rural settlements, and seasonal variation-but ben is primarily a chronic kidney disease without any acute episode in its evolution. in contrast, the hantaan infection usually evolves with acute renal failure. there is an analogy worth mentioning between ben and the bolivian hemorrhagic fever, an arenaviral hf disease spread by rodents (58) . as discussed by radovanovic (58) , both diseases are characterized by similarities in the topography of the disease area (flat plains, in river or stream valleys), clustering only in certain setdements and households, lack of evidence of human-to-human transmission, and the first recognition of both diseases after the beginning of malaria control programs using ddt after world war ii. although the evidence is speculative, radovanovic (58) postulates that residual ddt sprayed in the balkans in the mid-1940s led to the reduction in the number of cats, the most important natural barrier against rodents in the endemic (and nonendemic) households. as a result, a decade later, the rodent reservoir expanded and ben emerged and was described for the first time as a disease entity.

</p>

</div>



<div id="info_a_edge43_139" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<div id="info_a_edge44_140" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<div id="info_a_edge45_128" class= "paragraph">

<h3>paragraph source</h3>

<p>in areas where white-tailed deer and meningeal worm are endemic, meningeal worm prophylaxis with monthly, parenteral administration of avermectins has been recommended. 7 documentation of the development of anthelminthic resistance because of meningeal worm prophylaxis has led some farms to pursue drug-free meningeal worm prophylaxis, which may be instituted when designing housing and husbandry recommendations. meningeal worm is transmitted from white-tailed deer to camelids through arboreal snails. [7] [8] [9] as such, deer-proof fencing, introduction of guinea fowl, and the use of molluscicides can minimize transmission. removal of organic matter, in the form of leaf or wood piles, and establishment of pasture perimeters using limestone or gravel also inhibits snail and slug habitation in camelid pastures. 10 more complete assessments of parasite control measures are covered in the articles ''gastrointestinal parasitology'' by ballweber and ''neurological diseases'' by whitehead and bedenice elsewhere in this issue.

</p>

</div>



<div id="info_a_edge46_129" class= "paragraph">

<h3>paragraph source</h3>

<p>in areas where white-tailed deer and meningeal worm are endemic, meningeal worm prophylaxis with monthly, parenteral administration of avermectins has been recommended. 7 documentation of the development of anthelminthic resistance because of meningeal worm prophylaxis has led some farms to pursue drug-free meningeal worm prophylaxis, which may be instituted when designing housing and husbandry recommendations. meningeal worm is transmitted from white-tailed deer to camelids through arboreal snails. [7] [8] [9] as such, deer-proof fencing, introduction of guinea fowl, and the use of molluscicides can minimize transmission. removal of organic matter, in the form of leaf or wood piles, and establishment of pasture perimeters using limestone or gravel also inhibits snail and slug habitation in camelid pastures. 10 more complete assessments of parasite control measures are covered in the articles ''gastrointestinal parasitology'' by ballweber and ''neurological diseases'' by whitehead and bedenice elsewhere in this issue.

</p>

</div>



<div id="info_a_edge47_141" class= "paragraph">

<h3>paragraph source</h3>

<p>niaid [14] and cdc [15] also publish lists of pathogens of priority which comprise bacteria and viruses, but categorize these into three groups depending on pathogenicity, accessibility and the availability of vaccines and therapies. all the viruses listed by the who also occur on these lists, alongside bacterial pathogens of concern. one of these is yersinia pestis, causative of bubonic and pneumonic plague, which is recognized by all three bodies (who, cdc, niaid) as a current priority following the exceptionally large and serious outbreak between september 2017 to april 2018 in madagascar [16] , where the disease is endemic. in addition, niaid recognizes the added threat to human health posed by the acquisition of amr by pathogens and who has also published a priority list [17] of bacterial species for which r&d is required to develop new antibiotics (table 2) . globally, there are many regions of endemic disease which are maintained by reservoirs of zoonotic pathogens in the local wild animal species. these pathogens comprise viral, bacterial and rickettsial species and some have complex lifecycles, infecting an environmental (e.g. standing water) or animal reservoir and then being transmitted either by direct contact with man or indirectly, via an insect (e.g. mosquito) or mammalian (e.g. bat, civet, camel) vector, to man, from which they may be spread by human-tohuman transmission. (fig. 1 ). an example of a serious and widespread bacterial infection is mosquito-transmitted plasmodium falciparum, causing malaria in many tropical regions. malaria causes significant morbidity and co-morbidity in these regions in which it is endemic. due to the complex life cycle of the causative bacterium, it has been challenging to achieve a vaccine for malaria. the most advanced candidate is the rts,s/as01 vaccine [18] , which is undergoing a pilot implementation in three countries in sub-saharan africa, with a view to determining its impact on disease prior to a wider implementation [19] . despite the availability of approved vaccines [20, 21] , typhoid fever and cholera remain significantly debilitating enteric diseases in regions of the world where hygienic living conditions are poor and there is little access to health care. both infections are caused by bacteria (salmonella typhi and vibrio cholera, respectively) which exist in contaminated water and food. tuberculosis (tb) is a respiratory infection which is widespread globally. caused by mycobacterium tuberculosis, and transmitted to man from zoonotic reservoirs (badgers, cattle) with a high potential for subsequent human-to-human transmission, tb is prevalent in susceptible individuals living in overcrowded conditions [22] . although the bcg vaccine has been in routine use for many years, variable efficacy has been reported, depending on region of use [23, 24] with 60-80% reported in the united kingdom but lower levels in equatorial countries. the emergence of multi-drugresistant tb (mdr tb) in recent years raises the bar for treatment of this disease and makes a high level of vaccine efficacy even more important [25] . examples of zoonotic reservoirs which maintain endemic viral diseases are numerous and are summarized in table 3 . in particular, viral endemic disease is caused by the coronaviruses (mers and sars) in saudi arabia and the eastern mediterranean countries; and by the lassa arenavirus, which is endemic in west african countries and caused a serious outbreak of lassa fever in nigeria in 2018 [26] ; other viruses which are endemic and cause viral haemorrhagic fever include dengue, which is widespread in tropical and subtropical regions worldwide [27] , the filoviruses (ebola and marburg) which have a zoonotic reservoir in bats in sub-saharan africa [28] ; and yellow fever virus, which is prevalent in africa, central and south america and the caribbean [29] . interestingly, the same mosquito (aedes aegypti) which spreads yellow fever virus also transmits the dengue, chikungunya and zika viruses [30] . chikungunya virus has a widespread distribution in africa, asia, india and south america and with occasional cases in europe, and causes a debilitating, but rarely fatal, disease [31] . zika virus emerged in brazil in 2015 [32] , although it was first detected in monkeys in uganda as early as 1947 [33] , and the first documented human case occurred in nigeria in 1954 [34] . the large brazilian outbreak of zika viral disease culminated in 2017, with thousands of cases reported [35] . rift valley fever virus (rvfv) is another zoonotic virus which is endemic in sub-saharan africa and primarily infects cattle, sheep and goats, but can be transmitted to man by mosquito bite to cause an acute fever [36] . an example of a bacterium which has been classed as a re-emerging pathogen and which is endemic in global regions is y. pestis, causative of plague. bubonic [37] [38] [39] . over thousands of years, y. pestis has evolved away from the enteric yersinia species to become a lethal flea-transmitted bacterium [40] . it is transmitted to man, typically from a zoonotic reservoir in infected rats or other rodents (e.g. ground squirrels or prairie dogs), by flea-bite to cause bubonic plague [41] (fig. 2 ). if not detected and treated, this can develop into either septicemic plague or the most serious form of all, a secondary pneumonic plague. in turn, individuals with pneumonic plague can transmit this by aerosol droplet to others, to establish a primary pneumonic plague infection. each year, a few cases of plague are also reported in the southwestern united states, where the disease has been endemic in the rodent population since the late 1800s [42] . as well as infecting the rodent population, infected fleas can spread y. pestis to other wildlife species (e.g. rabbits/hares or, rarely, domesticated animals [43] ) which, in turn, raises the potential of transmission to man by inhalation to cause a primary pneumonic plague. in endemic areas, outbreaks of plague are often associated with seasonal environmental changes, causing rodents to stray closer to human habitation. the plague outbreak in madagascar during 2017/18 was particularly serious, with an estimated 2671 cases and 239 deaths (89% fatality rate) [44] . this outbreak was approximately sixfold greater than usual, with an unusual predominance of pneumonic, rather than bubonic plague.

</p>

</div>



<div id="info_a_edge48_142" class= "paragraph">

<h3>paragraph source</h3>

<p>a seminal paper published in 2008 listed 335 new human pathogens discovered between 1940 and 2004 [1] . the majority (60.3%) of these emerging infectious diseases (eid) originated from (wild) animal reservoirs, and approximately one in five was transmitted from animal reservoir hosts to humans by disease vectors (ticks, mosquitos, midges) [1] . pathogen discovery programs sampling wildlife hosts that are considered major reservoirs for eid (rodents, bats) have identified tremendous enzootic virus diversity, confirming their potential as sources of novel human pathogens [2] . since 2008, the discovery of severe fever with thrombocytopenia virus and middle east respiratory syndrome coronavirus (mers-cov), as well as unusual outbreaks of zika virus, yellow fever and ebola highlight the importance of demographic change, global travel and trade, and possibly climate change as drivers for emergence [3e7] (http://www.promedmail.org/, archive no. 20140322.2349696 2014-03 -22) . while the field is dominated by viruses from wildlife reservoirs, the q fever outbreak in the netherlands, the enterohemorrhagic escherichia coli outbreak in germany and the ongoing vast cholera outbreak in yemen are examples of bacterial pathogens that have expanded massivelydwhich is included in the definition of an eid (box 1) [8e10]. there is broad consensus that we need to prepare for more eid in the future.

</p>

</div>



<div id="info_a_edge48_143" class= "paragraph">

<h3>paragraph source</h3>

<p>during the past 15 years a number of new virus infections have emerged or re-emerged. most of them, such as sin nombre and andes hantaviruses, sars coronavirus, avian influenza, nipah and hendra viruses, have appeared in subtropical or tropical regions. dengue is spreading to new areas and west nile virus has reached the new world. infections by several of these viruses may lead to high mortality and also attract attention because they are potential bioweapons. some viruses such as tick-borne encephalitis virus are spreading to new areas along with climate and environmental changes. most of these infections are zoonoses and clearly viruses shared by animals and humans are, unlike human-only viruses, much more difficult to eradicate. here, we review zoonotic virus infections occurring in europe. the infections like lassa fever and dengue that are imported to europe but are not indigenous to european nature will not be discussed in detail in the review. we have divided the virus infections into two categories, those that are transmitted to humans directly from vertebrate animals (like rodents, foxes, bats and birds) and those that are primarily transmitted by arthropods (mosquitoes, ticks, sandflies). the latter class is formed by arboviruses but notably they have vertebrate hosts in nature.

</p>

</div>



<script>

//<![CDATA[

var acc = document.getElementsByClassName("edge");

var size = acc.length;

var i = 0;



for (; i < size; i++) {

    for (var k = 0; k < acc[i].childNodes.length; k++){

        if (acc[i].childNodes[k].tagName === "g"){

            acc[i].childNodes[k].style.display = "none";

        }

    }

    acc[i].addEventListener("click", function(event) {

        elems = event.target.parentNode.childNodes;

        for (var j = 0; j < elems.length; j++){

            if (elems[j].tagName === "g"){

                if (elems[j].style.display === "none"){

                    elems[j].style.display = "table";

                }

                else{

                    elems[j].style.display = "none";

                }

            }

        }

    },false);

}



var icons = document.getElementsByClassName("icon");

var size_icons = icons.length;

var j = 0;



for (; j < size_icons; j++) {



    icons[j].addEventListener("mouseover", showPopup);

    icons[j].addEventListener("mouseout", hidePopup);

}



function showPopup(evt) {



    var iconPos = this.getBoundingClientRect();

    var id = "info_".concat(this.id.substring(5));

    var paragraph = document.getElementById(id);


    paragraph.style.left = (iconPos.right + 20) + "px";
    alert(window.scrollY);
    paragraph.style.top = (window.scrollY + iconPos.top - 60) + "px";

    paragraph.style.display = "block";

}



function hidePopup(evt) {



    var id = "info_".concat(this.id.substring(5));

    var paragraph = document.getElementById(id);

    paragraph.style.display = "none";

}



//]]>

</script></body>

</html>